{
  "cells": [
    {
      "cell_type": "markdown",
      "source": [
        "Notebook2 is for generating responses from the finetuned model using dataset that has been created using notebook0"
      ],
      "metadata": {
        "id": "sKxmLKf6QfND"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "### Importing dependencies"
      ],
      "metadata": {
        "id": "JDYuXgClQtrl"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "9zu8Scr87U0A",
        "outputId": "1eba3854-bf52-43ac-c5d4-bab272779b82"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount(\"/content/drive\", force_remount=True).\n"
          ]
        }
      ],
      "source": [
        "from google.colab import drive\n",
        "drive.mount('/content/drive')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "xkhhGhWx_tP0",
        "outputId": "573e0221-5662-4ab4-f96d-f5c725813a5a"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: datasets in /usr/local/lib/python3.10/dist-packages (2.18.0)\n",
            "Requirement already satisfied: filelock in /usr/local/lib/python3.10/dist-packages (from datasets) (3.13.4)\n",
            "Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.10/dist-packages (from datasets) (1.25.2)\n",
            "Requirement already satisfied: pyarrow>=12.0.0 in /usr/local/lib/python3.10/dist-packages (from datasets) (14.0.2)\n",
            "Requirement already satisfied: pyarrow-hotfix in /usr/local/lib/python3.10/dist-packages (from datasets) (0.6)\n",
            "Requirement already satisfied: dill<0.3.9,>=0.3.0 in /usr/local/lib/python3.10/dist-packages (from datasets) (0.3.8)\n",
            "Requirement already satisfied: pandas in /usr/local/lib/python3.10/dist-packages (from datasets) (2.0.3)\n",
            "Requirement already satisfied: requests>=2.19.0 in /usr/local/lib/python3.10/dist-packages (from datasets) (2.31.0)\n",
            "Requirement already satisfied: tqdm>=4.62.1 in /usr/local/lib/python3.10/dist-packages (from datasets) (4.66.2)\n",
            "Requirement already satisfied: xxhash in /usr/local/lib/python3.10/dist-packages (from datasets) (3.4.1)\n",
            "Requirement already satisfied: multiprocess in /usr/local/lib/python3.10/dist-packages (from datasets) (0.70.16)\n",
            "Requirement already satisfied: fsspec[http]<=2024.2.0,>=2023.1.0 in /usr/local/lib/python3.10/dist-packages (from datasets) (2023.6.0)\n",
            "Requirement already satisfied: aiohttp in /usr/local/lib/python3.10/dist-packages (from datasets) (3.9.3)\n",
            "Requirement already satisfied: huggingface-hub>=0.19.4 in /usr/local/lib/python3.10/dist-packages (from datasets) (0.20.3)\n",
            "Requirement already satisfied: packaging in /usr/local/lib/python3.10/dist-packages (from datasets) (24.0)\n",
            "Requirement already satisfied: pyyaml>=5.1 in /usr/local/lib/python3.10/dist-packages (from datasets) (6.0.1)\n",
            "Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (1.3.1)\n",
            "Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (23.2.0)\n",
            "Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (1.4.1)\n",
            "Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (6.0.5)\n",
            "Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (1.9.4)\n",
            "Requirement already satisfied: async-timeout<5.0,>=4.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (4.0.3)\n",
            "Requirement already satisfied: typing-extensions>=3.7.4.3 in /usr/local/lib/python3.10/dist-packages (from huggingface-hub>=0.19.4->datasets) (4.11.0)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests>=2.19.0->datasets) (3.3.2)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests>=2.19.0->datasets) (3.6)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests>=2.19.0->datasets) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests>=2.19.0->datasets) (2024.2.2)\n",
            "Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.10/dist-packages (from pandas->datasets) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.10/dist-packages (from pandas->datasets) (2023.4)\n",
            "Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.10/dist-packages (from pandas->datasets) (2024.1)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/dist-packages (from python-dateutil>=2.8.2->pandas->datasets) (1.16.0)\n"
          ]
        }
      ],
      "source": [
        "!pip install datasets"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "### Adjusting dataset"
      ],
      "metadata": {
        "id": "DGfv4vZHQyTN"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "g96TTMlqDLPm",
        "outputId": "34564ba9-a97b-4d32-a7d2-814ef2011f27"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "We have 1344 rows with a valid response and others we dont have.\n"
          ]
        }
      ],
      "source": [
        "# loading dataset:\n",
        "from datasets import load_dataset, DatasetDict, Dataset\n",
        "import pandas as pd\n",
        "df=pd.read_json(\"/content/drive/MyDrive/gemma models/Datasets/BioNLP11ID_species_dataset_validation.json\")\n",
        "prompts=df['gemma_prompt'][:2000]\n",
        "print(f\"We have {len(prompts)} rows with a valid response and others we dont have.\")\n"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "print(prompts[0])"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "iAOE-EqiwVVm",
        "outputId": "8c9d18c6-0407-4cb4-e5af-933a0df9be1a"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "Instruction:\n",
            "Do NER on this sentence.\"enterocolitica , and association and bioconversion of the insect in case of P\"\n",
            "Reply in key value pair as Tag and Entity.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "### Hugging face authentication and model import\n",
        "- MOdels are saved in a private repo so we need to use acess token from that repo."
      ],
      "metadata": {
        "id": "QqOGlPj8Q3JN"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "from huggingface_hub import notebook_login\n",
        "from google.colab import userdata\n",
        "\n",
        "!huggingface-cli login --token hf_fNGItXcvvdgrfghjkFoFboizKtQvCp # this is not correct token ask for it from MayankD\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "FiVoe2M_yeVP",
        "outputId": "65f949e0-42a5-40c9-edfc-10208d19ec96"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Token will not been saved to git credential helper. Pass `add_to_git_credential=True` if you want to set the git credential as well.\n",
            "Token is valid (permission: read).\n",
            "Your token has been saved to /root/.cache/huggingface/token\n",
            "Login successful\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 173,
          "referenced_widgets": [
            "cdcb1e3947354fa398d4cbb64cb10fcf",
            "25048bfc6b6a44b287db0fcef1fbcd20",
            "0a0036442a6c499395b27756e526d273",
            "df4cd6edbe8b4780b87b2d6f7ad7207a",
            "3ac8cc0c89b94975b7579657180609ac",
            "abf40bbf88bd44cd8c48f2f18784ac3f",
            "1bfc6fea94474e4eb13abba0204dbc27",
            "de402b69291c4b74a77294a80d190004",
            "83eb5628623e49fbbd8a4880310fac65",
            "7f6f544bf3984a7bace828b8312d9042",
            "10432bc2ee904a6685e4cee084499d5d"
          ]
        },
        "id": "Wx0kWN688ssa",
        "outputId": "6e98382e-23f2-4877-f0b1-2ed767c70d56"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_token.py:88: UserWarning: \n",
            "The secret `HF_TOKEN` does not exist in your Colab secrets.\n",
            "To authenticate with the Hugging Face Hub, create a token in your settings tab (https://huggingface.co/settings/tokens), set it as secret in your Google Colab and restart your session.\n",
            "You will be able to reuse this secret in all of your notebooks.\n",
            "Please note that authentication is recommended but still optional to access public models or datasets.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Loading checkpoint shards:   0%|          | 0/2 [00:00<?, ?it/s]"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "cdcb1e3947354fa398d4cbb64cb10fcf"
            }
          },
          "metadata": {}
        }
      ],
      "source": [
        "from transformers import AutoModelForCausalLM, AutoTokenizer\n",
        "model_path =  \"MayankD/bloom3b_bc5cdr_species_5epochs\" # change to the path where your model is saved\n",
        "\n",
        "model = AutoModelForCausalLM.from_pretrained(model_path)\n",
        "tokenizer = AutoTokenizer.from_pretrained(model_path)\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "oBZqVcM4BfM8"
      },
      "outputs": [],
      "source": [
        "from transformers import pipeline\n",
        "# Define the pipeline with max_length set to 150\n",
        "gen = pipeline('text-generation', model=model, tokenizer=tokenizer, max_length=200, device='cuda')"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "### Generating responses\n",
        "-Also caluating time taken by each response."
      ],
      "metadata": {
        "id": "V6-RG0lBRO-F"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "9Rer6YoOAAZM",
        "outputId": "ddf8f408-60e4-4ca0-d234-5acf129c5a7d"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "1/1344 (0.68 seconds):\n",
            "{\n",
            "  \"id\": 0,\n",
            "  \"input\": \"Do NER on this sentence\\nenterocolitica , and association and bioconversion of the insect in case of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:enterocolitica']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "2/1344 (1.27 seconds):\n",
            "{\n",
            "  \"id\": 1,\n",
            "  \"input\": \"Do NER on this sentence\\nGenes involved in the biosynthesis , transport and regulation of the siderophore yersiniabactin are clustered in the high pathogenicity island of Y\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:Yersinia', 'E-Organism:yersiniae']\"\n",
            "}\n",
            "3/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 2,\n",
            "  \"input\": \"Do NER on this sentence\\nenterocolitica [ 103 ] and have counterparts in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:enterocolitica']\",\n",
            "  \"response\": \"this sentence\\nenterocolitica [ 103 ]\"\n",
            "}\n",
            "4/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 3,\n",
            "  \"input\": \"Do NER on this sentence\\nRemarkably , yersiniabactin is absent in all Y\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "5/1344 (0.95 seconds):\n",
            "{\n",
            "  \"id\": 4,\n",
            "  \"input\": \"Do NER on this sentence\\nThe proteins encoded by the P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "6/1344 (1.10 seconds):\n",
            "{\n",
            "  \"id\": 5,\n",
            "  \"input\": \"Do NER on this sentence\\nluminescens fecIRABCDE operon are similar to the components of the E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:luminescens']\",\n",
            "  \"response\": \"['E-Organism:luminescens']\"\n",
            "}\n",
            "7/1344 (0.65 seconds):\n",
            "{\n",
            "  \"id\": 6,\n",
            "  \"input\": \"Do NER on this sentence\\nHomologues are present in the genome of Y\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:luminescens']\"\n",
            "}\n",
            "8/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 7,\n",
            "  \"input\": \"Do NER on this sentence\\nIn addition , Y\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "9/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 8,\n",
            "  \"input\": \"Do NER on this sentence\\nNone of these iron acquisition systems is present in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "10/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 9,\n",
            "  \"input\": \"Do NER on this sentence\\nFurthermore , P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/transformers/pipelines/base.py:1157: UserWarning: You seem to be using the pipelines sequentially on GPU. In order to maximize efficiency please use a dataset\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[1;30;43mStreaming output truncated to the last 5000 lines.\u001b[0m\n",
            "}\n",
            "631/1344 (2.01 seconds):\n",
            "{\n",
            "  \"id\": 630,\n",
            "  \"input\": \"Do NER on this sentence\\nWe have isolated several Erwinia carotovora strains , such as Ecc15 , for their capacity to persist in the Drosophila larval gut and to trigger a strong systemic immune response following oral infection [ 8 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Erwinia', 'E-Organism:carotovora', 'S-Organism:Ecc15', 'S-Organism:Drosophila']\",\n",
            "  \"response\": \"this sentence\\nWe have isolated several Erwinia carotovora strains , such as Ecc15 , for their capacity to persist in the Drosophila larval gut and to trigger a strong systemic immune response following oral infection [ 8 ]\"\n",
            "}\n",
            "632/1344 (1.12 seconds):\n",
            "{\n",
            "  \"id\": 631,\n",
            "  \"input\": \"Do NER on this sentence\\nMore recently , we isolated Pseudomonas entomophila , a Gram - negative natural bacterial pathogen of Drosophila , from flies in Guadeloupe [ 9 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Pseudomonas', 'E-Organism:entomophila', 'S-Organism:Drosophila']\",\n",
            "  \"response\": \"this sentence\\nMore recently , we isolated Pseudomonas entomophila , a Gram - negative natural bacterial pathogen of Drosophila , from flies in Guadeloupe [ 9 ]\"\n",
            "}\n",
            "633/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 632,\n",
            "  \"input\": \"Do NER on this sentence\\nWe demonstrated that P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "634/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 633,\n",
            "  \"input\": \"Do NER on this sentence\\nImportantly , in contrast to Ecc15 , P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Ecc15']\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "635/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 634,\n",
            "  \"input\": \"Do NER on this sentence\\nThe genome sequence of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "636/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 635,\n",
            "  \"input\": \"Do NER on this sentence\\nTransposon mutagenesis of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "637/1344 (6.98 seconds):\n",
            "{\n",
            "  \"id\": 636,\n",
            "  \"input\": \"Do NER on this sentence\\nAmong them we identified the GacS / GacA two - component regulatory system which , in Pseudomonas fluorescens , has been well characterized and demonstrated to be an essential regulator of virulence [ 11 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Pseudomonas', 'E-Organism:fluorescens']\",\n",
            "  \"response\": \"this sentence\\nAmong them we identified the GacS / GacA two - component regulatory system which , in Pseudomonas fluorescens , has been well characterized and demonstrated to be an essential regulator of virulence [ 11 ]\"\n",
            "}\n",
            "638/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 637,\n",
            "  \"input\": \"Do NER on this sentence\\nHere we report an in vivo analysis of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "639/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 638,\n",
            "  \"input\": \"Do NER on this sentence\\nWe present in vivo evidence for a direct role in pathogenesis of the secreted P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:group', 'E-Organism:streptococci']\"\n",
            "}\n",
            "640/1344 (0.22 seconds):\n",
            "{\n",
            "  \"id\": 639,\n",
            "  \"input\": \"Do NER on this sentence\\nFurthermore , we demonstrate that the Drosophila Imd - regulated local gut response is necessary to combat infection and that P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Drosophila']\",\n",
            "  \"response\": \"this sentence\\nFurthermore , we demonstrate that the Drosophila Imd - regulated local gut response is necessary to combat infection and that P\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "641/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 640,\n",
            "  \"input\": \"Do NER on this sentence\\nResults P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "642/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 641,\n",
            "  \"input\": \"Do NER on this sentence\\nTo test whether P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "643/1344 (1.01 seconds):\n",
            "{\n",
            "  \"id\": 642,\n",
            "  \"input\": \"Do NER on this sentence\\nFigure 1A shows that a concentrated P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "644/1344 (0.42 seconds):\n",
            "{\n",
            "  \"id\": 643,\n",
            "  \"input\": \"Do NER on this sentence\\nNo killing of either larvae or adults was observed with a culture filtrate derived from the avirulent P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "645/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 644,\n",
            "  \"input\": \"Do NER on this sentence\\nThese data suggested that P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "646/1344 (0.56 seconds):\n",
            "{\n",
            "  \"id\": 645,\n",
            "  \"input\": \"Do NER on this sentence\\nThe protein profile for wild - type P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "647/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 646,\n",
            "  \"input\": \"Do NER on this sentence\\nWe subsequently purified this protease , termed AprA , to homogeneity from the P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "648/1344 (2.25 seconds):\n",
            "{\n",
            "  \"id\": 647,\n",
            "  \"input\": \"Do NER on this sentence\\n5 mg / ml ) of AprA led to modest lethality ( unpublished data ) , recapitulating the properties of the P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:invasive', 'E-Organism:GAS', 'B-Organism:non', 'I-Organism:invasive', 'E-Organism:GAS']\"\n",
            "}\n",
            "649/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 648,\n",
            "  \"input\": \"Do NER on this sentence\\nIn P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "650/1344 (7.63 seconds):\n",
            "{\n",
            "  \"id\": 649,\n",
            "  \"input\": \"Do NER on this sentence\\nTwo independent Tn5 insertions in the prtR gene affecting virulence have recently been identified in a random insertional mutagenesis screen of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "651/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 650,\n",
            "  \"input\": \"Do NER on this sentence\\nMeasurement of in vitro protease activity with azocasein revealed that the gacA mutant secretes no detectable protease activity and that prtR mutants retain only 30 % of wild - type supernatant activity , thus indicating a role for PrtR in the regulation of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "652/1344 (0.06 seconds):\n",
            "{\n",
            "  \"id\": 651,\n",
            "  \"input\": \"Do NER on this sentence\\nAltogether , this analysis indicates that P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "653/1344 (0.62 seconds):\n",
            "{\n",
            "  \"id\": 652,\n",
            "  \"input\": \"Do NER on this sentence\\nentomophila aprA Mutant Displays Attenuated Pathogenicity The experiments described above suggested that AprA plays an important in vivo role in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:entomophila']\"\n",
            "}\n",
            "654/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 653,\n",
            "  \"input\": \"Do NER on this sentence\\nTo analyze the in vivo contribution of AprA to P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "655/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 654,\n",
            "  \"input\": \"Do NER on this sentence\\nentomophila virulence , we performed a survival analysis of Drosophila larvae and adults after oral infection with various P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:entomophila', 'S-Organism:Drosophila']\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "656/1344 (0.72 seconds):\n",
            "{\n",
            "  \"id\": 655,\n",
            "  \"input\": \"Do NER on this sentence\\nOnly 40 % of the aprA - infected larvae succumbed within 3 days , while 70 % of the larvae died after infection with wild - type P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:faecalis']\"\n",
            "}\n",
            "657/1344 (8.66 seconds):\n",
            "{\n",
            "  \"id\": 656,\n",
            "  \"input\": \"Do NER on this sentence\\nWe next compared persistence of wild - type P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "658/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 657,\n",
            "  \"input\": \"Do NER on this sentence\\nWhereas the wild - type P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "659/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 658,\n",
            "  \"input\": \"Do NER on this sentence\\nIn support of this hypothesis , previous studies have demonstrated for both Serratia marcescens and P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Serratia', 'E-Organism:marcescens']\",\n",
            "  \"response\": \"['B-Organism:Serratia', 'E-Organism:marcescens']\"\n",
            "}\n",
            "660/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 659,\n",
            "  \"input\": \"Do NER on this sentence\\nThese data confirmed that the protease AprA itself contributes to P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "661/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 660,\n",
            "  \"input\": \"Do NER on this sentence\\nThe aprA Mutant Retains the Capacity to Trigger an Immune Response and to Induce Food - Uptake Cessation In addition to its pathogenicity , P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "662/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 661,\n",
            "  \"input\": \"Do NER on this sentence\\nIn contrast , P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "663/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 662,\n",
            "  \"input\": \"Do NER on this sentence\\nThe kinetics of the expression of the AMP , Diptericin , in larvae and adults after natural infection by P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "664/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 663,\n",
            "  \"input\": \"Do NER on this sentence\\nThe data presented above clearly indicate that AprA is a major virulence factor of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "665/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 664,\n",
            "  \"input\": \"Do NER on this sentence\\nIt also reveals that P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "666/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 665,\n",
            "  \"input\": \"Do NER on this sentence\\nThe observation that both factors are under the control of the GacS / GacA , confirmed that this two - component system is the master regulator of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "667/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 666,\n",
            "  \"input\": \"Do NER on this sentence\\nThe observation that P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "668/1344 (1.51 seconds):\n",
            "{\n",
            "  \"id\": 667,\n",
            "  \"input\": \"Do NER on this sentence\\nOur previous study had already revealed that the humoral arm of the Drosophila immune response mediated by the Imd pathway contributes to resistance against orally transmitted P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Drosophila']\",\n",
            "  \"response\": \"['B-Organism:invasive', 'E-Organism:GAS', 'S-Organism:GAS']\"\n",
            "}\n",
            "669/1344 (1.96 seconds):\n",
            "{\n",
            "  \"id\": 668,\n",
            "  \"input\": \"Do NER on this sentence\\nTo determine whether AprA plays a role in counteracting the Imd - dependent immune response , we compared persistence of wild - type P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['S-Organism:Pseudomonas', 'B-Organism:Salmonella', 'E-Organism:enterica', 'S-Organism:human']\"\n",
            "}\n",
            "670/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 669,\n",
            "  \"input\": \"Do NER on this sentence\\nTo investigate the connection between activation of the Imd pathway and the outcome of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "671/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 670,\n",
            "  \"input\": \"Do NER on this sentence\\nP\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "672/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 671,\n",
            "  \"input\": \"Do NER on this sentence\\nTo determine at which time Imd was required to impede P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "673/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 672,\n",
            "  \"input\": \"Do NER on this sentence\\nentomophila infection , we monitored survival to P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:entomophila']\",\n",
            "  \"response\": \"['E-Organism:entomophila']\"\n",
            "}\n",
            "674/1344 (1.41 seconds):\n",
            "{\n",
            "  \"id\": 673,\n",
            "  \"input\": \"Do NER on this sentence\\nFigure 4A shows that over - expressing imd 12 h prior to infection protected flies from a P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:pestis', 'S-Organism:flea', 'S-Organism:fly']\"\n",
            "}\n",
            "675/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 674,\n",
            "  \"input\": \"Do NER on this sentence\\nThis experiment indicates that P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "676/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 675,\n",
            "  \"input\": \"Do NER on this sentence\\nIt further suggests that P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "677/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 676,\n",
            "  \"input\": \"Do NER on this sentence\\nSince P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "678/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 677,\n",
            "  \"input\": \"Do NER on this sentence\\nAltogether , these data reveal that the Imd pathway is crucial for host survival after oral infection with P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "679/1344 (3.30 seconds):\n",
            "{\n",
            "  \"id\": 678,\n",
            "  \"input\": \"Do NER on this sentence\\nLocal , but Not Systemic Immunity , Contributes to Resistance against Oral Infection with P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['I-Organism:enterocolitica', 'I-Organism:strain', 'E-Organism:W14']\"\n",
            "}\n",
            "680/1344 (0.80 seconds):\n",
            "{\n",
            "  \"id\": 679,\n",
            "  \"input\": \"Do NER on this sentence\\nentomophila P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:entomophila']\",\n",
            "  \"response\": \"['E-Organism:entomophila']\"\n",
            "}\n",
            "681/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 680,\n",
            "  \"input\": \"Do NER on this sentence\\nWe next examined the respective contribution of the local gut and the systemic immune responses to controlling P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "682/1344 (0.46 seconds):\n",
            "{\n",
            "  \"id\": 681,\n",
            "  \"input\": \"Do NER on this sentence\\nWe thus compared resistance to P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "683/1344 (0.86 seconds):\n",
            "{\n",
            "  \"id\": 682,\n",
            "  \"input\": \"Do NER on this sentence\\nFigure 5C shows that a prior infection with Ecc15 protected flies against a subsequent P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Ecc15']\",\n",
            "  \"response\": \"['S-Organism:Drosophila']\"\n",
            "}\n",
            "684/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 683,\n",
            "  \"input\": \"Do NER on this sentence\\nentomophila infection only when Ecc15 was administrated orally , indicating that the local , but not the systemic immune response , plays an important role in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:entomophila', 'S-Organism:Ecc15']\",\n",
            "  \"response\": \"['E-Organism:entomophila']\"\n",
            "}\n",
            "685/1344 (8.16 seconds):\n",
            "{\n",
            "  \"id\": 684,\n",
            "  \"input\": \"Do NER on this sentence\\nTo ascertain that the protection was due to Imd pathway activation and not to a competition between P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "686/1344 (0.81 seconds):\n",
            "{\n",
            "  \"id\": 685,\n",
            "  \"input\": \"Do NER on this sentence\\nentomophila and Ecc15 , we next monitored resistance to P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:entomophila', 'S-Organism:Ecc15']\",\n",
            "  \"response\": \"['E-Organism:entomophila']\"\n",
            "}\n",
            "687/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 686,\n",
            "  \"input\": \"Do NER on this sentence\\nThis experiment shows that wild - type , but not Rel flies previously infected with Ecc15 evf - , resisted a second challenge by P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['S-Organism:Drosophila']\"\n",
            "}\n",
            "688/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 687,\n",
            "  \"input\": \"Do NER on this sentence\\nOver - expression of Imd in the gut protected against P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "689/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 688,\n",
            "  \"input\": \"Do NER on this sentence\\nHowever , the previous results did not address whether the Imd pathway induced during the course of a natural P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "690/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 689,\n",
            "  \"input\": \"Do NER on this sentence\\nentomophila infection contributes to fly resistance to P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:entomophila']\",\n",
            "  \"response\": \"['E-Organism:entomophila']\"\n",
            "}\n",
            "691/1344 (0.06 seconds):\n",
            "{\n",
            "  \"id\": 690,\n",
            "  \"input\": \"Do NER on this sentence\\nThese data demonstrate that the P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "692/1344 (1.80 seconds):\n",
            "{\n",
            "  \"id\": 691,\n",
            "  \"input\": \"Do NER on this sentence\\nAprA Confers Protection against Diptericin We have shown that AprA protects P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "693/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 692,\n",
            "  \"input\": \"Do NER on this sentence\\nWe confirmed that P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "694/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 693,\n",
            "  \"input\": \"Do NER on this sentence\\nTo investigate a possible in vivo role of AprA in the protection against AMPs synthesized in the gut , we compared survival to P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\"\n",
            "}\n",
            "695/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 694,\n",
            "  \"input\": \"Do NER on this sentence\\nThis strategy allowed us to study the contribution of each antibacterial peptide to the defense against a P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "696/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 695,\n",
            "  \"input\": \"Do NER on this sentence\\nIn a first set of experiments , we compared the survival rate of wild - type , imd , and imd ; da > Dpt or AttA flies to natural infection with P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:Pseudomonas', 'E-Organism:aeruginosa']\"\n",
            "}\n",
            "697/1344 (0.28 seconds):\n",
            "{\n",
            "  \"id\": 696,\n",
            "  \"input\": \"Do NER on this sentence\\nThis experiment demonstrates that these AMPs can confer protection against P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "698/1344 (0.46 seconds):\n",
            "{\n",
            "  \"id\": 697,\n",
            "  \"input\": \"Do NER on this sentence\\nTo investigate the in vivo relationship between AMP activity and microbial persistence we next compared the numbers of bacteria of wild - type P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "699/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 698,\n",
            "  \"input\": \"Do NER on this sentence\\nAltogether , our results provide an in vivo demonstration that AprA functions in the protection of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "700/1344 (1.27 seconds):\n",
            "{\n",
            "  \"id\": 699,\n",
            "  \"input\": \"Do NER on this sentence\\nTo study how flies deal with infections in a natural context , we investigated the interactions between Drosophila and a newly identified entomopathogen , P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Drosophila']\",\n",
            "  \"response\": \"['B-Organism:Drosophila', 'E-Organism:melanogaster']\"\n",
            "}\n",
            "701/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 700,\n",
            "  \"input\": \"Do NER on this sentence\\nOur analysis reveals that P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "702/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 701,\n",
            "  \"input\": \"Do NER on this sentence\\nTo explore the potential implication of AMPs in the Imd immune defense against oral infection by P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "703/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 702,\n",
            "  \"input\": \"Do NER on this sentence\\nOur study reveals that over - expression of either Diptericin or Attacin confers protection against P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "704/1344 (0.06 seconds):\n",
            "{\n",
            "  \"id\": 703,\n",
            "  \"input\": \"Do NER on this sentence\\nThis observation , combined with our data demonstrating that P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "705/1344 (0.06 seconds):\n",
            "{\n",
            "  \"id\": 704,\n",
            "  \"input\": \"Do NER on this sentence\\nFinally , it remains surprising that P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "706/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 705,\n",
            "  \"input\": \"Do NER on this sentence\\nAlternatively , our results suggest the possibility that bacteria such as P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "707/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 706,\n",
            "  \"input\": \"Do NER on this sentence\\nAprA Protects P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "708/1344 (1.56 seconds):\n",
            "{\n",
            "  \"id\": 707,\n",
            "  \"input\": \"Do NER on this sentence\\nProteases , especially metalloproteases , are also known to contribute to virulence in some pathogenic bacteria including Pseudomonas aeruginosa , S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Pseudomonas', 'E-Organism:aeruginosa']\",\n",
            "  \"response\": \"['B-Organism:Sodalis', 'E-Organism:glossinidius', 'S-Organism:Sodalis']\"\n",
            "}\n",
            "709/1344 (1.41 seconds):\n",
            "{\n",
            "  \"id\": 708,\n",
            "  \"input\": \"Do NER on this sentence\\nmarcescens , and Bacillus thuringiensis [ 20 , 21 , 26 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:marcescens', 'B-Organism:Bacillus', 'E-Organism:thuringiensis']\",\n",
            "  \"response\": \"this sentence\\nmarcescens , and Bacillus thuringiensis [ 20 , 21 , 26 ]\"\n",
            "}\n",
            "710/1344 (0.22 seconds):\n",
            "{\n",
            "  \"id\": 709,\n",
            "  \"input\": \"Do NER on this sentence\\nThis conclusion is based on our observations that ( i ) aprA mutants show attenuated virulence , ( ii ) aprA mutants survive less well than wild - type P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nThis conclusion is based on our observations that ( i ) aprA mutants show attenuated virulence , ( ii ) aprA mutants survive less well than wild - type P\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "711/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 710,\n",
            "  \"input\": \"Do NER on this sentence\\nAltogether , our study reveals that local AMP expression plays an important role in defense against oral pathogens and that entomopathogens such as P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "712/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 711,\n",
            "  \"input\": \"Do NER on this sentence\\nIt is interesting to note that S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "713/1344 (1.21 seconds):\n",
            "{\n",
            "  \"id\": 712,\n",
            "  \"input\": \"Do NER on this sentence\\nHowever , in the absence of a Serratia protease mutant , the in vivo relevance of the protease is not yet established\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Serratia', 'I-Organism:protease', 'E-Organism:mutant']\",\n",
            "  \"response\": \"['B-Organism:Serratia', 'E-Organism:entomophila']\"\n",
            "}\n",
            "714/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 713,\n",
            "  \"input\": \"Do NER on this sentence\\nIn B\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "715/1344 (1.96 seconds):\n",
            "{\n",
            "  \"id\": 714,\n",
            "  \"input\": \"Do NER on this sentence\\nAlternatively , in Photorhabdus luminescens , an AprA homologue , PrtA , has been proposed not to be a virulence factor , but rather to be involved in degrading host tissues following host death in order to promote efficient nutrition and development of its symbiotic nematode host [ 14 , 29 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Photorhabdus', 'E-Organism:luminescens']\",\n",
            "  \"response\": \"this sentence\\nAlternatively , in Photorhabdus luminescens , an AprA homologue , PrtA , has been proposed not to be a virulence factor , but rather to be involved in degrading host tissues following host death in order to promote efficient nutrition and development of its symbiotic nematode host [ 14 , 29 ]\"\n",
            "}\n",
            "716/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 715,\n",
            "  \"input\": \"Do NER on this sentence\\nP\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "717/1344 (0.62 seconds):\n",
            "{\n",
            "  \"id\": 716,\n",
            "  \"input\": \"Do NER on this sentence\\nentomophila Virulence Is Multi - Factorial In agreement with previous studies , we show that aprA expression in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:entomophila']\",\n",
            "  \"response\": \"['E-Organism:entomophila']\"\n",
            "}\n",
            "718/1344 (0.46 seconds):\n",
            "{\n",
            "  \"id\": 717,\n",
            "  \"input\": \"Do NER on this sentence\\nOur study indicates that it is the master regulator of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "719/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 718,\n",
            "  \"input\": \"Do NER on this sentence\\nAs opposed to the pleiotropic effects of GacS / GacA , PrtR appears to be a more specific regulator of aprA expression in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "720/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 719,\n",
            "  \"input\": \"Do NER on this sentence\\nentomophila , reminiscent of aprX regulation in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:entomophila']\",\n",
            "  \"response\": \"['E-Organism:entomophila']\"\n",
            "}\n",
            "721/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 720,\n",
            "  \"input\": \"Do NER on this sentence\\nThe difference in pathogenicity exhibited by gacA , prtR , and aprA mutants underlines the complexity of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "722/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 721,\n",
            "  \"input\": \"Do NER on this sentence\\nThis hypothesized persistence - promoting factor may have a function similar to the Erwinia virulence factor ( evf ) gene of E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "723/1344 (1.61 seconds):\n",
            "{\n",
            "  \"id\": 722,\n",
            "  \"input\": \"Do NER on this sentence\\ncarotovora Ecc15 that promotes persistence of Gram - negative bacteria in the larval gut [ 34 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:carotovora', 'E-Organism:Ecc15']\",\n",
            "  \"response\": \"this sentence\\ncarotovora Ecc15 that promotes persistence of Gram - negative bacteria in the larval gut [ 34 ]\"\n",
            "}\n",
            "724/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 723,\n",
            "  \"input\": \"Do NER on this sentence\\nAnother interesting feature of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "725/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 724,\n",
            "  \"input\": \"Do NER on this sentence\\nFood - uptake cessation or blockage in insects is induced by several other entomopathogenic bacteria including S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['S-Organism:Drosophila']\"\n",
            "}\n",
            "726/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 725,\n",
            "  \"input\": \"Do NER on this sentence\\nentomophila and Yersinia pestis , enabling persistence in the digestive tract of their insect hosts [ 35 , 36 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:entomophila', 'B-Organism:Yersinia', 'E-Organism:pestis']\",\n",
            "  \"response\": \"this sentence\\nentomophila and Yersinia pestis , enabling persistence in the digestive tract of their insect hosts [ 35 , 36 ]\"\n",
            "}\n",
            "727/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 726,\n",
            "  \"input\": \"Do NER on this sentence\\nIn S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "728/1344 (0.72 seconds):\n",
            "{\n",
            "  \"id\": 727,\n",
            "  \"input\": \"Do NER on this sentence\\nentomophila , it has been shown that genes encoded by a prophage were responsible for the anti - feeding reaction in its natural insect host , the grass grub Costelytra zealandica larvae [ 35 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:entomophila', 'B-Organism:Costelytra', 'E-Organism:zealandica']\",\n",
            "  \"response\": \"this sentence\\nentomophila , it has been shown that genes encoded by a prophage were responsible for the anti - feeding reaction in its natural insect host , the grass grub Costelytra zealandica larvae [ 35 ]\"\n",
            "}\n",
            "729/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 728,\n",
            "  \"input\": \"Do NER on this sentence\\nY\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "730/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 729,\n",
            "  \"input\": \"Do NER on this sentence\\nThe absence of homologous genes to these factors in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "731/1344 (0.81 seconds):\n",
            "{\n",
            "  \"id\": 730,\n",
            "  \"input\": \"Do NER on this sentence\\nFinally , our study shows that aprA , but not gacA , mutants of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "732/1344 (0.95 seconds):\n",
            "{\n",
            "  \"id\": 731,\n",
            "  \"input\": \"Do NER on this sentence\\nThe genome of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "733/1344 (0.06 seconds):\n",
            "{\n",
            "  \"id\": 732,\n",
            "  \"input\": \"Do NER on this sentence\\nTherefore , it remains to be determined whether the strategies developed by P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "734/1344 (1.02 seconds):\n",
            "{\n",
            "  \"id\": 733,\n",
            "  \"input\": \"Do NER on this sentence\\nApplication of this method to Escherichia coli motility identifies 95 % of the known flagellar and chemotaxis genes , and reveals three dozen novel loci that , to varying degrees and through diverse mechanisms , affect motility\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Escherichia', 'E-Organism:coli']\",\n",
            "  \"response\": \"['B-Organism:Escherichia', 'E-Organism:coli']\"\n",
            "}\n",
            "735/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 734,\n",
            "  \"input\": \"Do NER on this sentence\\nThe best - characterized chemotaxis machineries are those of E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "736/1344 (1.45 seconds):\n",
            "{\n",
            "  \"id\": 735,\n",
            "  \"input\": \"Do NER on this sentence\\ncoli and its close relative Salmonella enterica [ 5 , 6 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:coli', 'B-Organism:Salmonella', 'E-Organism:enterica']\",\n",
            "  \"response\": \"this sentence\\ncoli and its close relative Salmonella enterica [ 5 , 6 ]\"\n",
            "}\n",
            "737/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 736,\n",
            "  \"input\": \"Do NER on this sentence\\nWe demonstrate the manner in which this strategy can be used to explore the genetic basis of complex bacterial behavior by following the population dynamics of E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:luminescens']\"\n",
            "}\n",
            "738/1344 (1.82 seconds):\n",
            "{\n",
            "  \"id\": 737,\n",
            "  \"input\": \"Do NER on this sentence\\nWe have used a hyperactive Tn5 transposition system with reduced target specificity [ 16 ] to create a library of approximately 5 x 105 independent transposon insertional mutants in E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nWe have used a hyperactive Tn5 transposition system with reduced target specificity [ 16 ]\"\n",
            "}\n",
            "739/1344 (1.05 seconds):\n",
            "{\n",
            "  \"id\": 738,\n",
            "  \"input\": \"Do NER on this sentence\\ncoli MG1655\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:coli', 'E-Organism:MG1655']\",\n",
            "  \"response\": \"['I-Organism:coli', 'E-Organism:MG1655']\"\n",
            "}\n",
            "740/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 739,\n",
            "  \"input\": \"Do NER on this sentence\\nGenetic Basis of Swimming Motility and Chemotaxis Semi - solid tryptone agar provides a robust setting for studying E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "741/1344 (0.22 seconds):\n",
            "{\n",
            "  \"id\": 740,\n",
            "  \"input\": \"Do NER on this sentence\\nThe neutral behavior of the fliT mutation is not surprising since fliT null strains were found to make functional flagella in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nThe neutral behavior of the fliT mutation is not surprising since fliT null strains were found to make functional flagella in S\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "742/1344 (0.95 seconds):\n",
            "{\n",
            "  \"id\": 741,\n",
            "  \"input\": \"Do NER on this sentence\\nBacteriophage chi infects E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:chi']\",\n",
            "  \"response\": \"['E-Organism:chi']\"\n",
            "}\n",
            "743/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 742,\n",
            "  \"input\": \"Do NER on this sentence\\nA single round of enrichment consisted of inoculating our library of E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "744/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 743,\n",
            "  \"input\": \"Do NER on this sentence\\nAlthough this observation contradicts earlier studies on swarming in E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "745/1344 (1.45 seconds):\n",
            "{\n",
            "  \"id\": 744,\n",
            "  \"input\": \"Do NER on this sentence\\ncoli and S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:coli']\",\n",
            "  \"response\": \"['E-Organism:coli', 'B-Organism:Vibrio', 'E-Organism:cholerae']\"\n",
            "}\n",
            "746/1344 (0.22 seconds):\n",
            "{\n",
            "  \"id\": 745,\n",
            "  \"input\": \"Do NER on this sentence\\nYhjH contains an EAL domain , and expression of this protein leads to the degradation of c - di - GMP and enhanced motility in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nYhjH contains an EAL domain , and expression of this protein leads to the degradation of c - di - GMP and enhanced motility in S\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "747/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 746,\n",
            "  \"input\": \"Do NER on this sentence\\nE\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "748/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 747,\n",
            "  \"input\": \"Do NER on this sentence\\nThe Genetic Basis of Environmentally Induced Repression of Motility Chemotaxis in E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "749/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 748,\n",
            "  \"input\": \"Do NER on this sentence\\nWe demonstrated this approach in four distinct but complementary selections to reveal the genetic basis of E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "750/1344 (1.37 seconds):\n",
            "{\n",
            "  \"id\": 749,\n",
            "  \"input\": \"Do NER on this sentence\\nLarge scale variation in Enterococcus faecalis illustrated by the genome analysis of strain OG1RF A comparison of two strains of the hospital pathogen Enterococcus faecalis suggests that mediators of virulence differ between strains and that virulence does not depend on mobile gene elements Background Enterococci have emerged over the past few decades as the second to third most common cause of nosocomial infections , including urinary tract and soft tissue infections , bacteremia , and endocarditis [ 1 - 3 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Enterococcus', 'E-Organism:faecalis', 'S-Organism:OG1RF', 'B-Organism:Enterococcus', 'E-Organism:faecalis', 'S-Organism:Enterococci']\",\n",
            "  \"response\": \"this sentence\\nLarge scale variation in Enterococcus faecalis illustrated by the genome analysis of strain OG1RF A comparison of two strains of the hospital pathogen Enterococcus faecalis suggests that mediators of virulence differ between strains and that virulence does not depend on mobile gene elements Background Enterococci have emerged over the past few decades as the second to third most common cause of nosocomial infections , including urinary tract and soft tissue infections , bacteremia , and endocarditis [ 1 - 3 ]\"\n",
            "}\n",
            "751/1344 (1.35 seconds):\n",
            "{\n",
            "  \"id\": 750,\n",
            "  \"input\": \"Do NER on this sentence\\nEntecococcal infections are predominantly caused by E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:Ecc15']\"\n",
            "}\n",
            "752/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 751,\n",
            "  \"input\": \"Do NER on this sentence\\nfaecalis and E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:faecalis']\",\n",
            "  \"response\": \"['E-Organism:faecalis']\"\n",
            "}\n",
            "753/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 752,\n",
            "  \"input\": \"Do NER on this sentence\\nHowever , many , if not most , strains of these species are harmless commensals , with some enterococci being marketed in Europe to alleviate symptoms of irritable bowel syndrome and recurrent chronic sinusitis or bronchitis ( Cylactin ( R ) and Fargo688 ( R ) ( E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:enterococci', 'S-Organism:Cylactin', 'S-Organism:Fargo688']\",\n",
            "  \"response\": \"['E-Organism:enterococci']\"\n",
            "}\n",
            "754/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 753,\n",
            "  \"input\": \"Do NER on this sentence\\nfaecium ) and Symbioflor 1 ( E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:faecium', 'B-Organism:Symbioflor']\",\n",
            "  \"response\": \"['E-Organism:faecium']\"\n",
            "}\n",
            "755/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 754,\n",
            "  \"input\": \"Do NER on this sentence\\nAlthough hundreds of microbial genomes have been sequenced , only two E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "756/1344 (0.72 seconds):\n",
            "{\n",
            "  \"id\": 755,\n",
            "  \"input\": \"Do NER on this sentence\\nfaecalis genomes have been reported ( V583 as a clinical isolate [ 4 ] and Symbioflor 1 as a commensal isolate [ 5 ] ) , but only the V583 genome has been made publicly available\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:faecalis', 'S-Organism:V583', 'B-Organism:Symbioflor', 'S-Organism:V583']\",\n",
            "  \"response\": \"this sentence\\nfaecalis genomes have been reported ( V583 as a clinical isolate [ 4 ]\"\n",
            "}\n",
            "757/1344 (1.67 seconds):\n",
            "{\n",
            "  \"id\": 756,\n",
            "  \"input\": \"Do NER on this sentence\\nOG1 and its derivatives have been successfully used over the past 20 years in various animal models , starting with the demonstration that it can cause caries in germ - free rats [ 7 ] , and later to characterize factors important for E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:OG1', 'S-Organism:rats']\",\n",
            "  \"response\": \"this sentence\\nOG1 and its derivatives have been successfully used over the past 20 years in various animal models , starting with the demonstration that it can cause caries in germ - free rats [ 7 ]\"\n",
            "}\n",
            "758/1344 (2.06 seconds):\n",
            "{\n",
            "  \"id\": 757,\n",
            "  \"input\": \"Do NER on this sentence\\nOG1RF was also shown to be as virulent as V583 in the model host Caenorhabditis elegans [ 17 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:OG1RF', 'S-Organism:V583', 'B-Organism:Caenorhabditis', 'E-Organism:elegans']\",\n",
            "  \"response\": \"this sentence\\nOG1RF was also shown to be as virulent as V583 in the model host Caenorhabditis elegans [ 17 ]\"\n",
            "}\n",
            "759/1344 (1.51 seconds):\n",
            "{\n",
            "  \"id\": 758,\n",
            "  \"input\": \"Do NER on this sentence\\nThe ace gene , like the ebp locus , is ubiquitous in E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "760/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 759,\n",
            "  \"input\": \"Do NER on this sentence\\nAce mediates conditional ( that is , after growth at 46degreesC or in the presence of serum or collagen ) adherence of E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "761/1344 (0.31 seconds):\n",
            "{\n",
            "  \"id\": 760,\n",
            "  \"input\": \"Do NER on this sentence\\nThus , we analyzed the sequence of E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "762/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 761,\n",
            "  \"input\": \"Do NER on this sentence\\nfaecalis OG1RF\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:faecalis', 'E-Organism:OG1RF']\",\n",
            "  \"response\": \"['E-Organism:faecalis']\"\n",
            "}\n",
            "763/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 762,\n",
            "  \"input\": \"Do NER on this sentence\\nThe presence of the CRISPR loci among E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "764/1344 (1.11 seconds):\n",
            "{\n",
            "  \"id\": 763,\n",
            "  \"input\": \"Do NER on this sentence\\nTo determine the distribution of the CRISPR1 locus in E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "765/1344 (0.95 seconds):\n",
            "{\n",
            "  \"id\": 764,\n",
            "  \"input\": \"Do NER on this sentence\\nIn a more recent analysis of relationships between various E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "766/1344 (0.81 seconds):\n",
            "{\n",
            "  \"id\": 765,\n",
            "  \"input\": \"Do NER on this sentence\\nThe Symbioflor 1 strain , used as a probiotic , is one representative of the 7\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Symbioflor']\",\n",
            "  \"response\": \"['B-Organism:Symbioflor']\"\n",
            "}\n",
            "767/1344 (0.42 seconds):\n",
            "{\n",
            "  \"id\": 766,\n",
            "  \"input\": \"Do NER on this sentence\\n4 % of E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "768/1344 (1.57 seconds):\n",
            "{\n",
            "  \"id\": 767,\n",
            "  \"input\": \"Do NER on this sentence\\n8 kb region contains two loci , a WxL locus ( described below ) and a seven - gene locus that may encode a possible ABC transporter with similarity to one annotated in Pediococcus pentosaceus\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Pediococcus', 'E-Organism:pentosaceus']\",\n",
            "  \"response\": \"['B-Organism:Lactobacillus', 'E-Organism:casei']\"\n",
            "}\n",
            "769/1344 (0.31 seconds):\n",
            "{\n",
            "  \"id\": 768,\n",
            "  \"input\": \"Do NER on this sentence\\nComponents of the cell surface It has been shown in E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "770/1344 (0.95 seconds):\n",
            "{\n",
            "  \"id\": 769,\n",
            "  \"input\": \"Do NER on this sentence\\nWe found more polymorphisms in two families of E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "771/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 770,\n",
            "  \"input\": \"Do NER on this sentence\\nAnother family of E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "772/1344 (0.81 seconds):\n",
            "{\n",
            "  \"id\": 771,\n",
            "  \"input\": \"Do NER on this sentence\\nIn addition , it appears that the EF3248 locus diverges in the Symbioflor 1 strain\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Symbioflor']\",\n",
            "  \"response\": \"['B-Organism:Symbioflor']\"\n",
            "}\n",
            "773/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 772,\n",
            "  \"input\": \"Do NER on this sentence\\nHowever , because the Symbioflor 1 genome sequence is not currently available , it was not possible to compare their respective sequences in more detail\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Symbioflor']\",\n",
            "  \"response\": \"['B-Organism:Symbioflor']\"\n",
            "}\n",
            "774/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 773,\n",
            "  \"input\": \"Do NER on this sentence\\nMore analyses are required for a better understanding of the number , frequency and function of these WxL domain proteins and their possible relationship with the diversity of E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "775/1344 (1.20 seconds):\n",
            "{\n",
            "  \"id\": 774,\n",
            "  \"input\": \"Do NER on this sentence\\nFinally , as previously found using PCR , the cpsCDEFGHIJK operon capsule polysaccharide genes [ 37 ] were confirmed here as missing , although OG1RF carries the cpsA and cpsB genes , which were proposed to be essential for E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:OG1RF']\",\n",
            "  \"response\": \"this sentence\\nFinally , as previously found using PCR , the cpsCDEFGHIJK operon capsule polysaccharide genes [ 37 ]\"\n",
            "}\n",
            "776/1344 (1.91 seconds):\n",
            "{\n",
            "  \"id\": 775,\n",
            "  \"input\": \"Do NER on this sentence\\nMany soil and plant micro - organisms , including Bacillus subtilis [ 41 ] ( first iol operon identified ) , Klebsiella spp\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Bacillus', 'E-Organism:subtilis', 'S-Organism:Klebsiella']\",\n",
            "  \"response\": \"this sentence\\nMany soil and plant micro - organisms , including Bacillus subtilis [ 41 ]\"\n",
            "}\n",
            "777/1344 (1.27 seconds):\n",
            "{\n",
            "  \"id\": 776,\n",
            "  \"input\": \"Do NER on this sentence\\nThe OG1RF iol operon appears to be closely related to ones described in Clostridium perfringens [ 44 ] and Lactobacillus casei [ 45 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:OG1RF', 'B-Organism:Clostridium', 'E-Organism:perfringens', 'B-Organism:Lactobacillus', 'E-Organism:casei']\",\n",
            "  \"response\": \"this sentence\\nThe OG1RF iol operon appears to be closely related to ones described in Clostridium perfringens [ 44 ]\"\n",
            "}\n",
            "778/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 777,\n",
            "  \"input\": \"Do NER on this sentence\\nIn L\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "779/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 778,\n",
            "  \"input\": \"Do NER on this sentence\\nHowever , the order of the genes is not the same between E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nHowever , the order of the genes is not the same between E\\nReply in key value pair as Tag and Entity.\\\"Response\\n[]\"\n",
            "}\n",
            "780/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 779,\n",
            "  \"input\": \"Do NER on this sentence\\nfaecalis and L\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:faecalis']\",\n",
            "  \"response\": \"['E-Organism:faecalis']\"\n",
            "}\n",
            "781/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 780,\n",
            "  \"input\": \"Do NER on this sentence\\nIn addition , iolH , iolJ and iolK , present in L\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "782/1344 (1.46 seconds):\n",
            "{\n",
            "  \"id\": 781,\n",
            "  \"input\": \"Do NER on this sentence\\ncasei , are not present in OG1RF , nor are iolH and iolK present in the C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:casei', 'S-Organism:OG1RF']\",\n",
            "  \"response\": \"['B-Organism:casei', 'I-Organism:strain', 'E-Organism:5628']\"\n",
            "}\n",
            "783/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 782,\n",
            "  \"input\": \"Do NER on this sentence\\nreported that L\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "784/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 783,\n",
            "  \"input\": \"Do NER on this sentence\\nTo survey E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "785/1344 (0.77 seconds):\n",
            "{\n",
            "  \"id\": 784,\n",
            "  \"input\": \"Do NER on this sentence\\nThis region between EF2239 and EF2352 ( 111 Kb ) is also missing in the Symbioflor 1 strain ( referred to as gap 2 ) [ 5 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Symbioflor']\",\n",
            "  \"response\": \"this sentence\\nThis region between EF2239 and EF2352 ( 111 Kb ) is also missing in the Symbioflor 1 strain ( referred to as gap 2 ) [ 5 ]\"\n",
            "}\n",
            "786/1344 (1.11 seconds):\n",
            "{\n",
            "  \"id\": 785,\n",
            "  \"input\": \"Do NER on this sentence\\nThis fragment appears to be a patchwork composed of hypothetical genes , homologues of Tn916 - associated genes and homologues of genes from other Gram - positive organisms , including Listeria , E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:Listeria', 'E-Organism:sp']\"\n",
            "}\n",
            "787/1344 (0.28 seconds):\n",
            "{\n",
            "  \"id\": 786,\n",
            "  \"input\": \"Do NER on this sentence\\nIt is interesting to note that this region contains : a putative adhesin protein gene ( OG1RF _ 0039 ) at one end of the fragment ; homologues of 14 Tn916 - associated genes ( Tn916 _ 2 to Tn916 _ 12 , Tn916 _ 18 and Tn916 _ 19 , with an average of 70 % similarity ) ; and a gene encoding a putative integrase ( OG1RF _ 0088 ) at the other end - these three features are also present in Tn5386 in E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nIt is interesting to note that this region contains : a putative adhesin protein gene ( OG1RF _ 0039 ) at one end of the fragment ; homologues of 14 Tn916 - associated genes ( Tn916 _ 2 to Tn916 _ 12 , Tn916 _ 18 and Tn916 _ 19 , with an average of 70 % similarity ) ; and a gene encoding a putative integrase ( OG1RF _ 0088 ) at the other end - these three features are also present in Tn5386 in E\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "788/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 787,\n",
            "  \"input\": \"Do NER on this sentence\\nfaecium D344R [ 46 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:faecium', 'E-Organism:D344R']\",\n",
            "  \"response\": \"this sentence\\nfaecium D344R [ 46 ]\"\n",
            "}\n",
            "789/1344 (1.06 seconds):\n",
            "{\n",
            "  \"id\": 788,\n",
            "  \"input\": \"Do NER on this sentence\\nThis operon in OG1RF is similar to a nine - gene operon described in Streptococcus mutans [ 47 ] , as shown in Figure 6\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:OG1RF', 'B-Organism:Streptococcus', 'E-Organism:mutans']\",\n",
            "  \"response\": \"this sentence\\nThis operon in OG1RF is similar to a nine - gene operon described in Streptococcus mutans [ 47 ]\"\n",
            "}\n",
            "790/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 789,\n",
            "  \"input\": \"Do NER on this sentence\\nIn S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "791/1344 (0.95 seconds):\n",
            "{\n",
            "  \"id\": 790,\n",
            "  \"input\": \"Do NER on this sentence\\nNatural competence has not been reported for E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "792/1344 (1.52 seconds):\n",
            "{\n",
            "  \"id\": 791,\n",
            "  \"input\": \"Do NER on this sentence\\nIn Streptococcus pneumoniae [ 50 ] , the competence operon is tightly regulated by a quorum sensing two - component system ( ComDE ) and a competence - stimulating peptide ( CSP ; encoded by comC )\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Streptococcus', 'E-Organism:pneumoniae']\",\n",
            "  \"response\": \"this sentence\\nIn Streptococcus pneumoniae [ 50 ]\"\n",
            "}\n",
            "793/1344 (1.86 seconds):\n",
            "{\n",
            "  \"id\": 792,\n",
            "  \"input\": \"Do NER on this sentence\\nAlthough they are potential elements of a secretion apparatus , these two proteins do not share any homology at the protein level with the competence secretion apparatus ComAB of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:Escherichia', 'E-Organism:coli', 'B-Organism:Escherichia', 'E-Organism:coli']\"\n",
            "}\n",
            "794/1344 (1.25 seconds):\n",
            "{\n",
            "  \"id\": 793,\n",
            "  \"input\": \"Do NER on this sentence\\npneumoniae [ 51 ] nor CslAB from S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:pneumoniae']\",\n",
            "  \"response\": \"this sentence\\npneumoniae [ 51 ]\"\n",
            "}\n",
            "795/1344 (0.82 seconds):\n",
            "{\n",
            "  \"id\": 794,\n",
            "  \"input\": \"Do NER on this sentence\\nThe absence of these elements appears also to be a characteristic of the commensal strain Symbioflor 1 , although because the genome was not completely finished , the possibility remains that some of these regions were not sequenced\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Symbioflor']\",\n",
            "  \"response\": \"['B-Organism:Symbioflor']\"\n",
            "}\n",
            "796/1344 (1.15 seconds):\n",
            "{\n",
            "  \"id\": 795,\n",
            "  \"input\": \"Do NER on this sentence\\nMutations in this region have been associated with fusidic acid resistance in Staphylococcus aureus [ 55 , 56 ] , and thus the C1368A mutation is likely the cause of the fusidic acid resistance phenotype in OG1RF\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Staphylococcus', 'E-Organism:aureus', 'S-Organism:OG1RF']\",\n",
            "  \"response\": \"this sentence\\nMutations in this region have been associated with fusidic acid resistance in Staphylococcus aureus [ 55 , 56 ]\"\n",
            "}\n",
            "797/1344 (0.95 seconds):\n",
            "{\n",
            "  \"id\": 796,\n",
            "  \"input\": \"Do NER on this sentence\\nThese results , together with the previous results in C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "798/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 797,\n",
            "  \"input\": \"Do NER on this sentence\\nConclusion E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "799/1344 (1.80 seconds):\n",
            "{\n",
            "  \"id\": 798,\n",
            "  \"input\": \"Do NER on this sentence\\nfaecalis OG1RF carries a number of unique loci compared to V583\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:faecalis', 'E-Organism:OG1RF', 'S-Organism:V583']\",\n",
            "  \"response\": \"['I-Organism:faecalis', 'I-Organism:OG1', 'I-Organism:like', 'E-Organism:strain']\"\n",
            "}\n",
            "800/1344 (1.36 seconds):\n",
            "{\n",
            "  \"id\": 799,\n",
            "  \"input\": \"Do NER on this sentence\\nThe acquisition of MGE is believed to be an important mechanism by which the species E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "801/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 800,\n",
            "  \"input\": \"Do NER on this sentence\\nIt has been proposed that the ability of E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "802/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 801,\n",
            "  \"input\": \"Do NER on this sentence\\n[ 36 ] demonstrate that these factors may be present in harmless strains while absent in clinical isolates , indicating that E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\n[ 36 ]\"\n",
            "}\n",
            "803/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 802,\n",
            "  \"input\": \"Do NER on this sentence\\nIt is also important to remember that E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "804/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 803,\n",
            "  \"input\": \"Do NER on this sentence\\nSequencing of more E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "805/1344 (1.40 seconds):\n",
            "{\n",
            "  \"id\": 804,\n",
            "  \"input\": \"Do NER on this sentence\\nA Naturally Occurring Mutation in ropB Suppresses SpeB Expression and Reduces M1T1 Group A Streptococcal Systemic Virulence Epidemiological studies of group A streptococcus ( GAS ) have noted an inverse relationship between SpeB expression and invasive disease\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:M1T1', 'B-Organism:group', 'E-Organism:streptococcus', 'S-Organism:GAS']\",\n",
            "  \"response\": \"['B-Organism:Group', 'E-Organism:streptococcus', 'S-Organism:human']\"\n",
            "}\n",
            "806/1344 (1.26 seconds):\n",
            "{\n",
            "  \"id\": 805,\n",
            "  \"input\": \"Do NER on this sentence\\nIntroduction Streptococcus pyogenes ( group A streptococcus ; GAS ) is a Gram - positive , human - specific pathogen responsible for over 500 , 000 deaths each year [ 1 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Streptococcus', 'E-Organism:pyogenes', 'B-Organism:group', 'E-Organism:streptococcus', 'S-Organism:GAS', 'S-Organism:human']\",\n",
            "  \"response\": \"this sentence\\nIntroduction Streptococcus pyogenes ( group A streptococcus ; GAS ) is a Gram - positive , human - specific pathogen responsible for over 500 , 000 deaths each year [ 1 ]\"\n",
            "}\n",
            "807/1344 (1.02 seconds):\n",
            "{\n",
            "  \"id\": 806,\n",
            "  \"input\": \"Do NER on this sentence\\nWhile the M1T1 GAS has become the most common cause of streptococcal pharyngitis , this clone is also overrepresented in cases of severe invasive disease [ 6 ] , [ 7 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:M1T1', 'E-Organism:GAS']\",\n",
            "  \"response\": \"this sentence\\nWhile the M1T1 GAS has become the most common cause of streptococcal pharyngitis , this clone is also overrepresented in cases of severe invasive disease [ 6 ]\"\n",
            "}\n",
            "808/1344 (1.92 seconds):\n",
            "{\n",
            "  \"id\": 807,\n",
            "  \"input\": \"Do NER on this sentence\\nA study utilizing a zebrafish intramuscular infection model with serotype M5 GAS showed that inactivation of RopB resulted in decreased virulence [ 30 ] , whereas a study utilizing a murine intraperitoneal infection model with serotype M49 GAS showed that inactivation of RopB resulted in increased virulence [ 27 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:zebrafish', 'B-Organism:M5', 'E-Organism:GAS', 'S-Organism:murine', 'B-Organism:M49', 'E-Organism:GAS']\",\n",
            "  \"response\": \"this sentence\\nA study utilizing a zebrafish intramuscular infection model with serotype M5 GAS showed that inactivation of RopB resulted in decreased virulence [ 30 ]\"\n",
            "}\n",
            "809/1344 (1.54 seconds):\n",
            "{\n",
            "  \"id\": 808,\n",
            "  \"input\": \"Do NER on this sentence\\nIt is in the light of the current uncertainty surrounding RopB and its role in virulence that we sought to investigate the role of this transcriptional regulator in the serotype M1T1 GAS background that is the leading agent of severe human infection\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:M1T1', 'E-Organism:GAS', 'S-Organism:human']\",\n",
            "  \"response\": \"['B-Organism:invasive', 'E-Organism:GAS', 'S-Organism:GAS']\"\n",
            "}\n",
            "810/1344 (1.46 seconds):\n",
            "{\n",
            "  \"id\": 809,\n",
            "  \"input\": \"Do NER on this sentence\\nResults Sequence analysis of clinical isolate 5628 reveals intact speB and covR / S but mutation in ropB M1T1 GAS clinical isolate 5628 was found on screening to lack SpeB activity by azocasein assay\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:5628', 'B-Organism:M1T1', 'E-Organism:GAS', 'S-Organism:5628']\",\n",
            "  \"response\": \"['B-Organism:invasive', 'E-Organism:GAS', 'S-Organism:human']\"\n",
            "}\n",
            "811/1344 (1.21 seconds):\n",
            "{\n",
            "  \"id\": 810,\n",
            "  \"input\": \"Do NER on this sentence\\nRepair of the 5628 ropB allele restores SpeB expression and activity Western blot detected SpeB in overnight culture supernatants of 5628R + , containing the intact ropB allele , but not in the clinical isolate 5628 nor the isogenic mutant 5628RDeltaSpeB , containing the intact ropB allele but lacking speB ( Fig\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:5628', 'B-Organism:5628R', 'S-Organism:5628']\",\n",
            "  \"response\": \"['B-Organism:5628', 'E-Organism:5628R']\"\n",
            "}\n",
            "812/1344 (1.86 seconds):\n",
            "{\n",
            "  \"id\": 811,\n",
            "  \"input\": \"Do NER on this sentence\\nRopB mutation does not affect bacterial growth , hemolysis , DNase activity or resistance to neutrophil killing GAS strain 5628 and its isogenic mutants 5628R + and 5628RDeltaSpeB were grown in THB , and the OD600 was measured over time\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:GAS', 'I-Organism:strain', 'E-Organism:5628', 'B-Organism:5628R']\",\n",
            "  \"response\": \"['S-Organism:5628', 'B-Organism:5628R', 'S-Organism:5628R']\"\n",
            "}\n",
            "813/1344 (1.17 seconds):\n",
            "{\n",
            "  \"id\": 812,\n",
            "  \"input\": \"Do NER on this sentence\\nSpeB - positive bacteria revert to a SpeB - negative phenotype on subcutaneous infection C57BL6 / J mice were subcutaneously infected with the SpeB - positive strain 5628R +\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:C57BL6', 'E-Organism:mice', 'B-Organism:5628R']\",\n",
            "  \"response\": \"['S-Organism:5628', 'B-Organism:5628R']\"\n",
            "}\n",
            "814/1344 (1.65 seconds):\n",
            "{\n",
            "  \"id\": 813,\n",
            "  \"input\": \"Do NER on this sentence\\nSpeB is the principal target of RopB regulation in vivo Microarray analysis was performed on in vivo - derived RNA from the well characterized GAS strain 5448 and its derivative 5448R - , containing the mutated ropB allele from 5628\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:GAS', 'I-Organism:strain', 'E-Organism:5448', 'B-Organism:5448R', 'S-Organism:5628']\",\n",
            "  \"response\": \"['B-Organism:5448', 'E-Organism:5448']\"\n",
            "}\n",
            "815/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 814,\n",
            "  \"input\": \"Do NER on this sentence\\nThe most strongly down - regulated genes in 5448R - are members of the SpeB operon , as would be expected from the lack of SpeB expression and activity in 5628 ( Fig\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:5448R', 'S-Organism:5628']\",\n",
            "  \"response\": \"['B-Organism:5448R', 'B-Organism:5628']\"\n",
            "}\n",
            "816/1344 (1.17 seconds):\n",
            "{\n",
            "  \"id\": 815,\n",
            "  \"input\": \"Do NER on this sentence\\nApart from the gene encoding the superantigen SmeZ , there was an absence of virulence - related genes found to be strongly up - regulated in the strain 5448R - , which expressed the truncated RopB\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:5448R']\",\n",
            "  \"response\": \"['B-Organism:GAS', 'E-Organism:5448R']\"\n",
            "}\n",
            "817/1344 (1.57 seconds):\n",
            "{\n",
            "  \"id\": 816,\n",
            "  \"input\": \"Do NER on this sentence\\nRopB is required for virulence in systemic infection To determine the effect of ropB mutation on virulence , C57BL6 / J mice were subjected to intraperitoneal infection with mid - logarithmic phase bacteria\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:C57BL6', 'E-Organism:mice']\",\n",
            "  \"response\": \"['B-Organism:invasive', 'E-Organism:GAS', 'S-Organism:mouse']\"\n",
            "}\n",
            "818/1344 (7.43 seconds):\n",
            "{\n",
            "  \"id\": 817,\n",
            "  \"input\": \"Do NER on this sentence\\nGAS strain 5628R + expressing a full length RopB showed increased virulence compared to the strain 5628 with a truncated RopB ( Figure 4A )\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:GAS', 'I-Organism:strain', 'I-Organism:5628R', 'S-Organism:5628']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "819/1344 (1.40 seconds):\n",
            "{\n",
            "  \"id\": 818,\n",
            "  \"input\": \"Do NER on this sentence\\nThe well characterized strain 5448 and its isogenic mutant 5448R - , containing the ropB allele from 5628 , were further included to examine the effect of this allelic variation in a well - characterized virulent strain of GAS\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:5448', 'B-Organism:5448R', 'S-Organism:5628', 'S-Organism:GAS']\",\n",
            "  \"response\": \"['B-Organism', 'S-Organism:GAS', 'S-Organism:GAS']\"\n",
            "}\n",
            "820/1344 (1.07 seconds):\n",
            "{\n",
            "  \"id\": 819,\n",
            "  \"input\": \"Do NER on this sentence\\nThe two strains containing a truncated RopB ( 5628 and 5448R - ) demonstrated similarly reduced virulence compared to the two strains containing a full - length RopB ( 5628R + and 5448 ) ( Fig\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:5628', 'B-Organism:5448R', 'B-Organism:5628R', 'S-Organism:5448']\",\n",
            "  \"response\": \"['B-Organism:5448', 'E-Organism:RopB']\"\n",
            "}\n",
            "821/1344 (1.85 seconds):\n",
            "{\n",
            "  \"id\": 820,\n",
            "  \"input\": \"Do NER on this sentence\\nWe have shown that in M1T1 GAS , when the covR / S locus is intact and DNase Sda1 activity is unaffected , ropB point mutation results in reduced virulence in vivo , despite inactivation of SpeB activity\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:M1T1', 'E-Organism:GAS']\",\n",
            "  \"response\": \"['B-Organism:invasive', 'E-Organism:GAS', 'B-Organism:M1T1', 'E-Organism:GAS']\"\n",
            "}\n",
            "822/1344 (1.01 seconds):\n",
            "{\n",
            "  \"id\": 821,\n",
            "  \"input\": \"Do NER on this sentence\\nThe M1T1 strain 5628 contains a point mutation in the ropB gene that results in a truncation of the last 110 amino acids of the 280 amino acid protein\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:M1T1', 'I-Organism:strain', 'E-Organism:5628']\",\n",
            "  \"response\": \"['B-Organism:M1T1', 'E-Organism:GAS']\"\n",
            "}\n",
            "823/1344 (1.01 seconds):\n",
            "{\n",
            "  \"id\": 822,\n",
            "  \"input\": \"Do NER on this sentence\\nRestoration of full - length RopB in 5628R + resulted in gaining SpeB expression , but did not result in changes in hemolysis , DNase activity or resistance to killing by neutrophils\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:5628R']\",\n",
            "  \"response\": \"['B-Organism:M1T1', 'E-Organism:GAS']\"\n",
            "}\n",
            "824/1344 (6.66 seconds):\n",
            "{\n",
            "  \"id\": 823,\n",
            "  \"input\": \"Do NER on this sentence\\nThe clinical isolate 5628 expressing a truncated RopB , as well as the allelic exchange mutant 5448R - also expressing the truncated RopB from 5628 , showed reduced virulence compared to strains containing a full - length RopB\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:5628', 'B-Organism:5448R', 'S-Organism:5628']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "825/1344 (1.80 seconds):\n",
            "{\n",
            "  \"id\": 824,\n",
            "  \"input\": \"Do NER on this sentence\\nIn M1T1 GAS , an intact RopB and efficient SpeB production are necessary for systemic infection\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:M1T1', 'E-Organism:GAS']\",\n",
            "  \"response\": \"['B-Organism:invasive', 'E-Organism:GAS', 'B-Organism:M1T1', 'E-Organism:GAS']\"\n",
            "}\n",
            "826/1344 (1.71 seconds):\n",
            "{\n",
            "  \"id\": 825,\n",
            "  \"input\": \"Do NER on this sentence\\nFor example an Deltahfq mutant derivative in the pathogen Salmonella enterica serovars Typhimurium ( Salmonella Typhimurium ) was found to attenuate virulence in mice and was impaired in the invasion of non - phagocytic cells [ 20 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'I-Organism:enterica', 'I-Organism:serovars', 'E-Organism:Typhimurium', 'B-Organism:Salmonella', 'E-Organism:Typhimurium', 'S-Organism:mice']\",\n",
            "  \"response\": \"this sentence\\nFor example an Deltahfq mutant derivative in the pathogen Salmonella enterica serovars Typhimurium ( Salmonella Typhimurium ) was found to attenuate virulence in mice and was impaired in the invasion of non - phagocytic cells [ 20 ]\"\n",
            "}\n",
            "827/1344 (2.01 seconds):\n",
            "{\n",
            "  \"id\": 826,\n",
            "  \"input\": \"Do NER on this sentence\\nHfq was also reported to be essential for Vibrio cholerae virulence , with sRNAs in conjunction with Hfq suggested as likely critical regulators of Vibrio cholerae pathogenicity [ 21 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Vibrio', 'E-Organism:cholerae', 'B-Organism:Vibrio', 'E-Organism:cholerae']\",\n",
            "  \"response\": \"this sentence\\nHfq was also reported to be essential for Vibrio cholerae virulence , with sRNAs in conjunction with Hfq suggested as likely critical regulators of Vibrio cholerae pathogenicity [ 21 ]\"\n",
            "}\n",
            "828/1344 (1.31 seconds):\n",
            "{\n",
            "  \"id\": 827,\n",
            "  \"input\": \"Do NER on this sentence\\nIn another study , a smpB mutant of Salmonella Typhimurium exhibited defects in intra - macrophage survival , a model of Salmonella pathogenesis [ 23 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium', 'S-Organism:Salmonella']\",\n",
            "  \"response\": \"this sentence\\nIn another study , a smpB mutant of Salmonella Typhimurium exhibited defects in intra - macrophage survival , a model of Salmonella pathogenesis [ 23 ]\"\n",
            "}\n",
            "829/1344 (2.21 seconds):\n",
            "{\n",
            "  \"id\": 828,\n",
            "  \"input\": \"Do NER on this sentence\\nMore recently mutation of tmRNA was shown to be responsible for a marked reduction in Salmonella Typhimurium virulence in BALB / c mice [ 24 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium', 'B-Organism:BALB', 'E-Organism:mice']\",\n",
            "  \"response\": \"this sentence\\nMore recently mutation of tmRNA was shown to be responsible for a marked reduction in Salmonella Typhimurium virulence in BALB / c mice [ 24 ]\"\n",
            "}\n",
            "830/1344 (1.27 seconds):\n",
            "{\n",
            "  \"id\": 829,\n",
            "  \"input\": \"Do NER on this sentence\\nIn Yersinia pseudotuberculosis mutations of smpB and tmRNA generated a bacterium that exhibited defects in survival and replication in macrophages and was avirulent in mice [ 25 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Yersinia', 'E-Organism:pseudotuberculosis', 'S-Organism:mice']\",\n",
            "  \"response\": \"this sentence\\nIn Yersinia pseudotuberculosis mutations of smpB and tmRNA generated a bacterium that exhibited defects in survival and replication in macrophages and was avirulent in mice [ 25 ]\"\n",
            "}\n",
            "831/1344 (1.25 seconds):\n",
            "{\n",
            "  \"id\": 830,\n",
            "  \"input\": \"Do NER on this sentence\\nResults and Discussion Construction and growth characteristics of mutant and wildtype Salmonella Typhimurium strains Hfq and SmpB are essential for virulence in a variety of pathogenic bacteria including Salmonella Typhimurium [ 20 ] , [ 21 ] , [ 22 ] , [ 25 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium', 'B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"this sentence\\nResults and Discussion Construction and growth characteristics of mutant and wildtype Salmonella Typhimurium strains Hfq and SmpB are essential for virulence in a variety of pathogenic bacteria including Salmonella Typhimurium [ 20 ]\"\n",
            "}\n",
            "832/1344 (1.36 seconds):\n",
            "{\n",
            "  \"id\": 831,\n",
            "  \"input\": \"Do NER on this sentence\\nTo address the role of Hfq and SmpB in Salmonella Typhimurium pathogenesis and protein synthesis we first constructed non - polar deletion mutations of the hfq and smpB genes using a modified pKD13 derivative as the template\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "833/1344 (1.82 seconds):\n",
            "{\n",
            "  \"id\": 832,\n",
            "  \"input\": \"Do NER on this sentence\\nDeltahfq and DeltasmpB Salmonella Typhimurium mutant strains are avirulent To directly assess the role of Hfq and SmpB in Salmonella Typhimurium pathogenesis , BALB / c mice were infected intraperitoneally with 200 cfu of either Deltahfq mutant strain , DeltasmpB mutant strain or wildtype strain and monitored for 21 days ( Figure 1 )\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium', 'B-Organism:BALB', 'E-Organism:mice']\",\n",
            "  \"response\": \"['S-Organism:Salmonella', 'S-Organism:Typhimurium']\"\n",
            "}\n",
            "834/1344 (1.25 seconds):\n",
            "{\n",
            "  \"id\": 833,\n",
            "  \"input\": \"Do NER on this sentence\\nGlobal effects of Hfq and SmpB mutation on Salmonella Typhimurium chromosome LC - MS - based proteomics was used to obtain a general overview of the effects of Hfq and SmpB on global protein translation in Salmonella\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium', 'S-Organism:Salmonella']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "835/1344 (1.21 seconds):\n",
            "{\n",
            "  \"id\": 834,\n",
            "  \"input\": \"Do NER on this sentence\\nAcross all experimental datasets we observed a total of 1621 Salmonella Typhimurium proteins ( Table S1 ) , corresponding to approximately36 % coverage of predicted protein coding regions\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "836/1344 (1.21 seconds):\n",
            "{\n",
            "  \"id\": 835,\n",
            "  \"input\": \"Do NER on this sentence\\nAcross each of the four growth conditions examined our analysis revealed that Hfq had a far more significant effect on the Salmonella Typhimurium proteome than SmpB\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "837/1344 (1.35 seconds):\n",
            "{\n",
            "  \"id\": 836,\n",
            "  \"input\": \"Do NER on this sentence\\nConsidering the reported pleitropic nature of deletion of hfq on the biology of Salmonella Typhimurium it is likely that a proportion of the changes in protein abundances maybe a non - specific and / or secondary consequence of hfq deletion\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "838/1344 (1.41 seconds):\n",
            "{\n",
            "  \"id\": 837,\n",
            "  \"input\": \"Do NER on this sentence\\nExamination of transcript levels using microarray analysis ( > / = 2 - fold change ) revealed a more modest effect on Salmonella transcription , with approximately11 % ( 492 genes ) and approximately8 % ( 370 genes ) of the Salmonella Typhimurium genome ( 4550 genes ; [ 31 ] ) under control of Hfq and SmpB , respectively ( Table S4 )\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Salmonella', 'B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"this sentence\\nExamination of transcript levels using microarray analysis ( > / = 2 - fold change ) revealed a more modest effect on Salmonella transcription , with approximately11 % ( 492 genes ) and approximately8 % ( 370 genes ) of the Salmonella Typhimurium genome ( 4550 genes ; [ 31 ]\"\n",
            "}\n",
            "839/1344 (0.11 seconds):\n",
            "{\n",
            "  \"id\": 838,\n",
            "  \"input\": \"Do NER on this sentence\\nThe result for hfq transcriptional analysis is in general agreement with previous studies for Salmonella [ 10 ] and Pseudomonas aeruginosa [ 33 ] where approximately20 % and approximately15 % of all genes , respectively , were regulated by Hfq , but slightly higher than the number of Hfq - dependent genes reported for E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Salmonella', 'B-Organism:Pseudomonas', 'E-Organism:aeruginosa']\",\n",
            "  \"response\": \"this sentence\\nThe result for hfq transcriptional analysis is in general agreement with previous studies for Salmonella [ 10 ]\"\n",
            "}\n",
            "840/1344 (1.46 seconds):\n",
            "{\n",
            "  \"id\": 839,\n",
            "  \"input\": \"Do NER on this sentence\\ncoli [ 27 ] and Vibrio cholerae [ 21 ] , approximately6 % each\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:coli', 'B-Organism:Vibrio', 'E-Organism:cholerae']\",\n",
            "  \"response\": \"this sentence\\ncoli [ 27 ]\"\n",
            "}\n",
            "841/1344 (1.31 seconds):\n",
            "{\n",
            "  \"id\": 840,\n",
            "  \"input\": \"Do NER on this sentence\\nWe observed changes in expression for approximately50 % of the observed Salmonella Typhimurium proteome based on an ANOVA analysis ( p < 0\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "842/1344 (1.21 seconds):\n",
            "{\n",
            "  \"id\": 841,\n",
            "  \"input\": \"Do NER on this sentence\\nCollectively , these results point to a central role for Hfq in regulating stress response in Salmonella Typhimurium\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "843/1344 (1.31 seconds):\n",
            "{\n",
            "  \"id\": 842,\n",
            "  \"input\": \"Do NER on this sentence\\nFor example rpoE has been shown to be required for the virulence of Salmonella Typhimurium [ 35 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"this sentence\\nFor example rpoE has been shown to be required for the virulence of Salmonella Typhimurium [ 35 ]\"\n",
            "}\n",
            "844/1344 (1.61 seconds):\n",
            "{\n",
            "  \"id\": 843,\n",
            "  \"input\": \"Do NER on this sentence\\nPropanediol utilization operon The cob operon is required for the de novo biosynthesis of vitamin B12 in Salmonella Typhimurium\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "845/1344 (1.21 seconds):\n",
            "{\n",
            "  \"id\": 844,\n",
            "  \"input\": \"Do NER on this sentence\\nThis allows Salmonella Typhimurium to grow using propanediol as a sole carbon source and because it is required for growth in macrophages this may be the preferred intracellular metabolite [ 47 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"this sentence\\nThis allows Salmonella Typhimurium to grow using propanediol as a sole carbon source and because it is required for growth in macrophages this may be the preferred intracellular metabolite [ 47 ]\"\n",
            "}\n",
            "846/1344 (1.85 seconds):\n",
            "{\n",
            "  \"id\": 845,\n",
            "  \"input\": \"Do NER on this sentence\\nHowever , results from our global proteomics analysis indicate that SmpB directly or indirectly modulates the expression of at least 4 % of the entire Salmonella Typhimurium proteome determined from ANOVA analysis ( p < 0\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "847/1344 (1.72 seconds):\n",
            "{\n",
            "  \"id\": 846,\n",
            "  \"input\": \"Do NER on this sentence\\nThe majority of genes encoding these proteins showed no changes in mRNA level thus it is interesting to speculate an additional role for SmpB as a general post - transcriptional / translational regulator in Salmonella Typhimurium\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "848/1344 (1.31 seconds):\n",
            "{\n",
            "  \"id\": 847,\n",
            "  \"input\": \"Do NER on this sentence\\nThe RpoS regulon in Salmonella Typhimurium has been initially characterized , identifying 13 conserved RpoS - regulated genes [ 48 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"this sentence\\nThe RpoS regulon in Salmonella Typhimurium has been initially characterized , identifying 13 conserved RpoS - regulated genes [ 48 ]\"\n",
            "}\n",
            "849/1344 (1.31 seconds):\n",
            "{\n",
            "  \"id\": 848,\n",
            "  \"input\": \"Do NER on this sentence\\nProteins involved in motility in Salmonella Typhimurium also showed down - regulation during growth in LB media\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "850/1344 (1.35 seconds):\n",
            "{\n",
            "  \"id\": 849,\n",
            "  \"input\": \"Do NER on this sentence\\nThis subset included a putative serine / threonine protein kinase , a putative intracellular proteinase YhbO , SrfJ , a known Salmonella Typhimurium virulence protein , and the stress protein HtpG\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "851/1344 (0.11 seconds):\n",
            "{\n",
            "  \"id\": 850,\n",
            "  \"input\": \"Do NER on this sentence\\nComparative in vivo gene expression of the closely related bacteria Photorhabdus temperata and Xenorhabdus koppenhoeferi upon infection of the same insect host , Rhizotrogus majalis Background Photorhabdus and Xenorhabdus are Gram - negative , phylogenetically related , enterobacteria , forming mutualism with the entomopathogenic nematodes Heterorhabditis and Steinernema , respectively\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Photorhabdus', 'E-Organism:temperata', 'B-Organism:Xenorhabdus', 'E-Organism:koppenhoeferi', 'B-Organism:Rhizotrogus', 'E-Organism:majalis', 'S-Organism:Photorhabdus', 'S-Organism:Xenorhabdus', 'S-Organism:enterobacteria', 'S-Organism:Heterorhabditis', 'S-Organism:Steinernema']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "852/1344 (1.52 seconds):\n",
            "{\n",
            "  \"id\": 851,\n",
            "  \"input\": \"Do NER on this sentence\\nWe studied in vivo gene expression in Photorhabdus temperata and Xenorhabdus koppenhoeferi upon infection of the white grub Rhizotrogus majalis using selective capture of transcribed sequences technique\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Photorhabdus', 'E-Organism:temperata', 'B-Organism:Xenorhabdus', 'E-Organism:koppenhoeferi', 'B-Organism:Rhizotrogus', 'E-Organism:majalis']\",\n",
            "  \"response\": \"['B-Organism:Rhizotrogus', 'E-Organism:majalis']\"\n",
            "}\n",
            "853/1344 (0.46 seconds):\n",
            "{\n",
            "  \"id\": 852,\n",
            "  \"input\": \"Do NER on this sentence\\nResults A total of 40 genes in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "854/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 853,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and 39 in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "855/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 854,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi were found to be upregulated in R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "856/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 855,\n",
            "  \"input\": \"Do NER on this sentence\\nIn P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "857/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 856,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata lysR gene encoding transcriptional activator was induced , while genes yijC and rseA encoding transcriptional repressors were induced in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "858/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 857,\n",
            "  \"input\": \"Do NER on this sentence\\nLipopolysaccharide synthesis gene lpsE was induced in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "859/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 858,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi but not in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "860/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 859,\n",
            "  \"input\": \"Do NER on this sentence\\nGenes involved in TCA cycle were induced in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "861/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 860,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata whereas those involved in glyoxylate pathway were induced in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "862/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 861,\n",
            "  \"input\": \"Do NER on this sentence\\nPhotorhabdus temperata displayed upregulation of genes encoding siderophore - dependent iron uptake system , but X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Photorhabdus', 'E-Organism:temperata']\",\n",
            "  \"response\": \"['B-Organism:Photorhabdus', 'E-Organism:temperata']\"\n",
            "}\n",
            "863/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 862,\n",
            "  \"input\": \"Do NER on this sentence\\nPhotorhabdus temperata induced genes for amino acid acquisition but X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Photorhabdus', 'E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "864/1344 (0.46 seconds):\n",
            "{\n",
            "  \"id\": 863,\n",
            "  \"input\": \"Do NER on this sentence\\nConclusion This study identifies set of genes induced in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "865/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 864,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "866/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 865,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi upon infection of R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "867/1344 (1.56 seconds):\n",
            "{\n",
            "  \"id\": 866,\n",
            "  \"input\": \"Do NER on this sentence\\nThe published genome sequence of Photorhabdus luminescens TT01 strain indicates that virulence genes are encoded within a number of pathogenicity islands located on the bacterial chromosome [ 4 , 5 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Photorhabdus', 'I-Organism:luminescens', 'E-Organism:TT01']\",\n",
            "  \"response\": \"this sentence\\nThe published genome sequence of Photorhabdus luminescens TT01 strain indicates that virulence genes are encoded within a number of pathogenicity islands located on the bacterial chromosome [ 4 , 5 ]\"\n",
            "}\n",
            "868/1344 (1.31 seconds):\n",
            "{\n",
            "  \"id\": 867,\n",
            "  \"input\": \"Do NER on this sentence\\nFor example , in Photorhabdus , mutational inactivation of phoP gene results in increased sensitivity to insect immune response and decreased virulence towards insects [ 6 , 7 ] , while in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Photorhabdus']\",\n",
            "  \"response\": \"this sentence\\nFor example , in Photorhabdus , mutational inactivation of phoP gene results in increased sensitivity to insect immune response and decreased virulence towards insects [ 6 , 7 ]\"\n",
            "}\n",
            "869/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 868,\n",
            "  \"input\": \"Do NER on this sentence\\nP\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "870/1344 (0.80 seconds):\n",
            "{\n",
            "  \"id\": 869,\n",
            "  \"input\": \"Do NER on this sentence\\nluminescens produces a signaling molecule AI - 2 to resist reactive oxygen species [ 8 ] and phenylpropanoid chemical ST to inhibit the activity of antimicrobial enzyme PO and formation of melanotic nodules [ 9 ] , but the strategy used by X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:luminescens']\",\n",
            "  \"response\": \"this sentence\\nluminescens produces a signaling molecule AI - 2 to resist reactive oxygen species [ 8 ]\"\n",
            "}\n",
            "871/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 870,\n",
            "  \"input\": \"Do NER on this sentence\\nIn addition , P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "872/1344 (0.07 seconds):\n",
            "{\n",
            "  \"id\": 871,\n",
            "  \"input\": \"Do NER on this sentence\\nHowever , the genomes of Xenorhabdus bovienii and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Xenorhabdus', 'E-Organism:bovienii']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "873/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 872,\n",
            "  \"input\": \"Do NER on this sentence\\n[ 16 ] identified 29 promoters induced by P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\n[ 16 ]\"\n",
            "}\n",
            "874/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 873,\n",
            "  \"input\": \"Do NER on this sentence\\nluminescens in the presence of isolated Galleria mellonella homogenate using a differential fluorescence induction approach\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:luminescens', 'B-Organism:Galleria', 'E-Organism:mellonella']\",\n",
            "  \"response\": \"this sentence\\nluminescens in the presence of isolated Galleria mellonella homogenate using a differential fluorescence induction approach\\nReply in key value pair as Tag and Entity. Response of Y\\\"\\n['E-Organism:luminescens']\"\n",
            "}\n",
            "875/1344 (2.22 seconds):\n",
            "{\n",
            "  \"id\": 874,\n",
            "  \"input\": \"Do NER on this sentence\\nWe used selective capture of transcribed sequences ( SCOTS ) technique to study in vivo gene expression in Photorhabdus temperata ( associated with the nematode Heterorhabditis bacteriophora GPS11 ) and Xenorhabdus koppenhoeferi ( associated with the nematode Steinernema scarabaei AMK001 ) during infection of the same insect host Rhizotrogus majalis\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Photorhabdus', 'E-Organism:temperata', 'B-Organism:Heterorhabditis', 'I-Organism:bacteriophora', 'E-Organism:GPS11', 'B-Organism:Xenorhabdus', 'E-Organism:koppenhoeferi', 'B-Organism:Steinernema', 'I-Organism:scarabaei', 'E-Organism:AMK001', 'B-Organism:Rhizotrogus', 'E-Organism:majalis']\",\n",
            "  \"response\": \"['B-Organism:Photorhabdus', 'E-Organism:Xenorhabdus', 'B-Organism:Rhizotrogus', 'E-Organism:majalis']\"\n",
            "}\n",
            "876/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 875,\n",
            "  \"input\": \"Do NER on this sentence\\nThey were selected according to their distinct pathogenicity in insects of the family of Scarabaeidae - S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "877/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 876,\n",
            "  \"input\": \"Do NER on this sentence\\nscarabaei is more virulent to R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:scarabaei']\",\n",
            "  \"response\": \"['E-Organism:scarabaei']\"\n",
            "}\n",
            "878/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 877,\n",
            "  \"input\": \"Do NER on this sentence\\nmajalis than H\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:majalis']\",\n",
            "  \"response\": \"['E-Organism:majalis']\"\n",
            "}\n",
            "879/1344 (1.31 seconds):\n",
            "{\n",
            "  \"id\": 878,\n",
            "  \"input\": \"Do NER on this sentence\\nHowever , unlike their nematode partners , both bacterial species are highly virulent to Rhizotrogus majalis\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Rhizotrogus', 'E-Organism:majalis']\",\n",
            "  \"response\": \"['B-Organism:Photorhabdus', 'E-Organism:luminescens']\"\n",
            "}\n",
            "880/1344 (1.35 seconds):\n",
            "{\n",
            "  \"id\": 879,\n",
            "  \"input\": \"Do NER on this sentence\\nIn addition , although X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "881/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 880,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi grows slower than P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "882/1344 (1.31 seconds):\n",
            "{\n",
            "  \"id\": 881,\n",
            "  \"input\": \"Do NER on this sentence\\nResults Isolation of bacterial transcripts induced in Rhizotrogus majalis To isolate bacterial genes induced upon infection , total RNA was isolated from live grubs at 24 h post injection with P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Rhizotrogus', 'E-Organism:majalis']\",\n",
            "  \"response\": \"['B-Organism:Photorhabdus', 'E-Organism:luminescens']\"\n",
            "}\n",
            "883/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 882,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata or X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "884/1344 (0.77 seconds):\n",
            "{\n",
            "  \"id\": 883,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi , as well as from 48 h log - phase bacterial cultures grown in Brain Heart Infusion ( BHI ) broth that were used to inject R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "885/1344 (1.20 seconds):\n",
            "{\n",
            "  \"id\": 884,\n",
            "  \"input\": \"Do NER on this sentence\\nOut of these , the clones ( 150 for P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "886/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 885,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and 140 for X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "887/1344 (1.71 seconds):\n",
            "{\n",
            "  \"id\": 886,\n",
            "  \"input\": \"Do NER on this sentence\\n1 ) were sequenced and analyzed using the non - redundant algorithms of BLAST in NCBI and database of Xenorhabdus Genomes\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Xenorhabdus', 'E-Organism:Genomes']\",\n",
            "  \"response\": \"['B-Organism:Xenorhabdus', 'E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "888/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 887,\n",
            "  \"input\": \"Do NER on this sentence\\nThese sequenced clones were therefore considered to represent reliable transcripts specifically expressed in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "889/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 888,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata or X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "890/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 889,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi after 24 h infection of R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "891/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 890,\n",
            "  \"input\": \"Do NER on this sentence\\nAnalysis of bacterial genes induced in Rhizotrogus majalis Most identified transcripts were either similar to sequences from P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Rhizotrogus', 'E-Organism:majalis']\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "892/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 891,\n",
            "  \"input\": \"Do NER on this sentence\\nluminescens or X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:luminescens']\",\n",
            "  \"response\": \"['E-Organism:luminescens']\"\n",
            "}\n",
            "893/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 892,\n",
            "  \"input\": \"Do NER on this sentence\\nTotally , we isolated 40 different sequences from P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "894/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 893,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata ( Table 1 ) and 39 from X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "895/1344 (0.81 seconds):\n",
            "{\n",
            "  \"id\": 894,\n",
            "  \"input\": \"Do NER on this sentence\\nSemi - quantitative reverse transcriptase PCR performed on 10 randomly selected identified genes ( five from P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "896/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 895,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata : cysK , fliA , lysR , pchC and pmt1 and five from X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "897/1344 (1.76 seconds):\n",
            "{\n",
            "  \"id\": 896,\n",
            "  \"input\": \"Do NER on this sentence\\nIdentification of Photorhabdus temperata or Xenorhabdus koppenhoeferi specific genes To identify genes that were distinctive in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Photorhabdus', 'E-Organism:temperata', 'B-Organism:Xenorhabdus', 'E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['B-Organism:Xenorhabdus', 'E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "898/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 897,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata or X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "899/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 898,\n",
            "  \"input\": \"Do NER on this sentence\\nTwenty nine of the 40 distinct P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "900/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 899,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata transcripts have similar sequences in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "901/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 900,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi genome , thus 11 were P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "902/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 901,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata specific ; 30 of the 39 distinct X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "903/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 902,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi transcripts share sequence similarity with P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "904/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 903,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata genome , thus 9 were specific to X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "905/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 904,\n",
            "  \"input\": \"Do NER on this sentence\\nAlthough these 59 transcripts ( 29 + 30 ) corresponding to genes present in the genome of both bacteria , 21 from P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "906/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 905,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and 22 from X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "907/1344 (0.07 seconds):\n",
            "{\n",
            "  \"id\": 906,\n",
            "  \"input\": \"Do NER on this sentence\\nFurther , the level of changes in the transcripts exclusively induced in one or the other bacterium during infection of R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "908/1344 (0.72 seconds):\n",
            "{\n",
            "  \"id\": 907,\n",
            "  \"input\": \"Do NER on this sentence\\nmajalis was determined by quantitative real - time PCR ( qRT - PCR ) on following representative genes : cysK , def , dnaB , lysR , pchC , pckA , phoP , ptst1 , sctL , selD , tolB and virB from P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:majalis']\",\n",
            "  \"response\": \"['E-Organism:majalis']\"\n",
            "}\n",
            "909/1344 (0.10 seconds):\n",
            "{\n",
            "  \"id\": 908,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata , and aceK , clpP , cobJ , dacC , dld , malF , res , rtxC , tilS , virH , xhlA , and xkst4 from X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "910/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 909,\n",
            "  \"input\": \"Do NER on this sentence\\nOther identified genes with multiple connections to different gene products were different between P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "911/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 910,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "912/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 911,\n",
            "  \"input\": \"Do NER on this sentence\\nFor example , regulation of genes yncB and purH in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "913/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 912,\n",
            "  \"input\": \"Do NER on this sentence\\nIn X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "914/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 913,\n",
            "  \"input\": \"Do NER on this sentence\\nIn total , 20 genes from P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "915/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 914,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and 18 from X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "916/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 915,\n",
            "  \"input\": \"Do NER on this sentence\\nFor example , P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "917/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 916,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata pckA and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "918/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 917,\n",
            "  \"input\": \"Do NER on this sentence\\n5 ) , apart from a relatively small number of common strategies used by P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "919/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 918,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "920/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 919,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi to cause infection of R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "921/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 920,\n",
            "  \"input\": \"Do NER on this sentence\\nGenes involved in the synthesis of outer membrane proteins ( skp in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "922/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 921,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and ompF in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "923/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 922,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi ) and flagella ( fliA in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "924/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 923,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and fliM in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "925/1344 (0.81 seconds):\n",
            "{\n",
            "  \"id\": 924,\n",
            "  \"input\": \"Do NER on this sentence\\nFlagellum - mediated motility may provide an advantage for infection by attachment to the host surface [ 19 , 20 ] , whereas it has been reported in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nFlagellum - mediated motility may provide an advantage for infection by attachment to the host surface [ 19 , 20 ]\"\n",
            "}\n",
            "926/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 925,\n",
            "  \"input\": \"Do NER on this sentence\\nGene dacC encoding a peptidoglycan synthesis enzyme D - alanyl - D - alanine carboxypeptidase was induced in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "927/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 926,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi but not in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "928/1344 (0.72 seconds):\n",
            "{\n",
            "  \"id\": 927,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata , implying that peptidoglycan undergoes structural alterations in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "929/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 928,\n",
            "  \"input\": \"Do NER on this sentence\\nAs a well - defined virulence factor playing a major role in the development of septicemia [ 22 ] , lipopolysaccharide biosynthesis gene lpsE was induced in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nAs a well - defined virulence factor playing a major role in the development of septicemia [ 22 ]\"\n",
            "}\n",
            "930/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 929,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi , but not in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "931/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 930,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata , which may explain in part why relatively slower growing X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "932/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 931,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi did not differ from P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "933/1344 (0.62 seconds):\n",
            "{\n",
            "  \"id\": 932,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata in virulence to R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "934/1344 (2.35 seconds):\n",
            "{\n",
            "  \"id\": 933,\n",
            "  \"input\": \"Do NER on this sentence\\nGenes involved in regulatory networks Facing stimuli , different regulatory genes were upregulated in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "935/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 934,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "936/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 935,\n",
            "  \"input\": \"Do NER on this sentence\\nphoP and cpxR genes identified from P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "937/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 936,\n",
            "  \"input\": \"Do NER on this sentence\\nphoP has been previously identified to be involved in virulence as mutational inactivation of this gene rendered P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "938/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 937,\n",
            "  \"input\": \"Do NER on this sentence\\nluminescens avirulent to Spodoptera littoralis larvae [ 6 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:luminescens', 'B-Organism:Spodoptera', 'E-Organism:littoralis']\",\n",
            "  \"response\": \"this sentence\\nluminescens avirulent to Spodoptera littoralis larvae [ 6 ]\"\n",
            "}\n",
            "939/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 938,\n",
            "  \"input\": \"Do NER on this sentence\\nFurther , in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "940/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 939,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata , lysR gene encoding a transcriptional activator protein was found to be upregulated during infection of R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "941/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 940,\n",
            "  \"input\": \"Do NER on this sentence\\nmajalis , while two genes yijC and rseA encoding transcriptional repressors were identified in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:majalis']\",\n",
            "  \"response\": \"['E-Organism:majalis']\"\n",
            "}\n",
            "942/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 941,\n",
            "  \"input\": \"Do NER on this sentence\\nBesides transcriptional regulatory genes , gene tilS , encoding a putative cell cycle protein [ 4 ] , was induced in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nBesides transcriptional regulatory genes , gene tilS , encoding a putative cell cycle protein [ 4 ]\"\n",
            "}\n",
            "943/1344 (0.21 seconds):\n",
            "{\n",
            "  \"id\": 942,\n",
            "  \"input\": \"Do NER on this sentence\\nHowever , none of these regulatory factors were identified in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nHowever , none of these regulatory factors were identified in P\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "944/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 943,\n",
            "  \"input\": \"Do NER on this sentence\\nluminescens exposed to G\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:luminescens']\",\n",
            "  \"response\": \"['E-Organism:luminescens']\"\n",
            "}\n",
            "945/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 944,\n",
            "  \"input\": \"Do NER on this sentence\\nmellonella homogenate in a previous study [ 16 ] , suggesting that regulation of these factors either depends on host ( R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:mellonella']\",\n",
            "  \"response\": \"this sentence\\nmellonella homogenate in a previous study [ 16 ]\"\n",
            "}\n",
            "946/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 945,\n",
            "  \"input\": \"Do NER on this sentence\\nmajalis not G\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:majalis']\",\n",
            "  \"response\": \"['E-Organism:majalis']\"\n",
            "}\n",
            "947/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 946,\n",
            "  \"input\": \"Do NER on this sentence\\nGenes involved in virulence and secretion Identification of genes encoding hemolysin related proteins in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "948/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 947,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata ( fhaC ) and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "949/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 948,\n",
            "  \"input\": \"Do NER on this sentence\\nAmong the virulence genes identified in this study , tcaC was identified to be induced in both P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "950/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 949,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "951/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 950,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi during infection of R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "952/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 951,\n",
            "  \"input\": \"Do NER on this sentence\\nThe tca gene family encodes four toxin complexes TcaA , TcaB , TcaC and TcaD [ 33 ] , and has been found in several bacterial pathogens including P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nThe tca gene family encodes four toxin complexes TcaA , TcaB , TcaC and TcaD [ 33 ]\"\n",
            "}\n",
            "953/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 952,\n",
            "  \"input\": \"Do NER on this sentence\\nInterestingly , Tca toxin encoding genes were also found to be induced in a related species P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "954/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 953,\n",
            "  \"input\": \"Do NER on this sentence\\nluminescens treated with the homogenate of G\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:luminescens']\",\n",
            "  \"response\": \"['E-Organism:luminescens']\"\n",
            "}\n",
            "955/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 954,\n",
            "  \"input\": \"Do NER on this sentence\\nInduction of gene pplA , encoding a phospholipase , in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "956/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 955,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata is expected to contribute to its faster growth rate [ 37 ] compared to X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"this sentence\\ntemperata is expected to contribute to its faster growth rate [ 37 ]\"\n",
            "}\n",
            "957/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 956,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi in R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "958/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 957,\n",
            "  \"input\": \"Do NER on this sentence\\nUpregulation of gene sctL , encoding components of type III secretion machinery , possibly coordinates protein secretion system in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "959/1344 (1.36 seconds):\n",
            "{\n",
            "  \"id\": 958,\n",
            "  \"input\": \"Do NER on this sentence\\nUnlike Photorhabdus , the genomes of Xenorhabdus bovienii and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Photorhabdus', 'B-Organism:Xenorhabdus', 'E-Organism:bovienii']\",\n",
            "  \"response\": \"['B-Organism:Xenorhabdus', 'E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "960/1344 (1.35 seconds):\n",
            "{\n",
            "  \"id\": 959,\n",
            "  \"input\": \"Do NER on this sentence\\nNotably , in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:luminescens']\"\n",
            "}\n",
            "961/1344 (0.67 seconds):\n",
            "{\n",
            "  \"id\": 960,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi , virH with sequence similarity to a gene encoding putative toxin secretion transporter in Vibrio cholera , was upregulated during infection of R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi', 'B-Organism:Vibrio', 'E-Organism:cholera']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "962/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 961,\n",
            "  \"input\": \"Do NER on this sentence\\nThese include pmt1 , toxC and virB from P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "963/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 962,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata , and virH from X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "964/1344 (1.46 seconds):\n",
            "{\n",
            "  \"id\": 963,\n",
            "  \"input\": \"Do NER on this sentence\\nGenes involved in adaptation to host - induced stresses Compared to all other functional classes , more stress response genes were identified to be shared by P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "965/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 964,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "966/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 965,\n",
            "  \"input\": \"Do NER on this sentence\\nSimilarly , induction of a chaperone protein encoding gene dnaK in both bacteria may picture common stress responses between P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "967/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 966,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "968/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 967,\n",
            "  \"input\": \"Do NER on this sentence\\nThe glutathione synthetase gene gshB which mediates condensation of gamma - glutamylcysteine and glycine to form glutathione [ 47 , 48 ] was induced in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nThe glutathione synthetase gene gshB which mediates condensation of gamma - glutamylcysteine and glycine to form glutathione [ 47 , 48 ]\"\n",
            "}\n",
            "969/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 968,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata , but glutathione reductase gene gor which reduces glutathione disulfide to the sulfhydryl form of glutathione [ 47 , 48 ] was induced in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"this sentence\\ntemperata , but glutathione reductase gene gor which reduces glutathione disulfide to the sulfhydryl form of glutathione [ 47 , 48 ]\"\n",
            "}\n",
            "970/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 969,\n",
            "  \"input\": \"Do NER on this sentence\\nThus , although gshB and gor corresponding products are involved in different glutathione biosynthesis pathways , expression of these two genes may suggests possible role of detoxification mechanism in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "971/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 970,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "972/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 971,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi during infection of R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "973/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 972,\n",
            "  \"input\": \"Do NER on this sentence\\nApart from the common mechanisms handling stresses , gene msrA encoding a peptide methionine sulfoxide reductase was unique to X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "974/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 973,\n",
            "  \"input\": \"Do NER on this sentence\\nThree genes , aceE , aceK and dld , whose products catalyze subsequent steps of the glyoxylate pathway [ 53 ] , were induced in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nThree genes , aceE , aceK and dld , whose products catalyze subsequent steps of the glyoxylate pathway [ 53 ]\"\n",
            "}\n",
            "975/1344 (0.22 seconds):\n",
            "{\n",
            "  \"id\": 974,\n",
            "  \"input\": \"Do NER on this sentence\\nIn contrast , two genes kdgK and pckA involved in energy metabolism in the TCA cycle were upregulated in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nIn contrast , two genes kdgK and pckA involved in energy metabolism in the TCA cycle were upregulated in P\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "976/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 975,\n",
            "  \"input\": \"Do NER on this sentence\\nGenes related to the TCA cycle have been also found to be induced in other bacteria upon host infection , for example in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "977/1344 (0.06 seconds):\n",
            "{\n",
            "  \"id\": 976,\n",
            "  \"input\": \"Do NER on this sentence\\nluminescens in the presence of G\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:luminescens']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "978/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 977,\n",
            "  \"input\": \"Do NER on this sentence\\nmellonella homogenate [ 16 ] and in Vibrio cholerae upon infection of infant mice [ 56 , 57 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:mellonella', 'B-Organism:Vibrio', 'E-Organism:cholerae', 'S-Organism:mice']\",\n",
            "  \"response\": \"this sentence\\nmellonella homogenate [ 16 ]\"\n",
            "}\n",
            "979/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 978,\n",
            "  \"input\": \"Do NER on this sentence\\nTherefore , upregulation of bioB encoding a biontin synthetase protein further suggests that TCA cycle is involved in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "980/1344 (0.46 seconds):\n",
            "{\n",
            "  \"id\": 979,\n",
            "  \"input\": \"Do NER on this sentence\\nInduction of cysK gene in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "981/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 980,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata suggests that cysteine may be required when growing in the insect , but lysine may be needed for X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "982/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 981,\n",
            "  \"input\": \"Do NER on this sentence\\nIsolation of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "983/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 982,\n",
            "  \"input\": \"Do NER on this sentence\\nIndeed , the pathway analysis reveals that def gene product interacts with CysS , a cysteinyl - tRNA synthetase ( see Additional file 1 ) , also suggesting the requirement for cysteine in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "984/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 983,\n",
            "  \"input\": \"Do NER on this sentence\\nA protease gene , clpP , involved in protein degradation , was induced in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "985/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 984,\n",
            "  \"input\": \"Do NER on this sentence\\nThe signal peptidase II encoded by lspA gene induced in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "986/1344 (0.40 seconds):\n",
            "{\n",
            "  \"id\": 985,\n",
            "  \"input\": \"Do NER on this sentence\\nSince the product of lspA has been suggested as an important virulence factor operating via maturation of several lipoproteins [ 60 ] , we postulate that the product of lspA may play a pathological role in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nSince the product of lspA has been suggested as an important virulence factor operating via maturation of several lipoproteins [ 60 ]\"\n",
            "}\n",
            "987/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 986,\n",
            "  \"input\": \"Do NER on this sentence\\nIon scavenging systems induced in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "988/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 987,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "989/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 988,\n",
            "  \"input\": \"Do NER on this sentence\\nIn P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "990/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 989,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata , pchC , a gene involved in the biogenesis of pyochelin type of siderophore [ 61 ] , was upregulated indicating the importance of iron , whereas a siderophore - independent zinc ion transport system encoded by gene znuA [ 62 ] was induced in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"this sentence\\ntemperata , pchC , a gene involved in the biogenesis of pyochelin type of siderophore [ 61 ]\"\n",
            "}\n",
            "991/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 990,\n",
            "  \"input\": \"Do NER on this sentence\\nAlso supported by the pathway analysis , product of glnD identified from P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "992/1344 (4.25 seconds):\n",
            "{\n",
            "  \"id\": 991,\n",
            "  \"input\": \"Do NER on this sentence\\nIn addition to ion scavenging systems , artM , encoding a putative ABC amino acid transport system [ 63 ] was identified in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nIn addition to ion scavenging systems , artM , encoding a putative ABC amino acid transport system [ 63 ]\"\n",
            "}\n",
            "993/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 992,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata , but not in X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "994/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 993,\n",
            "  \"input\": \"Do NER on this sentence\\nUpregulation of artM in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "995/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 994,\n",
            "  \"input\": \"Do NER on this sentence\\nSimilarly , it has been discussed previously that histidine and phosphatidylethanolamine , the possible components presented in the insect host hemolymph , may be important food sources used by P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "996/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 995,\n",
            "  \"input\": \"Do NER on this sentence\\nAs cysteine synthesis gene cysK was also identified from P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "997/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 996,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata as discussed above , it may indicate that the level of cysteine either synthesized by P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "998/1344 (2.14 seconds):\n",
            "{\n",
            "  \"id\": 997,\n",
            "  \"input\": \"Do NER on this sentence\\nInterestingly , X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:Escherichia', 'E-Organism:coli', 'B-Organism:Escherichia', 'E-Organism:coli']\"\n",
            "}\n",
            "999/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 998,\n",
            "  \"input\": \"Do NER on this sentence\\nFurther , P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1000/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 999,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata but not X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "1001/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1000,\n",
            "  \"input\": \"Do NER on this sentence\\nIn E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1002/1344 (0.46 seconds):\n",
            "{\n",
            "  \"id\": 1001,\n",
            "  \"input\": \"Do NER on this sentence\\nDespite unknown mechanistic details of Tol system , upregulation of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1003/1344 (0.22 seconds):\n",
            "{\n",
            "  \"id\": 1002,\n",
            "  \"input\": \"Do NER on this sentence\\nGenes with unknown function Ten genes encoding products with unknown functions were identified to be upregulated in both bacteria during infection of R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "1004/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1003,\n",
            "  \"input\": \"Do NER on this sentence\\nThese include two P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1005/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 1004,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and three X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "1006/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 1005,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi genes which show sequence similarity to genes encoding hypothetical proteins with unknown functions , and two P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "1007/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 1006,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and three X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "1008/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1007,\n",
            "  \"input\": \"Do NER on this sentence\\nWhile genes encoding TcaC toxin and hemolysin related proteins were identified to be induced in both P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1009/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 1008,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "1010/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 1009,\n",
            "  \"input\": \"Do NER on this sentence\\nkoppenhoeferi , the two bacteria seem to use different regulatory genes , secretion systems and virulence factors to achieve infection of the same insect host R\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:koppenhoeferi']\",\n",
            "  \"response\": \"['E-Organism:koppenhoeferi']\"\n",
            "}\n",
            "1011/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1010,\n",
            "  \"input\": \"Do NER on this sentence\\nFurther , P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1012/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 1011,\n",
            "  \"input\": \"Do NER on this sentence\\ntemperata and X\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:temperata']\",\n",
            "  \"response\": \"['E-Organism:temperata']\"\n",
            "}\n",
            "1013/1344 (2.25 seconds):\n",
            "{\n",
            "  \"id\": 1012,\n",
            "  \"input\": \"Do NER on this sentence\\nTranscriptional profiling of Actinobacillus pleuropneumoniae during the acute phase of a natural infection in pigs Background Actinobacillus pleuropneumoniae is the etiological agent of porcine pleuropneumonia , a respiratory disease which causes great economic losses worldwide\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Actinobacillus', 'E-Organism:pleuropneumoniae', 'S-Organism:pigs', 'B-Organism:Actinobacillus', 'E-Organism:pleuropneumoniae']\",\n",
            "  \"response\": \"['B-Organism:Actinobacillus', 'E-Organism:pleuropneumoniae', 'S-Organism:human']\"\n",
            "}\n",
            "1014/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 1013,\n",
            "  \"input\": \"Do NER on this sentence\\nIn order to identify genes that are expressed in vivo during a natural infection , we undertook transcript profiling experiments with an A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1015/1344 (0.81 seconds):\n",
            "{\n",
            "  \"id\": 1014,\n",
            "  \"input\": \"Do NER on this sentence\\npleuropneumoniae DNA microarray , after recovery of bacterial mRNAs from serotype 5b - infected porcine lungs\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:pleuropneumoniae', 'B-Organism:serotype', 'E-Organism:5b']\",\n",
            "  \"response\": \"['E-Organism:pleuropneumoniae']\"\n",
            "}\n",
            "1016/1344 (2.51 seconds):\n",
            "{\n",
            "  \"id\": 1015,\n",
            "  \"input\": \"Do NER on this sentence\\nAppChip2 contains 2033 PCR amplicons based on the genomic sequence of App serotype 5b strain L20 , representing more than 95 % of ORFs greater than 160 bp in length\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:AppChip2', 'B-Organism:App', 'I-Organism:serotype', 'I-Organism:5b', 'I-Organism:strain', 'E-Organism:L20']\",\n",
            "  \"response\": \"['B-Organism:App', 'E-Organism:serotype', 'S-Organism:App', 'B-Organism:App', 'E-Organism:serotype']\"\n",
            "}\n",
            "1017/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1016,\n",
            "  \"input\": \"Do NER on this sentence\\nResults Transcriptional profiling of A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1018/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1017,\n",
            "  \"input\": \"Do NER on this sentence\\nNearly all A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1019/1344 (2.21 seconds):\n",
            "{\n",
            "  \"id\": 1018,\n",
            "  \"input\": \"Do NER on this sentence\\nBackground Actinobacillus pleuropneumoniae is the causative agent of porcine pleuropneumonia , a highly contagious respiratory disease that causes great economic losses worldwide [ 1 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Actinobacillus', 'E-Organism:pleuropneumoniae']\",\n",
            "  \"response\": \"this sentence\\nBackground Actinobacillus pleuropneumoniae is the causative agent of porcine pleuropneumonia , a highly contagious respiratory disease that causes great economic losses worldwide [ 1 ]\"\n",
            "}\n",
            "1020/1344 (0.95 seconds):\n",
            "{\n",
            "  \"id\": 1019,\n",
            "  \"input\": \"Do NER on this sentence\\nDespite years of research , mechanisms involved in A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1021/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1020,\n",
            "  \"input\": \"Do NER on this sentence\\nFirst , an IVET experiment with A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1022/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1021,\n",
            "  \"input\": \"Do NER on this sentence\\nLater , over the course of two STM experiments conducted with A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1023/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1022,\n",
            "  \"input\": \"Do NER on this sentence\\nBoth experiments showed that iron acquisition was highly important for the virulence of A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1024/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1023,\n",
            "  \"input\": \"Do NER on this sentence\\nMore recently , SCOTS experiments were conducted with A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1025/1344 (0.22 seconds):\n",
            "{\n",
            "  \"id\": 1024,\n",
            "  \"input\": \"Do NER on this sentence\\nThese studies demonstrated the importance of anaerobic metabolism and HlyX - regulated mechanisms during A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nThese studies demonstrated the importance of anaerobic metabolism and HlyX - regulated mechanisms during A\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "1026/1344 (1.91 seconds):\n",
            "{\n",
            "  \"id\": 1025,\n",
            "  \"input\": \"Do NER on this sentence\\nThe HlyX regulon was characterized recently , and iron - regulated protein B ( FrpB ) was identified as a potentially important virulence factor up - regulated when A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1027/1344 (1.76 seconds):\n",
            "{\n",
            "  \"id\": 1026,\n",
            "  \"input\": \"Do NER on this sentence\\nAlthough these in vivo studies have been very valuable in gaining a better understanding of the mechanisms involved in A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\"\n",
            "}\n",
            "1028/1344 (2.05 seconds):\n",
            "{\n",
            "  \"id\": 1027,\n",
            "  \"input\": \"Do NER on this sentence\\nhave evaluated the transcriptome of Vibrio cholerae in the early and late stages of the disease by collecting stools of naturally infected patients [ 22 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Vibrio', 'E-Organism:cholerae', 'S-Organism:patients']\",\n",
            "  \"response\": \"this sentence\\nhave evaluated the transcriptome of Vibrio cholerae in the early and late stages of the disease by collecting stools of naturally infected patients [ 22 ]\"\n",
            "}\n",
            "1029/1344 (2.31 seconds):\n",
            "{\n",
            "  \"id\": 1028,\n",
            "  \"input\": \"Do NER on this sentence\\nused fresh sputum samples from a patient with cystic fibrosis to study the in vivo transcriptome of Pseudomonas aeruginosa [ 23 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:patient', 'B-Organism:Pseudomonas', 'E-Organism:aeruginosa']\",\n",
            "  \"response\": \"this sentence\\nused fresh sputum samples from a patient with cystic fibrosis to study the in vivo transcriptome of Pseudomonas aeruginosa [ 23 ]\"\n",
            "}\n",
            "1030/1344 (0.46 seconds):\n",
            "{\n",
            "  \"id\": 1029,\n",
            "  \"input\": \"Do NER on this sentence\\nIn this study , we have conducted in vivo transcript profiling with DNA microarrays of A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1031/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1030,\n",
            "  \"input\": \"Do NER on this sentence\\nOf these , 72 were up - regulated , and three of these encode lipoproteins or OMPs that are well - conserved among strains of A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1032/1344 (1.36 seconds):\n",
            "{\n",
            "  \"id\": 1031,\n",
            "  \"input\": \"Do NER on this sentence\\nWhile it can be argued that different animals might react differently to an infection by A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "1033/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 1032,\n",
            "  \"input\": \"Do NER on this sentence\\npleuropneumoniae , and therefore elicit different transcriptional changes in the pathogen , we are confident that the results that were obtained reflect accurately the conditions encountered by A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:pleuropneumoniae']\",\n",
            "  \"response\": \"['E-Organism:pleuropneumoniae']\"\n",
            "}\n",
            "1034/1344 (1.11 seconds):\n",
            "{\n",
            "  \"id\": 1033,\n",
            "  \"input\": \"Do NER on this sentence\\nThe infectious field strain that was isolated from the lung tissues was named 896 - 07 , and serotyping analysis showed that it belongs to serotype 5b , one of the most prevalent in North America [ 4 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:serotype', 'E-Organism:5b']\",\n",
            "  \"response\": \"this sentence\\nThe infectious field strain that was isolated from the lung tissues was named 896 - 07 , and serotyping analysis showed that it belongs to serotype 5b , one of the most prevalent in North America [ 4 ]\"\n",
            "}\n",
            "1035/1344 (1.71 seconds):\n",
            "{\n",
            "  \"id\": 1034,\n",
            "  \"input\": \"Do NER on this sentence\\nComparative genomic hybridizations CGH experiments conducted previously in our laboratory have shown that the majority of genes in the reference 5b L20 strain were conserved in reference strains and fresh field isolates\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:5b', 'E-Organism:L20']\",\n",
            "  \"response\": \"['B-Organism:isolate', 'I-Organism:reference', 'E-Organism:strain', 'S-Organism:L20']\"\n",
            "}\n",
            "1036/1344 (1.56 seconds):\n",
            "{\n",
            "  \"id\": 1035,\n",
            "  \"input\": \"Do NER on this sentence\\nAppChip2 was designed using the complete genome sequence from the serotype 5b reference strain L20\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:AppChip2', 'B-Organism:serotype', 'I-Organism:5b', 'I-Organism:reference', 'I-Organism:strain', 'E-Organism:L20']\",\n",
            "  \"response\": \"['B-Organism:serotype', 'E-Organism:5b', 'S-Organism:L20']\"\n",
            "}\n",
            "1037/1344 (2.06 seconds):\n",
            "{\n",
            "  \"id\": 1036,\n",
            "  \"input\": \"Do NER on this sentence\\nIn order to validate its use to assess transcriptomic events taking place with our infectious field strain during the infection , we first used it to analyze the gene content of the 896 - 07 field strain\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:896', 'E-Organism:07']\",\n",
            "  \"response\": \"['B-Organism:non', 'I-Organism:invasive', 'E-Organism:GAS', 'S-Organism:GAS']\"\n",
            "}\n",
            "1038/1344 (4.66 seconds):\n",
            "{\n",
            "  \"id\": 1037,\n",
            "  \"input\": \"Do NER on this sentence\\nThe CGH data obtained with strain 896 - 07 was added to that obtained with other field and reference strains [ 25 ] in order to generate a strain dendogram ( Figure 2 )\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:896', 'E-Organism:07']\",\n",
            "  \"response\": \"this sentence\\nThe CGH data obtained with strain 896 - 07 was added to that obtained with other field and reference strains [ 25 ]\"\n",
            "}\n",
            "1039/1344 (1.11 seconds):\n",
            "{\n",
            "  \"id\": 1038,\n",
            "  \"input\": \"Do NER on this sentence\\nSerotyping data correlates well with genetic data since strain 896 - 07 forms a cluster with the serotype 5b reference strain L20\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:896', 'E-Organism:07', 'B-Organism:serotype', 'I-Organism:5b', 'I-Organism:reference', 'I-Organism:strain', 'E-Organism:L20']\",\n",
            "  \"response\": \"['B-Organism:serotype', 'E-Organism:5b']\"\n",
            "}\n",
            "1040/1344 (0.07 seconds):\n",
            "{\n",
            "  \"id\": 1039,\n",
            "  \"input\": \"Do NER on this sentence\\nOnly 28 genes were identified as divergent / absent in our field strain , when compared to the reference serotype 5b L20 strain ( Table 1 )\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:5b', 'E-Organism:L20']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "1041/1344 (5.46 seconds):\n",
            "{\n",
            "  \"id\": 1040,\n",
            "  \"input\": \"Do NER on this sentence\\nOut of these 28 genes , 20 can be found in two major regions of the genome : 14 putative ORFs code for phage structures , while 6 are located in the A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nOut of these 28 genes , 20 can be found in two major regions of the genome : 14 putative ORFs code for phage structures , while 6 are located in the A\\nReply in key value pair as Tag and Entity. and Cysteinyl tRNA synthetase gene ( tssC ) and the genes encoding the putative ABC transporter ( glnN / glnP / glnQ ) and the putative gluconeogenesis gene ( glnH / glnI ), respectively, found in Bacillus subtilis [ 40 ]\"\n",
            "}\n",
            "1042/1344 (1.46 seconds):\n",
            "{\n",
            "  \"id\": 1041,\n",
            "  \"input\": \"Do NER on this sentence\\nIn Aggregatibacter actinomycetemcomitans , a close relative of A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Aggregatibacter', 'E-Organism:actinomycetemcomitans']\",\n",
            "  \"response\": \"['B-Organism:Aggregatibacter', 'E-Organism:actinomycetemcomitans']\"\n",
            "}\n",
            "1043/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1042,\n",
            "  \"input\": \"Do NER on this sentence\\nIn A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1044/1344 (0.91 seconds):\n",
            "{\n",
            "  \"id\": 1043,\n",
            "  \"input\": \"Do NER on this sentence\\nSome genes might also be present in the genome of strain 896 - 07 and absent in reference strain L20 , and therefore absent on AppChip2\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:896', 'E-Organism:07', 'S-Organism:L20', 'S-Organism:AppChip2']\",\n",
            "  \"response\": \"['B-Organism:896P', 'E-Organism:07']\"\n",
            "}\n",
            "1045/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1044,\n",
            "  \"input\": \"Do NER on this sentence\\nOur results indicated that A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1046/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1045,\n",
            "  \"input\": \"Do NER on this sentence\\nIn vivo transcript profiling Before investigating the in vivo transcript profile of A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1047/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1046,\n",
            "  \"input\": \"Do NER on this sentence\\nThe transcriptome of A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1048/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1047,\n",
            "  \"input\": \"Do NER on this sentence\\n2 % ) could be found in the list of divergent or highly divergent / absent genes in at least one reference strain of A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1049/1344 (1.50 seconds):\n",
            "{\n",
            "  \"id\": 1048,\n",
            "  \"input\": \"Do NER on this sentence\\nOf the last five genes , four are divergent in only one strain ( hktE , fur , phoR and hisH , respectively in reference strains from serotypes 8 , 2 , 15 and 7 ) , and the APL _ 0999 gene codes for a hypothetical protein that is divergent in serotype 2 , 7 - 11 and 14 reference strains\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:serotypes', 'S-Organism:15', 'B-Organism:serotype', 'S-Organism:11', 'S-Organism:14']\",\n",
            "  \"response\": \"['B-Organism:serotype', 'E-Organism:reference']\"\n",
            "}\n",
            "1050/1344 (0.46 seconds):\n",
            "{\n",
            "  \"id\": 1049,\n",
            "  \"input\": \"Do NER on this sentence\\na ) Validation of microarray results by qRT - PCR In order to confirm results obtained using the AppChip2 A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:AppChip2']\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1051/1344 (1.31 seconds):\n",
            "{\n",
            "  \"id\": 1050,\n",
            "  \"input\": \"Do NER on this sentence\\nb ) Comparison with other in vivo studies Among previous in vivo experiments that were conducted with A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "1052/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1051,\n",
            "  \"input\": \"Do NER on this sentence\\nIn their experiment , Selective Capture of Transcribed Sequences ( SCOTS ) was performed on lung tissues after a 7 day infection with A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1053/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 1052,\n",
            "  \"input\": \"Do NER on this sentence\\nOf the 150 differentially expressed genes that we have identified ( see Additional file 1 ) , 15 were also identified in other in vivo studies conducted with A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1054/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1053,\n",
            "  \"input\": \"Do NER on this sentence\\nThe latter , the gene encoding the recently discovered fourth A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1055/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1054,\n",
            "  \"input\": \"Do NER on this sentence\\nWhile STM and IVET can theoretically provide a global overview of transcriptomic events , the experiments conducted with A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1056/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1055,\n",
            "  \"input\": \"Do NER on this sentence\\nwith IVET [ 12 ] , well short of the approximately 2000 genes of the A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nwith IVET [ 12 ]\"\n",
            "}\n",
            "1057/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1056,\n",
            "  \"input\": \"Do NER on this sentence\\nSTM relies on transposon mutagenesis , and there are some insertional hot spots in the A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1058/1344 (3.01 seconds):\n",
            "{\n",
            "  \"id\": 1057,\n",
            "  \"input\": \"Do NER on this sentence\\nWe also compared our results to the transcriptional profile of a virulent strain of Pseudomonas aeruginosa , another lung pathogen , isolated from a chronically infected cystic fibrosis patient [ 23 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Pseudomonas', 'E-Organism:aeruginosa', 'S-Organism:patient']\",\n",
            "  \"response\": \"this sentence\\nWe also compared our results to the transcriptional profile of a virulent strain of Pseudomonas aeruginosa , another lung pathogen , isolated from a chronically infected cystic fibrosis patient [ 23 ]\"\n",
            "}\n",
            "1059/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1058,\n",
            "  \"input\": \"Do NER on this sentence\\nSince P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1060/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1059,\n",
            "  \"input\": \"Do NER on this sentence\\n, we noticed that few of the landmark virulence genes in A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1061/1344 (1.01 seconds):\n",
            "{\n",
            "  \"id\": 1060,\n",
            "  \"input\": \"Do NER on this sentence\\nOut of the three Apx toxins present in serotype 5b , only the gene encoding the ApxIV toxin was up - regulated\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:serotype', 'E-Organism:5b']\",\n",
            "  \"response\": \"['B-Organism:serotype', 'E-Organism:5b']\"\n",
            "}\n",
            "1062/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1061,\n",
            "  \"input\": \"Do NER on this sentence\\nIn the case of the P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1063/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1062,\n",
            "  \"input\": \"Do NER on this sentence\\nWhether this is also the case for A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1064/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1063,\n",
            "  \"input\": \"Do NER on this sentence\\nThese genes are scattered in the L20 genome , but all their respective operonic structures are well conserved when compared to those observed in E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:L20']\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1065/1344 (0.31 seconds):\n",
            "{\n",
            "  \"id\": 1064,\n",
            "  \"input\": \"Do NER on this sentence\\nThis would probably indicate that A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1066/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1065,\n",
            "  \"input\": \"Do NER on this sentence\\nFurther strengthening this hypothesis is the fact that two subunits from DNA polymerase III ( dnaN , dnaX ) , which is the primary enzyme for replicative DNA functions in E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1067/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1066,\n",
            "  \"input\": \"Do NER on this sentence\\nIn E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1068/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1067,\n",
            "  \"input\": \"Do NER on this sentence\\nTherefore , it would seem that the in vivo environment towards the end of the acute phase of pleuropneumonia is not iron - restricted for A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1069/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 1068,\n",
            "  \"input\": \"Do NER on this sentence\\nNo sRNAs have been identified in A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1070/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1069,\n",
            "  \"input\": \"Do NER on this sentence\\nIndeed , an acapsular mutant of A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1071/1344 (0.07 seconds):\n",
            "{\n",
            "  \"id\": 1070,\n",
            "  \"input\": \"Do NER on this sentence\\n39x ) and codes for a putative secreted serine protease that seems to be well conserved in other pathogenic bacteria , such as Haemophilus parasuis , Neisseria meningitidis and N\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Haemophilus', 'E-Organism:parasuis', 'B-Organism:Neisseria', 'E-Organism:meningitidis']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "1072/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 1071,\n",
            "  \"input\": \"Do NER on this sentence\\nIn the lungs , A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1073/1344 (1.36 seconds):\n",
            "{\n",
            "  \"id\": 1072,\n",
            "  \"input\": \"Do NER on this sentence\\nHere , we report that genes apfB and apfC , coding for the type IV pilin biogenesis proteins ApfB and ApfC , were up - regulated in vivo during natural infection by an A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "1074/1344 (1.35 seconds):\n",
            "{\n",
            "  \"id\": 1073,\n",
            "  \"input\": \"Do NER on this sentence\\nCurli fibers are important for biofilm formation in numerous pathogens , including E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "1075/1344 (0.06 seconds):\n",
            "{\n",
            "  \"id\": 1074,\n",
            "  \"input\": \"Do NER on this sentence\\ncoli and Salmonella enteritidis , and are involved in the first steps of attachment\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:coli', 'B-Organism:Salmonella', 'E-Organism:enteritidis']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "1076/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1075,\n",
            "  \"input\": \"Do NER on this sentence\\nIn A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1077/1344 (1.56 seconds):\n",
            "{\n",
            "  \"id\": 1076,\n",
            "  \"input\": \"Do NER on this sentence\\npleuropneumoniae L20 , this organization is not respected : APL _ 0220 seems to form an operon with the genes APL _ 0221 and APL _ 0222\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:pleuropneumoniae', 'E-Organism:L20']\",\n",
            "  \"response\": \"['I-Organism:pleuropneumoniae', 'I-Organism:strain', 'E-Organism:L20']\"\n",
            "}\n",
            "1078/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1077,\n",
            "  \"input\": \"Do NER on this sentence\\nFinally , the kpsF gene , coding for an arabinose - 5 - phosphate isomerase , should also be mentioned since it is involved in the pathway responsible for the synthesis of 2 - keto - 3deozyoctulosonic acid ( Kdo ) , which is present in the A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1079/1344 (1.41 seconds):\n",
            "{\n",
            "  \"id\": 1078,\n",
            "  \"input\": \"Do NER on this sentence\\npleuropneumoniae LPS core oligosaccharide [ 52 ] and capsule of serotype 5 [ 53 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:pleuropneumoniae', 'B-Organism:serotype']\",\n",
            "  \"response\": \"this sentence\\npleuropneumoniae LPS core oligosaccharide [ 52 ]\"\n",
            "}\n",
            "1080/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1079,\n",
            "  \"input\": \"Do NER on this sentence\\nIn E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1081/1344 (5.31 seconds):\n",
            "{\n",
            "  \"id\": 1080,\n",
            "  \"input\": \"Do NER on this sentence\\nAlthough it is not known whether maltose or maltodextrin are readily available as carbohydrate sources in mammalian lungs , it was previously shown that maltodextrin utilization is important for the colonization of the oro - pharynx in Group A Streptococcus [ 57 , 58 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Group', 'E-Organism:Streptococcus']\",\n",
            "  \"response\": \"this sentence\\nAlthough it is not known whether maltose or maltodextrin are readily available as carbohydrate sources in mammalian lungs , it was previously shown that maltodextrin utilization is important for the colonization of the oro - pharynx in Group A Streptococcus [ 57 , 58 ]\"\n",
            "}\n",
            "1082/1344 (1.36 seconds):\n",
            "{\n",
            "  \"id\": 1081,\n",
            "  \"input\": \"Do NER on this sentence\\nIt was also recently shown that catabolism of maltose provides a competitive advantage to pathogenic and commensal E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "1083/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1082,\n",
            "  \"input\": \"Do NER on this sentence\\nIn E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1084/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1083,\n",
            "  \"input\": \"Do NER on this sentence\\nWhile ascorbate use has not been linked to bacterial virulence before , it has been shown on multiple occasions that genes involved in anaerobic metabolism are important for the virulence of A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1085/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1084,\n",
            "  \"input\": \"Do NER on this sentence\\nThis suggests that HlyX is probably not highly expressed during the acute phase of the disease caused by A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1086/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1085,\n",
            "  \"input\": \"Do NER on this sentence\\nIn E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1087/1344 (3.09 seconds):\n",
            "{\n",
            "  \"id\": 1086,\n",
            "  \"input\": \"Do NER on this sentence\\nThe phosphate regulon in E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nThe phosphate regulon in E\\nReply in key value pair as Tag and Entity. and return it in a B-Organism-Response-File like structure [ 16 ]\"\n",
            "}\n",
            "1088/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1087,\n",
            "  \"input\": \"Do NER on this sentence\\nA\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1089/1344 (1.56 seconds):\n",
            "{\n",
            "  \"id\": 1088,\n",
            "  \"input\": \"Do NER on this sentence\\npleuropneumoniae strain L20 harbours an operon composed of the genes ptsSCAB - phoBR\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:pleuropneumoniae', 'I-Organism:strain', 'E-Organism:L20']\",\n",
            "  \"response\": \"['I-Organism:pleuropneumoniae', 'I-Organism:strain', 'E-Organism:L20']\"\n",
            "}\n",
            "1090/1344 (1.56 seconds):\n",
            "{\n",
            "  \"id\": 1089,\n",
            "  \"input\": \"Do NER on this sentence\\nSince it seems unlikely that porcine lungs are devoid of phosphate , the A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:Porcine', 'E-Organism:lungs', 'S-Organism:Porcine']\"\n",
            "}\n",
            "1091/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1090,\n",
            "  \"input\": \"Do NER on this sentence\\nHowever , since this was deduced from sequence homologies , the real function of irp in A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1092/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1091,\n",
            "  \"input\": \"Do NER on this sentence\\nThe protein sequence deduced from the APL _ 0920 gene sequence is present in all sequenced A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1093/1344 (1.91 seconds):\n",
            "{\n",
            "  \"id\": 1092,\n",
            "  \"input\": \"Do NER on this sentence\\npleuropneumoniae strains and in Mannheimia haemolytica PHL213 , but no other strong homologies can be found in other bacterial species\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:pleuropneumoniae', 'B-Organism:Mannheimia', 'I-Organism:haemolytica', 'E-Organism:PHL213']\",\n",
            "  \"response\": \"['I-Organism:pleuropneumoniae', 'I-Organism:serotype', 'E-Organism:5']\"\n",
            "}\n",
            "1094/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1093,\n",
            "  \"input\": \"Do NER on this sentence\\nIt is the most important adhesin in B\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1095/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 1094,\n",
            "  \"input\": \"Do NER on this sentence\\npertussis , and deletion of FHA in B\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:pertussis']\",\n",
            "  \"response\": \"['I-Organism:pertussis', 'E-Organism:mutants']\"\n",
            "}\n",
            "1096/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1095,\n",
            "  \"input\": \"Do NER on this sentence\\nThe A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1097/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1096,\n",
            "  \"input\": \"Do NER on this sentence\\nIn B\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1098/1344 (1.51 seconds):\n",
            "{\n",
            "  \"id\": 1097,\n",
            "  \"input\": \"Do NER on this sentence\\nIn Histophilus somni , the FhaB homolog is thought to also have an important role in biofilm formation in cases of myocarditis caused by this micro - organism [ 75 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Histophilus', 'E-Organism:somni']\",\n",
            "  \"response\": \"this sentence\\nIn Histophilus somni , the FhaB homolog is thought to also have an important role in biofilm formation in cases of myocarditis caused by this micro - organism [ 75 ]\"\n",
            "}\n",
            "1099/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1098,\n",
            "  \"input\": \"Do NER on this sentence\\nUp - regulation of the A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1100/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1099,\n",
            "  \"input\": \"Do NER on this sentence\\nSurprisingly , this gene was not identified as up - regulated in previous adhesion experiments performed in our lab [ 9 ] , and it even showed a slight , although not significant , down - regulation when A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nSurprisingly , this gene was not identified as up - regulated in previous adhesion experiments performed in our lab [ 9 ]\"\n",
            "}\n",
            "1101/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1100,\n",
            "  \"input\": \"Do NER on this sentence\\nThe three proteins that are encoded by genes fhaB , irp and APL _ 0920 have all these features , and might therefore represent good candidates to include in future cross - serotype vaccines against A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1102/1344 (0.51 seconds):\n",
            "{\n",
            "  \"id\": 1101,\n",
            "  \"input\": \"Do NER on this sentence\\nFHA of B\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1103/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1102,\n",
            "  \"input\": \"Do NER on this sentence\\nThus the A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1104/1344 (0.47 seconds):\n",
            "{\n",
            "  \"id\": 1103,\n",
            "  \"input\": \"Do NER on this sentence\\nConclusions To the best of our knowledge , this study represents the first and only genome - wide in vivo gene expression experiment to be conducted with A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1105/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1104,\n",
            "  \"input\": \"Do NER on this sentence\\nBy comparing our results to those obtained following comparative genomics and proteomics experiments , we were able to identify three genes that were conserved in tested reference strains and in fresh field isolates of A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1106/1344 (0.30 seconds):\n",
            "{\n",
            "  \"id\": 1105,\n",
            "  \"input\": \"Do NER on this sentence\\nThese three genes , namely fhaB ( APL _ 0959 ) , irp ( APL _ 0919 ) and APL _ 0920 , could all represent excellent candidates for the development of a protective cross - serotype subunit vaccine against A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1107/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1106,\n",
            "  \"input\": \"Do NER on this sentence\\nWhile our results can help decipher some of the adaptations A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1108/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1107,\n",
            "  \"input\": \"Do NER on this sentence\\nThe acute stage and the chronic stage of the disease are probably highly different conditions , and it would be interesting in future works to compare our present data with the transcriptomic profile of A\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1109/1344 (1.17 seconds):\n",
            "{\n",
            "  \"id\": 1108,\n",
            "  \"input\": \"Do NER on this sentence\\nTo survive in the host environment , Salmonella enterica uses a type III secretion system and effector proteins , which are activated by the SsrA - SsrB two - component system in response to the host environment\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\"\n",
            "}\n",
            "1110/1344 (3.44 seconds):\n",
            "{\n",
            "  \"id\": 1109,\n",
            "  \"input\": \"Do NER on this sentence\\nTo better understand the phenomenon of regulatory evolution in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nTo better understand the phenomenon of regulatory evolution in S\\nReply in key value pair as Tag and Entity. and delivered into the same strain of Salmonella typhimurium ( referred to as Deltadefector ) as a part of the same genetic operon [ 4 ]\"\n",
            "}\n",
            "1111/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1110,\n",
            "  \"input\": \"Do NER on this sentence\\nFurther , we used a genetic screen and computational analyses integrating experimental data from S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1112/1344 (0.67 seconds):\n",
            "{\n",
            "  \"id\": 1111,\n",
            "  \"input\": \"Do NER on this sentence\\nenterica and sequence data from an orthologous regulatory system in the insect endosymbiont , Sodalis glossinidius , to identify the conserved yet flexible palindrome sequence that defines DNA recognition by SsrB\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:enterica', 'B-Organism:Sodalis', 'E-Organism:glossinidius']\",\n",
            "  \"response\": \"['E-Organism:enterica']\"\n",
            "}\n",
            "1113/1344 (1.51 seconds):\n",
            "{\n",
            "  \"id\": 1112,\n",
            "  \"input\": \"Do NER on this sentence\\nFor instance , Salmonella enterica has a multi - faceted pathogenic strategy fine - tuned by several transcriptional regulators\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\"\n",
            "}\n",
            "1114/1344 (1.20 seconds):\n",
            "{\n",
            "  \"id\": 1113,\n",
            "  \"input\": \"Do NER on this sentence\\nIn the Salmonella genus , the SPI - 2 genomic island is found only in pathogenic serotypes of Salmonella enterica that infect warm - blooded animals and is absent from Salmonella bongori , which colonizes cold - blooded animals [ 18 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Salmonella', 'B-Organism:Salmonella', 'E-Organism:enterica', 'B-Organism:Salmonella', 'E-Organism:bongori']\",\n",
            "  \"response\": \"this sentence\\nIn the Salmonella genus , the SPI - 2 genomic island is found only in pathogenic serotypes of Salmonella enterica that infect warm - blooded animals and is absent from Salmonella bongori , which colonizes cold - blooded animals [ 18 ]\"\n",
            "}\n",
            "1115/1344 (0.07 seconds):\n",
            "{\n",
            "  \"id\": 1114,\n",
            "  \"input\": \"Do NER on this sentence\\nIt is generally accepted that SPI - 2 was acquired by Salmonella enterica after divergence from S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "1116/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 1115,\n",
            "  \"input\": \"Do NER on this sentence\\nWe recently demonstrated the evolutionary significance of cis - regulatory mutations for pathoadaptation of Salmonella enterica serovar Typhimurium ( S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'I-Organism:enterica', 'I-Organism:serovar', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\"\n",
            "}\n",
            "1117/1344 (0.77 seconds):\n",
            "{\n",
            "  \"id\": 1116,\n",
            "  \"input\": \"Do NER on this sentence\\nCombining our data with previous biochemical work , along with comparative genomic analyses with an orthologous T3SS - encoding genomic island in the tsetse fly endosymbiont , Sodalis glossinidius , we reveal the flexible DNA palindrome that distributes SsrB in the genome to influence transcriptional activation of the SPI - 2 T3SS and almost all of its accessory effector proteins\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Sodalis', 'E-Organism:glossinidius']\",\n",
            "  \"response\": \"['B-Organism:Symbioflor']\"\n",
            "}\n",
            "1118/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1117,\n",
            "  \"input\": \"Do NER on this sentence\\nResults Transcriptional profiling of an ssrB mutant To begin to understand regulatory evolution and network expansion of the SsrB response regulator , we profiled the transcriptome of an ssrB mutant and compared it to S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1119/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 1118,\n",
            "  \"input\": \"Do NER on this sentence\\nNext , we performed a Clusters of Orthologous Groups ( COG ) analysis [ 22 ] on the 118 genes that had an ortholog in the annotated genome of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nNext , we performed a Clusters of Orthologous Groups ( COG ) analysis [ 22 ]\"\n",
            "}\n",
            "1120/1344 (1.55 seconds):\n",
            "{\n",
            "  \"id\": 1119,\n",
            "  \"input\": \"Do NER on this sentence\\nTyphimurium strain LT2 [ 23 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Typhimurium', 'I-Organism:strain', 'E-Organism:LT2']\",\n",
            "  \"response\": \"this sentence\\nTyphimurium strain LT2 [ 23 ]\"\n",
            "}\n",
            "1121/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1120,\n",
            "  \"input\": \"Do NER on this sentence\\nThe SsrA - SsrB system was acquired by horizontal gene transfer into the S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1122/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 1121,\n",
            "  \"input\": \"Do NER on this sentence\\nenterica species after divergence from what is now extant S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:enterica']\",\n",
            "  \"response\": \"['E-Organism:enterica']\"\n",
            "}\n",
            "1123/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1122,\n",
            "  \"input\": \"Do NER on this sentence\\nAs such , S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1124/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1123,\n",
            "  \"input\": \"Do NER on this sentence\\nOrthologous genes ancestral to both species but regulated by SsrB in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1125/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1124,\n",
            "  \"input\": \"Do NER on this sentence\\nTo expand on this , we used a reciprocal BLAST - based analysis and identified 47 orthologs in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1126/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1125,\n",
            "  \"input\": \"Do NER on this sentence\\n1 - fold ) was subtler than for the S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1127/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1126,\n",
            "  \"input\": \"Do NER on this sentence\\nWe also determined the distribution of down - regulated genes among genomic islands [ 24 ] , including prophages , pathogenicity islands ( SPI - islands ) and additional regions of difference ( ROD ) between S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nWe also determined the distribution of down - regulated genes among genomic islands [ 24 ]\"\n",
            "}\n",
            "1128/1344 (0.56 seconds):\n",
            "{\n",
            "  \"id\": 1127,\n",
            "  \"input\": \"Do NER on this sentence\\nenterica and S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:enterica']\",\n",
            "  \"response\": \"['E-Organism:enterica']\"\n",
            "}\n",
            "1129/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1128,\n",
            "  \"input\": \"Do NER on this sentence\\nFor this we used a BLAST - based comparison of genome - wide synteny between S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1130/1344 (1.65 seconds):\n",
            "{\n",
            "  \"id\": 1129,\n",
            "  \"input\": \"Do NER on this sentence\\nTyphimurium and S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Typhimurium']\",\n",
            "  \"response\": \"['I-Organism:typhimurium', 'I-Organism:strain', 'E-Organism:BHG23']\"\n",
            "}\n",
            "1131/1344 (0.95 seconds):\n",
            "{\n",
            "  \"id\": 1130,\n",
            "  \"input\": \"Do NER on this sentence\\nGenome - wide SsrB interactions To examine SsrB allocation on the chromosome in vivo , we isolated functional SsrB - DNA interactions using chromatin immunoprecipitation and examined the bound DNA by chip analysis ( ChIP - on - chip ) using an S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nGenome - wide SsrB interactions To examine SsrB allocation on the chromosome in vivo , we isolated functional SsrB - DNA interactions using chromatin immunoprecipitation and examined the bound DNA by chip analysis ( ChIP - on - chip ) using an S\\nReply in key value pair as Tag and Entity. Response: [ ('SsrB ','SsrB') ]\"\n",
            "}\n",
            "1132/1344 (1.31 seconds):\n",
            "{\n",
            "  \"id\": 1131,\n",
            "  \"input\": \"Do NER on this sentence\\nTyphimurium SL1344 array containing 44 , 021 probes\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Typhimurium', 'E-Organism:SL1344']\",\n",
            "  \"response\": \"['B-Organism:Typhimurium', 'E-Organism:SL1344']\"\n",
            "}\n",
            "1133/1344 (0.86 seconds):\n",
            "{\n",
            "  \"id\": 1132,\n",
            "  \"input\": \"Do NER on this sentence\\nGiven the strong influence of SsrB on horizontally acquired genes ( Table 1 ) , we plotted the ChIP - on - chip data against all genomic islands in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1134/1344 (1.20 seconds):\n",
            "{\n",
            "  \"id\": 1133,\n",
            "  \"input\": \"Do NER on this sentence\\nTyphimurium SL1344\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Typhimurium', 'E-Organism:SL1344']\",\n",
            "  \"response\": \"['B-Organism:Typhimurium', 'E-Organism:SL1344']\"\n",
            "}\n",
            "1135/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 1134,\n",
            "  \"input\": \"Do NER on this sentence\\nTo generate a consensus set of SsrB regulated genes , we performed an analysis to identify operons in the S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1136/1344 (1.70 seconds):\n",
            "{\n",
            "  \"id\": 1135,\n",
            "  \"input\": \"Do NER on this sentence\\nTyphimurium SL1344 genome that encoded at least one gene down - regulated in DeltassrB cells and that possessed an SsrB binding peak in the upstream regulatory region as defined by our ChIP - on - chip analysis\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Typhimurium', 'E-Organism:SL1344']\",\n",
            "  \"response\": \"['B-Organism:Typhimurium', 'E-Organism:SL1344']\"\n",
            "}\n",
            "1137/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1136,\n",
            "  \"input\": \"Do NER on this sentence\\nWe used the PhoP response regulator from E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1138/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1137,\n",
            "  \"input\": \"Do NER on this sentence\\nIdentification of a functional DNA palindrome conserved in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1139/1344 (1.21 seconds):\n",
            "{\n",
            "  \"id\": 1138,\n",
            "  \"input\": \"Do NER on this sentence\\nTyphimurium SPI - 2 and the Sodalis glossinidius SSR - 3 region The analysis of regulatory evolution is particularly challenging because it is difficult to distinguish neutral stochastic change from functional divergence\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Typhimurium', 'B-Organism:Sodalis', 'E-Organism:glossinidius']\",\n",
            "  \"response\": \"['B-Organism:Typhimurium', 'E-Organism:Sodalis']\"\n",
            "}\n",
            "1140/1344 (1.26 seconds):\n",
            "{\n",
            "  \"id\": 1139,\n",
            "  \"input\": \"Do NER on this sentence\\nThe tsetse fly endosymbiont Sodalis glossinidius contains the Sodalis Symbiosis Region - 3 ( SSR - 3 ) that is similar in content and synteny with the S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Sodalis', 'E-Organism:glossinidius', 'S-Organism:Sodalis']\",\n",
            "  \"response\": \"['B-Organism:Symbioflor']\"\n",
            "}\n",
            "1141/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1140,\n",
            "  \"input\": \"Do NER on this sentence\\nInterestingly , two copies of the palindrome occur upstream of the ssrA - ssrB operon in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1142/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1141,\n",
            "  \"input\": \"Do NER on this sentence\\nSsrB directs transcriptional input using a flexible palindrome architecture The presence of a conserved palindrome sequence in SPI - 2 promoters and in related sequences from the endosymbiont S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1143/1344 (0.22 seconds):\n",
            "{\n",
            "  \"id\": 1142,\n",
            "  \"input\": \"Do NER on this sentence\\nWe chose the ssaG promoter for these experiments because 16 of 18 bases were identical between SPI - 2 and SSR - 3 from S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nWe chose the ssaG promoter for these experiments because 16 of 18 bases were identical between SPI - 2 and SSR - 3 from S\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "1144/1344 (0.70 seconds):\n",
            "{\n",
            "  \"id\": 1143,\n",
            "  \"input\": \"Do NER on this sentence\\nTo demonstrate a functional role for the palindrome identified in Sodalis , we engineered luciferase transcriptional reporters that either contained ( 7  - 4 - 7   ) or lacked ( X - X - X ) the identified palindrome from the Sodalis SG1292 promoter ( ssaG ortholog ) and transformed them into wild - type S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Sodalis', 'S-Organism:Sodalis']\",\n",
            "  \"response\": \"['S-Organism:Sodalis']\"\n",
            "}\n",
            "1145/1344 (3.44 seconds):\n",
            "{\n",
            "  \"id\": 1144,\n",
            "  \"input\": \"Do NER on this sentence\\nWe then searched for representative candidates of this motif in the S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "1146/1344 (0.42 seconds):\n",
            "{\n",
            "  \"id\": 1145,\n",
            "  \"input\": \"Do NER on this sentence\\nThe SsrA - SsrB two - component regulatory system in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1147/1344 (1.11 seconds):\n",
            "{\n",
            "  \"id\": 1146,\n",
            "  \"input\": \"Do NER on this sentence\\nA search of the S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1148/1344 (1.11 seconds):\n",
            "{\n",
            "  \"id\": 1147,\n",
            "  \"input\": \"Do NER on this sentence\\nWe showed that the SsrB regulatory palindrome is also present in the orthologous SSR - 3 island of the endosymbiont Sodalis glossinidius and that the palindrome evolved in Sodalis can act as a cis regulatory input function in Salmonella\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Sodalis', 'E-Organism:glossinidius', 'S-Organism:Sodalis', 'S-Organism:Salmonella']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\"\n",
            "}\n",
            "1149/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 1148,\n",
            "  \"input\": \"Do NER on this sentence\\nThus , in addition to supporting a pathogenic lifestyle within a host in Salmonella , it seems probable that this common promoter architecture may direct the activation of the SSR - 3 T3SS of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Salmonella']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\"\n",
            "}\n",
            "1150/1344 (0.67 seconds):\n",
            "{\n",
            "  \"id\": 1149,\n",
            "  \"input\": \"Do NER on this sentence\\nglossinidius in its endosymbiotic relationship with the tsetse fly host , although we acknowledge this requires experimental validation\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:glossinidius', 'B-Organism:tsetse', 'E-Organism:fly']\",\n",
            "  \"response\": \"['E-Organism:glossinidius']\"\n",
            "}\n",
            "1151/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1150,\n",
            "  \"input\": \"Do NER on this sentence\\nThe SSR - 3 region in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1152/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1151,\n",
            "  \"input\": \"Do NER on this sentence\\nThe SsrB ortholog in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1153/1344 (1.20 seconds):\n",
            "{\n",
            "  \"id\": 1152,\n",
            "  \"input\": \"Do NER on this sentence\\nThe endosymbiotic relationship of Sodalis with the tsetse fly - where long - term vertical transmission has ostensibly been formative in shaping regulatory circuitry at certain promoters - may obviate the need for rapid transcriptional surge , leading to regulatory drift or selection against positive feedback\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:tsetse', 'E-Organism:fly']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\"\n",
            "}\n",
            "1154/1344 (1.32 seconds):\n",
            "{\n",
            "  \"id\": 1153,\n",
            "  \"input\": \"Do NER on this sentence\\nMicroevolution of Group A Streptococci In Vivo : Capturing Regulatory Networks Engaged in Sociomicrobiology , Niche Adaptation , and Hypervirulence The onset of infection and the switch from primary to secondary niches are dramatic environmental changes that not only alter bacterial transcriptional programs , but also perturb their sociomicrobiology , often driving minor subpopulations with mutant phenotypes to prevail in specific niches\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Group', 'E-Organism:Streptococci']\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1155/1344 (1.80 seconds):\n",
            "{\n",
            "  \"id\": 1154,\n",
            "  \"input\": \"Do NER on this sentence\\nHaving previously reported that M1T1 Streptococcus pyogenes become hypervirulent in mice due to selection of mutants in the covRS regulatory genes , we set out to dissect the impact of these mutations in vitro and in vivo from the impact of other adaptive events\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:M1T1', 'I-Organism:Streptococcus', 'E-Organism:pyogenes', 'S-Organism:mice']\",\n",
            "  \"response\": \"['B-Organism:Streptococcus', 'E-Organism:pyogenes', 'S-Organism:human']\"\n",
            "}\n",
            "1156/1344 (1.51 seconds):\n",
            "{\n",
            "  \"id\": 1155,\n",
            "  \"input\": \"Do NER on this sentence\\nIntroduction Group A streptococci ( GAS ) are human pathogens that infect over 700 million children and adults each year [ 1 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Group', 'E-Organism:streptococci', 'S-Organism:GAS', 'S-Organism:human']\",\n",
            "  \"response\": \"this sentence\\nIntroduction Group A streptococci ( GAS ) are human pathogens that infect over 700 million children and adults each year [ 1 ]\"\n",
            "}\n",
            "1157/1344 (1.37 seconds):\n",
            "{\n",
            "  \"id\": 1156,\n",
            "  \"input\": \"Do NER on this sentence\\nAmong the various GAS serotypes , the globally disseminated M1T1 clonal strain remains the most frequently isolated from cases of invasive and non - invasive infections [ 2 ] , [ 3 ] , and although disease severity partially depends on host genetic factors [ 4 ] , [ 5 ] , [ 6 ] , M1T1 GAS possesses unique genomic features that contribute to its evolutionary fitness [ 7 ] , [ 8 ] , [ 9 ] , [ 10 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:GAS', 'S-Organism:M1T1', 'B-Organism:M1T1', 'E-Organism:GAS']\",\n",
            "  \"response\": \"this sentence\\nAmong the various GAS serotypes , the globally disseminated M1T1 clonal strain remains the most frequently isolated from cases of invasive and non - invasive infections [ 2 ]\"\n",
            "}\n",
            "1158/1344 (1.72 seconds):\n",
            "{\n",
            "  \"id\": 1157,\n",
            "  \"input\": \"Do NER on this sentence\\nWe previously reported that virulent representatives of M1T1 GAS , with the phenotype SpeBhi / SpeA - / Sda1low , irreversibly switch to the hypervirulent SpeB - / SpeA + / Sda1high phenotype after > / = 3 days in vivo [ 12 ] , [ 16 ] and that the parent phenotype vanishes by day 7 post - infection [ 16 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:M1T1', 'E-Organism:GAS']\",\n",
            "  \"response\": \"this sentence\\nWe previously reported that virulent representatives of M1T1 GAS , with the phenotype SpeBhi / SpeA - / Sda1low , irreversibly switch to the hypervirulent SpeB - / SpeA + / Sda1high phenotype after > / = 3 days in vivo [ 12 ]\"\n",
            "}\n",
            "1159/1344 (1.52 seconds):\n",
            "{\n",
            "  \"id\": 1158,\n",
            "  \"input\": \"Do NER on this sentence\\nCovRS is a two - component regulatory system , in which CovS transduces external signals [ 17 ] , [ 18 ] to CovR , which in turn represses the transcription of several group A streptococcal ( GAS ) virulence gene sets , including the capsule synthesis operon ( hasABC ) , the streptolysin S operon ( SLS or sagA - H ) , and the streptokinase gene ( ska ) [ 19 ] , [ 20 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:group', 'E-Organism:streptococcal', 'S-Organism:GAS']\",\n",
            "  \"response\": \"this sentence\\nCovRS is a two - component regulatory system , in which CovS transduces external signals [ 17 ]\"\n",
            "}\n",
            "1160/1344 (0.81 seconds):\n",
            "{\n",
            "  \"id\": 1159,\n",
            "  \"input\": \"Do NER on this sentence\\nSuch comprehensive analyses highlighted the behavior of genomic subsystems that may be involved directly or indirectly in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1161/1344 (1.50 seconds):\n",
            "{\n",
            "  \"id\": 1160,\n",
            "  \"input\": \"Do NER on this sentence\\nResults Phenotype Microarrays show no major nutritional or metabolic differences between wild type and animal - passaged , hypervirulent M1T1 GAS We used Biolog ( R ) Phenotype Microarrays ( PM ) to screen 1900 different growth conditions , including a large set of different carbon and nitrogen sources , pH values and salt concentrations , as well as different concentrations of various antimicrobial agents ( Fig\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:M1T1', 'E-Organism:GAS']\",\n",
            "  \"response\": \"['B-Organism:GAS', 'E-Organism:wild type']\"\n",
            "}\n",
            "1162/1344 (2.15 seconds):\n",
            "{\n",
            "  \"id\": 1161,\n",
            "  \"input\": \"Do NER on this sentence\\nS1 ) , and found that both WT and AP GAS have similar growth requirements in vitro with minor differences\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:AP', 'E-Organism:GAS']\",\n",
            "  \"response\": \"['B-Organism:invasive', 'E-Organism:GAS', 'B-Organism:non', 'I-Organism:invasive', 'E-Organism:GAS']\"\n",
            "}\n",
            "1163/1344 (1.96 seconds):\n",
            "{\n",
            "  \"id\": 1162,\n",
            "  \"input\": \"Do NER on this sentence\\nmutation - selection events Several studies , including ours , have examined phenotypic , proteomic , and transcriptional differences between WT and animal - passaged M1T1 GAS strains [ For example , 11 , 12 , 13 , 16 , 26 , 27 , 37 ] , but none attempted to capture the dynamic changes in the bacterial population in vivo\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:M1T1', 'E-Organism:GAS']\",\n",
            "  \"response\": \"this sentence\\nmutation - selection events Several studies , including ours , have examined phenotypic , proteomic , and transcriptional differences between WT and animal - passaged M1T1 GAS strains [ For example , 11 , 12 , 13 , 16 , 26 , 27 , 37 ]\"\n",
            "}\n",
            "1164/1344 (1.46 seconds):\n",
            "{\n",
            "  \"id\": 1163,\n",
            "  \"input\": \"Do NER on this sentence\\nMost genes within the Trx locus are downregulated in AP GAS relative to WT , but upregulated in the WT bacteria in vivo ( Fig\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:AP', 'E-Organism:GAS']\",\n",
            "  \"response\": \"['B-Organism:GAS', 'E-Organism:5448']\"\n",
            "}\n",
            "1165/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 1164,\n",
            "  \"input\": \"Do NER on this sentence\\nThe bigger picture : niche adaptation and the evolution of hypervirulence in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1166/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 1165,\n",
            "  \"input\": \"Do NER on this sentence\\nInactivation of DNA - Binding Response Regulator Sak189 Abrogates beta - Antigen Expression and Affects Virulence of Streptococcus agalactiae Background Streptococcus agalactiae is able to colonize numerous tissues employing different mechanisms of gene regulation , particularly via two - component regulatory systems\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Streptococcus', 'E-Organism:agalactiae', 'B-Organism:Streptococcus', 'E-Organism:agalactiae']\",\n",
            "  \"response\": \"['B-Organism:Streptococcus', 'E-Organism:agalactiae']\"\n",
            "}\n",
            "1167/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1166,\n",
            "  \"input\": \"Do NER on this sentence\\nIn S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1168/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1167,\n",
            "  \"input\": \"Do NER on this sentence\\nFinally , insertional inactivation of sak189 gene , but not sak188 gene , significantly affected virulent properties of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1169/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1168,\n",
            "  \"input\": \"Do NER on this sentence\\nIt also controls the virulent properties of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1170/1344 (1.12 seconds):\n",
            "{\n",
            "  \"id\": 1169,\n",
            "  \"input\": \"Do NER on this sentence\\nIntroduction Streptococcus agalactiae ( group B streptococcus ) is an important cause of neonatal invasive infections and variety of diseases in human and animals [ 1 ] , [ 2 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Streptococcus', 'E-Organism:agalactiae', 'B-Organism:group', 'E-Organism:streptococcus', 'S-Organism:human']\",\n",
            "  \"response\": \"this sentence\\nIntroduction Streptococcus agalactiae ( group B streptococcus ) is an important cause of neonatal invasive infections and variety of diseases in human and animals [ 1 ]\"\n",
            "}\n",
            "1171/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1170,\n",
            "  \"input\": \"Do NER on this sentence\\nIn contrast to other pathogenic gram - positive cocci such as S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1172/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 1171,\n",
            "  \"input\": \"Do NER on this sentence\\npneumoniae and S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:pneumoniae']\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "1173/1344 (0.71 seconds):\n",
            "{\n",
            "  \"id\": 1172,\n",
            "  \"input\": \"Do NER on this sentence\\npyogenes , S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:pyogenes']\",\n",
            "  \"response\": \"['E-Organism:pyogenes']\"\n",
            "}\n",
            "1174/1344 (2.06 seconds):\n",
            "{\n",
            "  \"id\": 1173,\n",
            "  \"input\": \"Do NER on this sentence\\nFor example , 20 TCSs have been identified after the complete genome sequencing in the strain NEM316 [ 6 ] , and 17 TCSs - in the strain 2603 V / R [ 7 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:NEM316', 'B-Organism:strain', 'I-Organism:2603']\",\n",
            "  \"response\": \"this sentence\\nFor example , 20 TCSs have been identified after the complete genome sequencing in the strain NEM316 [ 6 ]\"\n",
            "}\n",
            "1175/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1174,\n",
            "  \"input\": \"Do NER on this sentence\\nS\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1176/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 1175,\n",
            "  \"input\": \"Do NER on this sentence\\nagalactiae CsrRS / CovRS demonstrated high similarity with CsrRS / CovRS of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:agalactiae']\",\n",
            "  \"response\": \"['E-Organism:agalactiae']\"\n",
            "}\n",
            "1177/1344 (0.85 seconds):\n",
            "{\n",
            "  \"id\": 1176,\n",
            "  \"input\": \"Do NER on this sentence\\nInactivation of covR in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1178/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1177,\n",
            "  \"input\": \"Do NER on this sentence\\nTranscriptome analysis revealed the CovRS core - regulon which consisted of 39 genes including genes of the stress response , the genes important for adhesion of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1179/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1178,\n",
            "  \"input\": \"Do NER on this sentence\\nDltR / DltS of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1180/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1179,\n",
            "  \"input\": \"Do NER on this sentence\\nThe role of DltR / DltS in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1181/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1180,\n",
            "  \"input\": \"Do NER on this sentence\\nRgfA / RgfC controls adhesion of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1182/1344 (0.22 seconds):\n",
            "{\n",
            "  \"id\": 1181,\n",
            "  \"input\": \"Do NER on this sentence\\nRecently , putative two - component regulatory system genes encoding for sensor histidine kinase ( sak188 ) and DNA - binding response regulator ( sak189 ) were identified in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nRecently , putative two - component regulatory system genes encoding for sensor histidine kinase ( sak188 ) and DNA - binding response regulator ( sak189 ) were identified in S\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "1183/1344 (1.95 seconds):\n",
            "{\n",
            "  \"id\": 1182,\n",
            "  \"input\": \"Do NER on this sentence\\nagalactiae strains 98 - D60C and A909 [ 13 ] , [ 14 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:agalactiae', 'I-Organism:strains', 'I-Organism:98', 'E-Organism:D60C', 'S-Organism:A909']\",\n",
            "  \"response\": \"this sentence\\nagalactiae strains 98 - D60C and A909 [ 13 ]\"\n",
            "}\n",
            "1184/1344 (1.41 seconds):\n",
            "{\n",
            "  \"id\": 1183,\n",
            "  \"input\": \"Do NER on this sentence\\nSak188 and Sak189 proteins revealed 78 % and 83 % similarity with sensor histidine kinase HK06 and DNA - binding response regulator RR06 of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\"\n",
            "}\n",
            "1185/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1184,\n",
            "  \"input\": \"Do NER on this sentence\\nIn S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1186/1344 (0.67 seconds):\n",
            "{\n",
            "  \"id\": 1185,\n",
            "  \"input\": \"Do NER on this sentence\\npneumoniae genome , the hk06 and rr06 genes are adjacent to cbpA gene encoding choline binding protein , CbpA , a surface - exposed protective antigen and virulence factor of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:pneumoniae']\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "1187/1344 (0.80 seconds):\n",
            "{\n",
            "  \"id\": 1186,\n",
            "  \"input\": \"Do NER on this sentence\\nInactivation of RR06 in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1188/1344 (0.31 seconds):\n",
            "{\n",
            "  \"id\": 1187,\n",
            "  \"input\": \"Do NER on this sentence\\nGiven the certain similarity in the sequences , functions and relative locations of Sak188 / Sak189 and HK06 / RR06 , and beta - antigen and CbpA , we suggested that Sak188 / Sak189 TCS affected beta - antigen expression and virulent properties of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1189/1344 (1.50 seconds):\n",
            "{\n",
            "  \"id\": 1188,\n",
            "  \"input\": \"Do NER on this sentence\\nagalactiae in the same manner as HK06 / RR06 affected CbpA in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:agalactiae']\",\n",
            "  \"response\": \"['I-Organism:agalactiae', 'I-Organism:strain', 'E-Organism:MT102']\"\n",
            "}\n",
            "1190/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1189,\n",
            "  \"input\": \"Do NER on this sentence\\nThe goal of the present study was to investigate the function of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1191/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1190,\n",
            "  \"input\": \"Do NER on this sentence\\nResults An occurrence of bac gene , sak188 , sak189 TCS genes and erythromycin resistance among S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1192/1344 (1.50 seconds):\n",
            "{\n",
            "  \"id\": 1191,\n",
            "  \"input\": \"Do NER on this sentence\\nagalactiae strains A total of 239 S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:agalactiae', 'E-Organism:strains']\",\n",
            "  \"response\": \"['I-Organism:agalactiae', 'I-Organism:strain', 'E-Organism:W14']\"\n",
            "}\n",
            "1193/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1192,\n",
            "  \"input\": \"Do NER on this sentence\\nGenomic architecture of DNA fragments containing sak188 , sak189 and bac genes in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1194/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 1193,\n",
            "  \"input\": \"Do NER on this sentence\\nagalactiae [ 13 ] , [ 14 ] , and rr06 , hk06 and cbpA genes in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:agalactiae']\",\n",
            "  \"response\": \"this sentence\\nagalactiae [ 13 ]\"\n",
            "}\n",
            "1195/1344 (1.51 seconds):\n",
            "{\n",
            "  \"id\": 1194,\n",
            "  \"input\": \"Do NER on this sentence\\nAs result , most of human bac gene positive strains showed resistance or intermediate resistance to Em in the standard diffusion disk test , that is in agreement with previous observations demonstrating an increase in Em resistance in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:human']\",\n",
            "  \"response\": \"['B-Organism:invasive', 'E-Organism:GAS', 'S-Organism:GAS']\"\n",
            "}\n",
            "1196/1344 (1.41 seconds):\n",
            "{\n",
            "  \"id\": 1195,\n",
            "  \"input\": \"Do NER on this sentence\\nOne of the strains , 168 / 00 , which demonstrated the large diameter of growth inhibition zone , was selected for further study\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:168', 'E-Organism:00']\",\n",
            "  \"response\": \"['B-Organism', 'E-Organism:GAS', 'S-Organism:GAS']\"\n",
            "}\n",
            "1197/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1196,\n",
            "  \"input\": \"Do NER on this sentence\\nConstruction and characterization of mutant strains To investigate a functional role of the two - component regulatory system Sak188 / Sak189 in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1198/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1197,\n",
            "  \"input\": \"Do NER on this sentence\\nSubsequently , EcoRI fragments were cloned into suicide vector pVA891 - 2 , and the recombinant plasmids named as pHK and pRR , respectively , were used to transform S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1199/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1198,\n",
            "  \"input\": \"Do NER on this sentence\\nIn S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1200/1344 (1.41 seconds):\n",
            "{\n",
            "  \"id\": 1199,\n",
            "  \"input\": \"Do NER on this sentence\\nIn this study , we investigated the functional role of Sak188 / Sak189 two - component system in regulation of the expression of beta - antigen , an important virulence factor of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1201/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1200,\n",
            "  \"input\": \"Do NER on this sentence\\nFinally , insertional inactivation of sak189 gene , but not sak188 gene , significantly affects virulent properties of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1202/1344 (0.46 seconds):\n",
            "{\n",
            "  \"id\": 1201,\n",
            "  \"input\": \"Do NER on this sentence\\nVariety of publications are currently available regarding the occurrence of bac gene among S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1203/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1202,\n",
            "  \"input\": \"Do NER on this sentence\\nIn present study , the sak188 and sak189 genes of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1204/1344 (1.41 seconds):\n",
            "{\n",
            "  \"id\": 1203,\n",
            "  \"input\": \"Do NER on this sentence\\nComparative analysis of Sak188 and Sak189 amino acid sequences in the wild - type strain 168 / 00 and deduced protein sequences in mutant strains predicted the replacement of 95 amino acids at the C - terminal end of Sak188 protein by 191 amino acids , and replacement of 51 amino acids at the C - terminal end of Sak189 protein by 15 amino acids ( Fig\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:168', 'E-Organism:00']\",\n",
            "  \"response\": \"['S-Organism:168', 'B-Organism:M1T1', 'E-Organism:GAS']\"\n",
            "}\n",
            "1205/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1204,\n",
            "  \"input\": \"Do NER on this sentence\\nIn S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1206/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1205,\n",
            "  \"input\": \"Do NER on this sentence\\nThus , we can not rule out the possibility of a cross - talk between Sak189 response regulator and sensor kinase ( s ) in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1207/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1206,\n",
            "  \"input\": \"Do NER on this sentence\\nFor example , in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1208/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1207,\n",
            "  \"input\": \"Do NER on this sentence\\nIntraperitoneal infection of laboratory mice demonstrated that Sak189 DNA - binding response regulatory protein is important for virulent properties of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:mice']\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1209/1344 (4.09 seconds):\n",
            "{\n",
            "  \"id\": 1208,\n",
            "  \"input\": \"Do NER on this sentence\\nIt could be predicted given that inactivation of other S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nIt could be predicted given that inactivation of other S\\nReply in key value pair as Tag and Entity. and pathogenicity islands (PAI ) [ 7 ]\"\n",
            "}\n",
            "1210/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 1209,\n",
            "  \"input\": \"Do NER on this sentence\\nThus , we suggest that beta - antigen located on the bacterial surface can interfere with the disease process and play a role of anti - virulent factor ( at least , in mice ) similarly to hyaluronate lyase of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:mice']\",\n",
            "  \"response\": \"['B-Organism:group', 'E-Organism:streptococci']\"\n",
            "}\n",
            "1211/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1210,\n",
            "  \"input\": \"Do NER on this sentence\\nIn conclusion , the data presented here characterize the functional role of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1212/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1211,\n",
            "  \"input\": \"Do NER on this sentence\\nInactivation of histidine kinase Sak188 does not affect transcription of bac gene and expression of beta - antigen , that suggests the cross - talk between Sak189 and other sensor kinase ( s ) in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1213/1344 (1.76 seconds):\n",
            "{\n",
            "  \"id\": 1212,\n",
            "  \"input\": \"Do NER on this sentence\\nAcid pH activation of the PmrA / PmrB two - component regulatory system of Salmonella enterica Acid pH often triggers changes in gene expression\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\"\n",
            "}\n",
            "1214/1344 (1.61 seconds):\n",
            "{\n",
            "  \"id\": 1213,\n",
            "  \"input\": \"Do NER on this sentence\\nThe Gram - negative pathogen Salmonella enterica serovar Typhimurium experiences a number of acidic environments both inside and outside animal hosts\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'I-Organism:enterica', 'I-Organism:serovar', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "1215/1344 (1.71 seconds):\n",
            "{\n",
            "  \"id\": 1214,\n",
            "  \"input\": \"Do NER on this sentence\\nThe Gram - negative bacterium Salmonella enterica serovar Typhimurium , for example , experiences a number of acidic environments both inside and outside animal hosts\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'I-Organism:enterica', 'I-Organism:serovar', 'E-Organism:Typhimurium']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "1216/1344 (0.30 seconds):\n",
            "{\n",
            "  \"id\": 1215,\n",
            "  \"input\": \"Do NER on this sentence\\nThe periplasmic histidine and glutamates are conserved in the PmrB periplasmic domain of other enteric species , raising the possibility that the signalling pathway described in this article may be operating in other organisms in addition to S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1217/1344 (0.21 seconds):\n",
            "{\n",
            "  \"id\": 1216,\n",
            "  \"input\": \"Do NER on this sentence\\n( 1981 ) determined that the intracellular pH in Escherichia coli cells was 7\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Escherichia', 'E-Organism:coli']\",\n",
            "  \"response\": \"this sentence\\n( 1981 ) determined that the intracellular pH in Escherichia coli cells was 7\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "1218/1344 (1.05 seconds):\n",
            "{\n",
            "  \"id\": 1217,\n",
            "  \"input\": \"Do NER on this sentence\\nThe CadC protein of E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1219/1344 (1.11 seconds):\n",
            "{\n",
            "  \"id\": 1218,\n",
            "  \"input\": \"Do NER on this sentence\\nThis may be particularly important for Salmonella living in soil due to the fact that the antimicrobial peptide polymyxin B is produced by the soil bacterium Paenibacillus polymyxa ( Paulus and Gray , 1964 ) and because the solubility of metals such as Fe3 + increases in acid pH\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Salmonella', 'B-Organism:Paenibacillus', 'E-Organism:polymyxa']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\"\n",
            "}\n",
            "1220/1344 (1.47 seconds):\n",
            "{\n",
            "  \"id\": 1219,\n",
            "  \"input\": \"Do NER on this sentence\\nINTRODUCTION Salmonella enterica serovar Typhimurium is a facultative intracellular pathogen that initiates disease , which is normally limited to gastroenteritis in humans\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'I-Organism:enterica', 'I-Organism:serovar', 'E-Organism:Typhimurium', 'S-Organism:humans']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "1221/1344 (0.40 seconds):\n",
            "{\n",
            "  \"id\": 1220,\n",
            "  \"input\": \"Do NER on this sentence\\nRESULTS Mutation of the mlc gene reduces Salmonella invasiveness The process of invasion into non - phagocytic epithelial cells , which is known to be mediated by proteins secreted by the SPI1 - encoded type III machinery ( 3 ) , is an important initial step in the pathogenesis of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Salmonella']\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1222/1344 (1.36 seconds):\n",
            "{\n",
            "  \"id\": 1221,\n",
            "  \"input\": \"Do NER on this sentence\\nWe examined the ability of an mlc mutant , SR1304 , to invade cultured HEp - 2 and HeLa epithelial cells\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:SR1304', 'B-Organism:HEp']\",\n",
            "  \"response\": \"['B-Organism:invasive', 'E-Organism:GAS', 'S-Organism:human']\"\n",
            "}\n",
            "1223/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1222,\n",
            "  \"input\": \"Do NER on this sentence\\nS\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1224/1344 (1.90 seconds):\n",
            "{\n",
            "  \"id\": 1223,\n",
            "  \"input\": \"Do NER on this sentence\\nWhen these cells were grown in static cultures to exponential phase , SL1344 - pMAB69 expressed 28\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:SL1344', 'E-Organism:pMAB69']\",\n",
            "  \"response\": \"['B-Organism:tubercle', 'E-Organism:bacillus', 'S-Organism:human', 'S-Organism:mouse']\"\n",
            "}\n",
            "1225/1344 (0.81 seconds):\n",
            "{\n",
            "  \"id\": 1224,\n",
            "  \"input\": \"Do NER on this sentence\\n5 Miller units of beta - galactosidase , whereas SR1304 - pMAB69 expressed 49\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:SR1304', 'E-Organism:pMAB69']\",\n",
            "  \"response\": \"['B-Organism:Symbioflor']\"\n",
            "}\n",
            "1226/1344 (2.21 seconds):\n",
            "{\n",
            "  \"id\": 1225,\n",
            "  \"input\": \"Do NER on this sentence\\nThis difference could be attributed to differences in the experimental model conditions ; in our study , total RNA from a Salmonella strain was used in the primer extension assay , whereas Baxter and Jones ( 18 ) used total RNA prepared from E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:Salmonella']\",\n",
            "  \"response\": \"['B-Organism:Sodalis', 'E-Organism:glossinidius', 'B-Organism:Salmonella', 'E-Organism:Typhimurium']\"\n",
            "}\n",
            "1227/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 1226,\n",
            "  \"input\": \"Do NER on this sentence\\nThe Mlc - His6 protein was produced in E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1228/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1227,\n",
            "  \"input\": \"Do NER on this sentence\\nThe addition of excess glucose to the culture medium results in the reduction of cell association by S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1229/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1228,\n",
            "  \"input\": \"Do NER on this sentence\\nIt is known that S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1230/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1229,\n",
            "  \"input\": \"Do NER on this sentence\\nS\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1231/1344 (1.32 seconds):\n",
            "{\n",
            "  \"id\": 1230,\n",
            "  \"input\": \"Do NER on this sentence\\nSalK / SalR , a Two - Component Signal Transduction System , Is Essential for Full Virulence of Highly Invasive Streptococcus suis Serotype 2 Background Streptococcus suis serotype 2 ( S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Streptococcus', 'I-Organism:suis', 'I-Organism:Serotype', 'B-Organism:Streptococcus', 'I-Organism:suis', 'I-Organism:serotype']\",\n",
            "  \"response\": \"['B-Organism:Streptococcus', 'E-Organism:serotype']\"\n",
            "}\n",
            "1232/1344 (1.46 seconds):\n",
            "{\n",
            "  \"id\": 1231,\n",
            "  \"input\": \"Do NER on this sentence\\nsuis 2 , SS2 ) has evolved into a highly infectious entity , which caused the two recent large - scale outbreaks of human SS2 epidemic in China , and is characterized by a toxic shock - like syndrome\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:suis', 'S-Organism:SS2', 'S-Organism:human', 'S-Organism:SS2']\",\n",
            "  \"response\": \"['I-Organism:suis', 'I-Organism:strain', 'E-Organism:V583']\"\n",
            "}\n",
            "1233/1344 (1.10 seconds):\n",
            "{\n",
            "  \"id\": 1232,\n",
            "  \"input\": \"Do NER on this sentence\\nFurther bioinformatics analysis revealed a unique two - component signal transduction system ( TCSTS ) located in the candidate 89K PAI , which is orthologous to the SalK / SalR regulatory system of Streptococcus salivarius\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Streptococcus', 'E-Organism:salivarius']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\"\n",
            "}\n",
            "1234/1344 (2.00 seconds):\n",
            "{\n",
            "  \"id\": 1233,\n",
            "  \"input\": \"Do NER on this sentence\\nIntroduction Streptococcus suis ( S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Streptococcus', 'E-Organism:suis']\",\n",
            "  \"response\": \"['B-Organism:Streptococcus', 'E-Organism:subspecies', 'B-Organism:Streptococcus', 'E-Organism:subspecies']\"\n",
            "}\n",
            "1235/1344 (0.22 seconds):\n",
            "{\n",
            "  \"id\": 1234,\n",
            "  \"input\": \"Do NER on this sentence\\nAmong the known 35 serotypes [ 1 ] , [ 2 ] , S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nAmong the known 35 serotypes [ 1 ]\"\n",
            "}\n",
            "1236/1344 (1.55 seconds):\n",
            "{\n",
            "  \"id\": 1235,\n",
            "  \"input\": \"Do NER on this sentence\\nsuis serotype 2 ( S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:suis', 'I-Organism:serotype']\",\n",
            "  \"response\": \"['I-Organism:enterocolitica', 'I-Organism:serotype', 'E-Organism:GAS']\"\n",
            "}\n",
            "1237/1344 (1.06 seconds):\n",
            "{\n",
            "  \"id\": 1236,\n",
            "  \"input\": \"Do NER on this sentence\\nsuis 2 or SS2 ) is the most virulent and the most frequently isolated serotype\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:suis', 'S-Organism:SS2']\",\n",
            "  \"response\": \"['B-Organism:group', 'E-Organism:streptococci']\"\n",
            "}\n",
            "1238/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1237,\n",
            "  \"input\": \"Do NER on this sentence\\nFurther bioinformatics analysis of the 89K island revealed a distinct two - component signal transduction system ( TCSTS ) encoded therein appears to be orthologous to the SalK / SalR system of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1239/1344 (0.66 seconds):\n",
            "{\n",
            "  \"id\": 1238,\n",
            "  \"input\": \"Do NER on this sentence\\nResults Discovery and Characterization of SalK / SalR The newly decoded genomic sequence of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1240/1344 (1.61 seconds):\n",
            "{\n",
            "  \"id\": 1239,\n",
            "  \"input\": \"Do NER on this sentence\\nsuis 05ZYH33 makes it possible to systematically investigate the genetic basis of streptococcal pathogenicity\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:suis', 'E-Organism:05ZYH33']\",\n",
            "  \"response\": \"['I-Organism:streptococcal', 'I-Organism:strain', 'E-Organism:GAS']\"\n",
            "}\n",
            "1241/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 1240,\n",
            "  \"input\": \"Do NER on this sentence\\nOn the negative strand of 89K , peptides encoded by 05SSU0944 and 05SSU0943 exhibit 27 % and 41 % amino acid sequence identity with the SalK and the SalR proteins of Streptococcus salivarius , respectively , forming a TCSTS belonging to the Nar / LuxR family\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Streptococcus', 'E-Organism:salivarius']\",\n",
            "  \"response\": \"['B-Organism:Streptococcus', 'E-Organism:salivarius']\"\n",
            "}\n",
            "1242/1344 (1.65 seconds):\n",
            "{\n",
            "  \"id\": 1241,\n",
            "  \"input\": \"Do NER on this sentence\\nDownstream of salKR , the 05SSU0942 , 05SSU0941 , 05SSU0940 , 05SSU0939 , 05SSU0938 genes were predicted to encode peptides with significant identities to the orthologues of Streptococcus agalactiae\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Streptococcus', 'E-Organism:agalactiae']\",\n",
            "  \"response\": \"['B-Organism:Streptococcus', 'E-Organism:agalactiae']\"\n",
            "}\n",
            "1243/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1242,\n",
            "  \"input\": \"Do NER on this sentence\\nTranscriptional Analysis of the salKR Locus and its Flanking Sequences To confirm the predicted transcripts of salKR and the flanking genes , RNA extracted from S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1244/1344 (1.41 seconds):\n",
            "{\n",
            "  \"id\": 1243,\n",
            "  \"input\": \"Do NER on this sentence\\nsuis 05ZYH33 cells was subjected to RT - PCR analysis by using primers amplifying each intergenic region of salKR and the flanking genes\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:suis', 'E-Organism:05ZYH33']\",\n",
            "  \"response\": \"['B-Organism:invasive', 'E-Organism:GAS', 'S-Organism:GAS']\"\n",
            "}\n",
            "1245/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 1244,\n",
            "  \"input\": \"Do NER on this sentence\\nFurthermore , orotidine - 5 - phosphate decarboxylase ( 05SSU1009 ) , a newly identified pathogenesis - related protein in Vibrio vulnificus [ 31 ] , was also found to be repressed greatly in DeltasalKR\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Vibrio', 'E-Organism:vulnificus']\",\n",
            "  \"response\": \"this sentence\\nFurthermore , orotidine - 5 - phosphate decarboxylase ( 05SSU1009 ) , a newly identified pathogenesis - related protein in Vibrio vulnificus [ 31 ]\"\n",
            "}\n",
            "1246/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1245,\n",
            "  \"input\": \"Do NER on this sentence\\n[ 32 ] has shown that mutation of phosphotransferase system ( PST ) in extraintestinal pathogenic E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:extraintestinal', 'I-Organism:pathogenic']\",\n",
            "  \"response\": \"this sentence\\n[ 32 ]\"\n",
            "}\n",
            "1247/1344 (0.95 seconds):\n",
            "{\n",
            "  \"id\": 1246,\n",
            "  \"input\": \"Do NER on this sentence\\nSecondly , some of the TCSTSs of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1248/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1247,\n",
            "  \"input\": \"Do NER on this sentence\\nPrior to this study , SalK / SalR is only known to be associated with the production of a kind of lantibiotic peptide named salivaricin A ( SalA ) in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1249/1344 (1.61 seconds):\n",
            "{\n",
            "  \"id\": 1248,\n",
            "  \"input\": \"Do NER on this sentence\\nRpoE fine tunes expression of a subset of SsrB - regulated virulence factors in Salmonella enterica serovar Typhimurium Background The survival of Salmonella enterica within the intracellular host niche requires highly co - ordinated expression of virulence effectors predominantly regulated by the SsrAB two - component regulatory system\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'I-Organism:enterica', 'I-Organism:serovar', 'E-Organism:Typhimurium', 'B-Organism:Salmonella', 'E-Organism:enterica']\",\n",
            "  \"response\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\"\n",
            "}\n",
            "1250/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1249,\n",
            "  \"input\": \"Do NER on this sentence\\nS\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1251/1344 (1.01 seconds):\n",
            "{\n",
            "  \"id\": 1250,\n",
            "  \"input\": \"Do NER on this sentence\\nenterica serovar Typhimurium mutants lacking the ssrAB genes are avirulent in mice , highlighting the importance of this regulatory system in vivo\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:enterica', 'I-Organism:serovar', 'I-Organism:Typhimurium', 'E-Organism:mutants', 'S-Organism:mice']\",\n",
            "  \"response\": \"['B-Organism:enterica', 'E-Organism:serovar']\"\n",
            "}\n",
            "1252/1344 (1.20 seconds):\n",
            "{\n",
            "  \"id\": 1251,\n",
            "  \"input\": \"Do NER on this sentence\\nBackground Salmonella enterica are enteric pathogens that acquired a type III secretion system ( T3SS ) through horizontal gene transfer of a genomic island termed Salmonella Pathogenicity Island 2 ( SPI - 2 ) [ 1 , 2 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:enterica']\",\n",
            "  \"response\": \"this sentence\\nBackground Salmonella enterica are enteric pathogens that acquired a type III secretion system ( T3SS ) through horizontal gene transfer of a genomic island termed Salmonella Pathogenicity Island 2 ( SPI - 2 ) [ 1 , 2 ]\"\n",
            "}\n",
            "1253/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1252,\n",
            "  \"input\": \"Do NER on this sentence\\nSsrB - regulated translocated effectors protect S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1254/1344 (1.26 seconds):\n",
            "{\n",
            "  \"id\": 1253,\n",
            "  \"input\": \"Do NER on this sentence\\nDeletion of rseA in Yersinia pseudotuberculosis causes elevated virulence effector synthesis and secretion [ 18 ] , establishing links between alternative sigma factors and virulence - specific regulators\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Yersinia', 'E-Organism:pseudotuberculosis']\",\n",
            "  \"response\": \"this sentence\\nDeletion of rseA in Yersinia pseudotuberculosis causes elevated virulence effector synthesis and secretion [ 18 ]\"\n",
            "}\n",
            "1255/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 1254,\n",
            "  \"input\": \"Do NER on this sentence\\nTo start , we first screened four alternative sigma factor mutants of S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1256/1344 (0.22 seconds):\n",
            "{\n",
            "  \"id\": 1255,\n",
            "  \"input\": \"Do NER on this sentence\\nAn unmarked in - frame deletion of rpoE was then generated in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nAn unmarked in - frame deletion of rpoE was then generated in S\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "1257/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 1256,\n",
            "  \"input\": \"Do NER on this sentence\\nTyphimurium strain SL1344 and we verified that this in - frame deletion had the same effect on SseB as the rpoE : : cat mutant used previously ( Figure 1B )\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Typhimurium', 'I-Organism:strain', 'E-Organism:SL1344']\",\n",
            "  \"response\": \"['B-Organism:Typhimurium', 'E-Organism:strain']\"\n",
            "}\n",
            "1258/1344 (0.22 seconds):\n",
            "{\n",
            "  \"id\": 1257,\n",
            "  \"input\": \"Do NER on this sentence\\nTogether these data establish a role for RpoE in the fine - tuning of virulence gene expression in S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nTogether these data establish a role for RpoE in the fine - tuning of virulence gene expression in S\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "1259/1344 (2.54 seconds):\n",
            "{\n",
            "  \"id\": 1258,\n",
            "  \"input\": \"Do NER on this sentence\\nCaenorhabditis elegans Semi - Automated Liquid Screen Reveals a Specialized Role for the Chemotaxis Gene cheB2 in Pseudomonas aeruginosa Virulence Pseudomonas aeruginosa is an opportunistic human pathogen that causes infections in a variety of animal and plant hosts\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Caenorhabditis', 'E-Organism:elegans', 'B-Organism:Pseudomonas', 'E-Organism:aeruginosa', 'B-Organism:Pseudomonas', 'E-Organism:aeruginosa', 'S-Organism:human']\",\n",
            "  \"response\": \"['B-Organism:Enterobacter', 'E-Organism:carotovora', 'S-Organism:enterobacter', 'B-Organism:Eccilococccus', 'E-Organism:carotovora']\"\n",
            "}\n",
            "1260/1344 (1.16 seconds):\n",
            "{\n",
            "  \"id\": 1259,\n",
            "  \"input\": \"Do NER on this sentence\\nCaenorhabditis elegans is a simple model with which one can identify bacterial virulence genes\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Caenorhabditis', 'E-Organism:elegans']\",\n",
            "  \"response\": \"['B-Organism:Candida', 'E-Organism:albicans']\"\n",
            "}\n",
            "1261/1344 (1.40 seconds):\n",
            "{\n",
            "  \"id\": 1260,\n",
            "  \"input\": \"Do NER on this sentence\\nPrevious studies with C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1262/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 1261,\n",
            "  \"input\": \"Do NER on this sentence\\nelegans have shown that depending on the growth medium , P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:elegans']\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "1263/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1262,\n",
            "  \"input\": \"Do NER on this sentence\\nWe screened a 2 , 200 - member STM mutant library generated in a cystic fibrosis airway P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1264/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1263,\n",
            "  \"input\": \"Do NER on this sentence\\nTwelve mutants were isolated each showing at least 70 % attenuation in C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1265/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1264,\n",
            "  \"input\": \"Do NER on this sentence\\nIn P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1266/1344 (0.91 seconds):\n",
            "{\n",
            "  \"id\": 1265,\n",
            "  \"input\": \"Do NER on this sentence\\nWe show that , in contrast to the cheB2 mutant , a cheB1 mutant is not attenuated for virulence in C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism', 'E-Organism:GAS']\"\n",
            "}\n",
            "1267/1344 (0.25 seconds):\n",
            "{\n",
            "  \"id\": 1266,\n",
            "  \"input\": \"Do NER on this sentence\\nWe conclude that the virulence defect of the cheB2 mutant is not linked with a global motility defect but that instead the cheB2 gene is involved in a specific chemotactic response , which takes place during infection and is required for P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nWe conclude that the virulence defect of the cheB2 mutant is not linked with a global motility defect but that instead the cheB2 gene is involved in a specific chemotactic response , which takes place during infection and is required for P\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "1268/1344 (1.66 seconds):\n",
            "{\n",
            "  \"id\": 1267,\n",
            "  \"input\": \"Do NER on this sentence\\nIntroduction The ubiquitous Gram - negative bacterium Pseudomonas aeruginosa is an opportunistic pathogen able to infect a broad range of animals and plants hosts including humans\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Pseudomonas', 'E-Organism:aeruginosa', 'S-Organism:humans']\",\n",
            "  \"response\": \"['B-Organism:Pseudomonas', 'E-Organism:aureus', 'S-Organism:human']\"\n",
            "}\n",
            "1269/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1268,\n",
            "  \"input\": \"Do NER on this sentence\\nIn the course of infection , P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1270/1344 (1.06 seconds):\n",
            "{\n",
            "  \"id\": 1269,\n",
            "  \"input\": \"Do NER on this sentence\\nIt has been demonstrated that the nematode Caenorhabditis elegans can be an effective model for studying virulence mechanisms used by a variety of bacterial pathogens [ 5 ] , [ 6 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Caenorhabditis', 'E-Organism:elegans']\",\n",
            "  \"response\": \"this sentence\\nIt has been demonstrated that the nematode Caenorhabditis elegans can be an effective model for studying virulence mechanisms used by a variety of bacterial pathogens [ 5 ]\"\n",
            "}\n",
            "1271/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1270,\n",
            "  \"input\": \"Do NER on this sentence\\nP\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1272/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 1271,\n",
            "  \"input\": \"Do NER on this sentence\\naeruginosa is capable of killing C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:aeruginosa']\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "1273/1344 (4.44 seconds):\n",
            "{\n",
            "  \"id\": 1272,\n",
            "  \"input\": \"Do NER on this sentence\\nWhen the P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "1274/1344 (1.60 seconds):\n",
            "{\n",
            "  \"id\": 1273,\n",
            "  \"input\": \"Do NER on this sentence\\naeruginosa strain PA14 is cultured on a high - osmolarity peptone - glucose - sorbitol medium ( PGS ) , worms succumb to intoxication termed  fast killing  , as the exposed worms die within hours [ 7 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:aeruginosa', 'I-Organism:strain', 'E-Organism:PA14']\",\n",
            "  \"response\": \"this sentence\\naeruginosa strain PA14 is cultured on a high - osmolarity peptone - glucose - sorbitol medium ( PGS ) , worms succumb to intoxication termed  fast killing  , as the exposed worms die within hours [ 7 ]\"\n",
            "}\n",
            "1275/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1274,\n",
            "  \"input\": \"Do NER on this sentence\\nThe P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1276/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 1275,\n",
            "  \"input\": \"Do NER on this sentence\\naeruginosa isolate PAO1 cultured on Brain Heart Infusion ( BHI ) agar induces worm paralysis and death within hours [ 9 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:aeruginosa', 'I-Organism:isolate', 'E-Organism:PAO1']\",\n",
            "  \"response\": \"this sentence\\naeruginosa isolate PAO1 cultured on Brain Heart Infusion ( BHI ) agar induces worm paralysis and death within hours [ 9 ]\"\n",
            "}\n",
            "1277/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1276,\n",
            "  \"input\": \"Do NER on this sentence\\nFinally , C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1278/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1277,\n",
            "  \"input\": \"Do NER on this sentence\\nIt is known that the P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1279/1344 (8.40 seconds):\n",
            "{\n",
            "  \"id\": 1278,\n",
            "  \"input\": \"Do NER on this sentence\\naeruginosa isolates PA14 and PAO1 show genomic diversity\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:aeruginosa', 'I-Organism:isolates', 'E-Organism:PA14', 'S-Organism:PAO1']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "1280/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1279,\n",
            "  \"input\": \"Do NER on this sentence\\nIn this study , we used the P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1281/1344 (1.66 seconds):\n",
            "{\n",
            "  \"id\": 1280,\n",
            "  \"input\": \"Do NER on this sentence\\naeruginosa strain TBCF10839 ( TB ) , which was isolated from a Cystic Fibrosis ( CF ) patient [ 14 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:aeruginosa', 'I-Organism:strain', 'E-Organism:TBCF10839', 'S-Organism:TB', 'S-Organism:patient']\",\n",
            "  \"response\": \"this sentence\\naeruginosa strain TBCF10839 ( TB ) , which was isolated from a Cystic Fibrosis ( CF ) patient [ 14 ]\"\n",
            "}\n",
            "1282/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1281,\n",
            "  \"input\": \"Do NER on this sentence\\nThe TB strain belongs to an abundant clonal complex in the P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:TB']\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1283/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1282,\n",
            "  \"input\": \"Do NER on this sentence\\nWe developed a high - throughput killing assay using C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1284/1344 (0.56 seconds):\n",
            "{\n",
            "  \"id\": 1283,\n",
            "  \"input\": \"Do NER on this sentence\\nelegans as a host , which will be appropriate for a systematic screen of mutant libraries of any P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:elegans']\",\n",
            "  \"response\": \"['E-Organism:elegans']\"\n",
            "}\n",
            "1285/1344 (1.25 seconds):\n",
            "{\n",
            "  \"id\": 1284,\n",
            "  \"input\": \"Do NER on this sentence\\nWe screened a TB STM ( signature tagged mutagenesis ) mutant library of 2 , 200 non - redundant clones [ 19 ] , [ 20 ] and selected a small group of mutants , significantly attenuated for virulence in C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:TB', 'I-Organism:STM', 'I-Organism:signature', 'I-Organism:tagged', 'I-Organism:mutagenesis', 'E-Organism:mutant']\",\n",
            "  \"response\": \"this sentence\\nWe screened a TB STM ( signature tagged mutagenesis ) mutant library of 2 , 200 non - redundant clones [ 19 ]\"\n",
            "}\n",
            "1286/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1285,\n",
            "  \"input\": \"Do NER on this sentence\\nBy testing these mutants in additional phenotypic assays , including adherence to epithelial cells and virulence in a mouse model , we identified a gene , cheB2 , necessary for in vivo P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:mouse']\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1287/1344 (0.65 seconds):\n",
            "{\n",
            "  \"id\": 1286,\n",
            "  \"input\": \"Do NER on this sentence\\nThe cheB2 gene belongs to the P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:cheB']\"\n",
            "}\n",
            "1288/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1287,\n",
            "  \"input\": \"Do NER on this sentence\\nP\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1289/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1288,\n",
            "  \"input\": \"Do NER on this sentence\\nThe chemotaxis gene cluster I ( PA1456 - PA1464 ) and cluster V ( PA3349 - PA3348 ) have previously been shown to be essential for chemotaxis and flagellar mobility in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1290/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1289,\n",
            "  \"input\": \"Do NER on this sentence\\nThe che2 genes from cluster II , and more particularly the cheB2 gene , were initially identified as required for an optimal chemotactic response in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1291/1344 (1.41 seconds):\n",
            "{\n",
            "  \"id\": 1290,\n",
            "  \"input\": \"Do NER on this sentence\\nIn Escherichia coli , the loss of CheB leads decreased receptor sensitivity , due to an inability to reset the MCP  s [ 30 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Escherichia', 'E-Organism:coli']\",\n",
            "  \"response\": \"this sentence\\nIn Escherichia coli , the loss of CheB leads decreased receptor sensitivity , due to an inability to reset the MCP  s [ 30 ]\"\n",
            "}\n",
            "1292/1344 (2.01 seconds):\n",
            "{\n",
            "  \"id\": 1291,\n",
            "  \"input\": \"Do NER on this sentence\\nIt has been reported previously in Salmonella typhimurium that a loss of proper chemotaxis control through a cheB mutation , leads to a  tumbly  swimming phenotype and a strong reduction in the isolates ability to invade human epithelial HEP2 cells as well as a reduction in infectivity in a mouse ligated - loop model [ 31 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Salmonella', 'E-Organism:typhimurium', 'S-Organism:human', 'S-Organism:mouse']\",\n",
            "  \"response\": \"this sentence\\nIt has been reported previously in Salmonella typhimurium that a loss of proper chemotaxis control through a cheB mutation , leads to a  tumbly  swimming phenotype and a strong reduction in the isolates ability to invade human epithelial HEP2 cells as well as a reduction in infectivity in a mouse ligated - loop model [ 31 ]\"\n",
            "}\n",
            "1293/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1292,\n",
            "  \"input\": \"Do NER on this sentence\\nResults High Throughput Assay for Nematode Killing by P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1294/1344 (0.77 seconds):\n",
            "{\n",
            "  \"id\": 1293,\n",
            "  \"input\": \"Do NER on this sentence\\naeruginosa It was reported earlier that relatively high - throughput screening procedures performed on plates could be used to identify virulence factors in various pathogens such as Staphylococcus aureus [ 32 ] , P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:aeruginosa', 'B-Organism:Staphylococcus', 'E-Organism:aureus']\",\n",
            "  \"response\": \"this sentence\\naeruginosa It was reported earlier that relatively high - throughput screening procedures performed on plates could be used to identify virulence factors in various pathogens such as Staphylococcus aureus [ 32 ]\"\n",
            "}\n",
            "1295/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 1294,\n",
            "  \"input\": \"Do NER on this sentence\\naeruginosa [ 8 ] or Serratia marcescens [ 33 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:aeruginosa', 'B-Organism:Serratia', 'E-Organism:marcescens']\",\n",
            "  \"response\": \"this sentence\\naeruginosa [ 8 ]\"\n",
            "}\n",
            "1296/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1295,\n",
            "  \"input\": \"Do NER on this sentence\\nHere , we developed a high throughput screening method , in which C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1297/1344 (1.46 seconds):\n",
            "{\n",
            "  \"id\": 1296,\n",
            "  \"input\": \"Do NER on this sentence\\nelegans killing is assessed in a liquid assay and which allows a quick selection of attenuated candidates within large transposon libraries of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:elegans']\",\n",
            "  \"response\": \"this sentence\\nelegans killing is assessed in a liquid assay and which allows a quick selection of attenuated candidates within large transposon libraries of P\\nReply in key value pair as Tag and Entity. Response of S\\\"\\nReply in key value pair as Tag and Entity.\\n\\nResponse of S\\n['E-Organism:enterocolitica']\"\n",
            "}\n",
            "1298/1344 (1.51 seconds):\n",
            "{\n",
            "  \"id\": 1297,\n",
            "  \"input\": \"Do NER on this sentence\\nWe prepared a bacterial culture medium that both allowed the worms to thrive feeding on the non - pathogenic E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:invasive', 'E-Organism:GAS', 'S-Organism:GAS']\"\n",
            "}\n",
            "1299/1344 (1.55 seconds):\n",
            "{\n",
            "  \"id\": 1298,\n",
            "  \"input\": \"Do NER on this sentence\\ncoli OP50 , and bacterial isolates being tested to have normal ( see Materials and Methods )\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:coli', 'E-Organism:OP50']\",\n",
            "  \"response\": \"['I-Organism:tuberculosis', 'I-Organism:strain', 'E-Organism:MT103']\"\n",
            "}\n",
            "1300/1344 (2.95 seconds):\n",
            "{\n",
            "  \"id\": 1299,\n",
            "  \"input\": \"Do NER on this sentence\\nUnder our assay conditions , more than 90 % of the worms grown on E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1301/1344 (1.26 seconds):\n",
            "{\n",
            "  \"id\": 1300,\n",
            "  \"input\": \"Do NER on this sentence\\nIn contrast , more than 50 % of the worms were killed within 24 h when cultured with any one of three isolates of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:Pseudomonas', 'E-Organism:aeruginosa']\"\n",
            "}\n",
            "1302/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1301,\n",
            "  \"input\": \"Do NER on this sentence\\nIn order to confirm that this assay would allow the recovery of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1303/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1302,\n",
            "  \"input\": \"Do NER on this sentence\\nC\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1304/1344 (7.32 seconds):\n",
            "{\n",
            "  \"id\": 1303,\n",
            "  \"input\": \"Do NER on this sentence\\nScreening the TB Transposon Library for Mutants Attenuated in Virulence A signature tagged mutagenesis ( STM ) library was previously generated in the virulent P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:TB']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "1305/1344 (1.51 seconds):\n",
            "{\n",
            "  \"id\": 1304,\n",
            "  \"input\": \"Do NER on this sentence\\naeruginosa CF isolate , TB , using the plasposon vector pTnModOGm [ 20 ] , [ 35 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:aeruginosa', 'I-Organism:CF', 'E-Organism:isolate', 'S-Organism:TB']\",\n",
            "  \"response\": \"this sentence\\naeruginosa CF isolate , TB , using the plasposon vector pTnModOGm [ 20 ]\"\n",
            "}\n",
            "1306/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1305,\n",
            "  \"input\": \"Do NER on this sentence\\nAfter this initial segregation , 2 , 200 clones were picked into a 96 - well plate and tested with the C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1307/1344 (7.27 seconds):\n",
            "{\n",
            "  \"id\": 1306,\n",
            "  \"input\": \"Do NER on this sentence\\nThe second matched PA4380 , which encodes a protein with 76 % similarity to Pseudomonas fluorescens ColS , a histidine kinase sensor from a two - component regulatory system\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Pseudomonas', 'E-Organism:fluorescens']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "1308/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1307,\n",
            "  \"input\": \"Do NER on this sentence\\nIn P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1309/1344 (0.65 seconds):\n",
            "{\n",
            "  \"id\": 1308,\n",
            "  \"input\": \"Do NER on this sentence\\nfluorescens ColS has been implicated in root colonization while in Pseudomonas putida it has been reported to be involved in regulating TN4652 transposition and heavy metal resistance [ 37 ] , [ 38 ] , [ 39 ] , [ 40 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:fluorescens', 'B-Organism:Pseudomonas', 'E-Organism:putida']\",\n",
            "  \"response\": \"this sentence\\nfluorescens ColS has been implicated in root colonization while in Pseudomonas putida it has been reported to be involved in regulating TN4652 transposition and heavy metal resistance [ 37 ]\"\n",
            "}\n",
            "1310/1344 (1.35 seconds):\n",
            "{\n",
            "  \"id\": 1309,\n",
            "  \"input\": \"Do NER on this sentence\\nFirst , PA4953 encodes a protein with similarity to the E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "1311/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1310,\n",
            "  \"input\": \"Do NER on this sentence\\nSecondly , we found an insertion in PA0173 ( cheB2 ) , one of four cheB gene homologues in the P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1312/1344 (1.36 seconds):\n",
            "{\n",
            "  \"id\": 1311,\n",
            "  \"input\": \"Do NER on this sentence\\nIn the remaining mutants , one contained a transposon insertion in PA5479 , which encodes a protein similar to GltP from E\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "1313/1344 (0.27 seconds):\n",
            "{\n",
            "  \"id\": 1312,\n",
            "  \"input\": \"Do NER on this sentence\\nThe uvrC gene is located downstream from the gacA gene encoding a response regulator known to be involved in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1314/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1313,\n",
            "  \"input\": \"Do NER on this sentence\\nIn order to compare our liquid assay with the established C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1315/1344 (6.67 seconds):\n",
            "{\n",
            "  \"id\": 1314,\n",
            "  \"input\": \"Do NER on this sentence\\nPhenotypic Analysis of Selected TB mutants Several of the clones recovered were mutated in genes putatively associated with adherence and motility , such as cheB2 ( PA0173 ) , motB ( PA4953 ) and pilY1 ( PA4554 )\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:TB', 'E-Organism:mutants']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "1316/1344 (3.20 seconds):\n",
            "{\n",
            "  \"id\": 1315,\n",
            "  \"input\": \"Do NER on this sentence\\nIt has been shown that adherence is an essential first step in the process of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nIt has been shown that adherence is an essential first step in the process of P\\nReply in key value pair as Tag and Entity. and received in the surface of the host, in this case mouse blood, within 30 minutes [ 9 ]\"\n",
            "}\n",
            "1317/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1316,\n",
            "  \"input\": \"Do NER on this sentence\\nMotility is also an important factor for P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1318/1344 (0.95 seconds):\n",
            "{\n",
            "  \"id\": 1317,\n",
            "  \"input\": \"Do NER on this sentence\\n005 ) , showed a delayed C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1319/1344 (4.14 seconds):\n",
            "{\n",
            "  \"id\": 1318,\n",
            "  \"input\": \"Do NER on this sentence\\nIn addition , we engineered a similar cheB2 mutation in the P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:invasive', 'E-Organism:GAS', 'B-Organism:non', 'I-Organism:invasive', 'E-Organism:GAS']\"\n",
            "}\n",
            "1320/1344 (1.56 seconds):\n",
            "{\n",
            "  \"id\": 1319,\n",
            "  \"input\": \"Do NER on this sentence\\naeruginosa PA14 strain , yielding PA140173i ( see Materials and Methods ) , and observed similar virulence attenuation on C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:aeruginosa', 'I-Organism:PA14', 'E-Organism:strain', 'S-Organism:PA140173i']\",\n",
            "  \"response\": \"['E-Organism:enterococci', 'B-Organism:enterococci', 'E-Organism:strains']\"\n",
            "}\n",
            "1321/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1320,\n",
            "  \"input\": \"Do NER on this sentence\\nThis vector contains the phiCTX attachment site allowing integration of exogenous DNA fragments at the attB site within the genome of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1322/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1321,\n",
            "  \"input\": \"Do NER on this sentence\\nA cheB1 Mutant Is not Attenuated for C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1323/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 1322,\n",
            "  \"input\": \"Do NER on this sentence\\nelegans Virulence Several chemotactic gene clusters have been identified in the P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:elegans']\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "1324/1344 (1.41 seconds):\n",
            "{\n",
            "  \"id\": 1323,\n",
            "  \"input\": \"Do NER on this sentence\\nWe investigated whether the loss of virulence observed in the cheB2 mutant on C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['B-Organism:cheB2', 'E-Organism:flc', 'S-Organism:mouse']\"\n",
            "}\n",
            "1325/1344 (0.21 seconds):\n",
            "{\n",
            "  \"id\": 1324,\n",
            "  \"input\": \"Do NER on this sentence\\nThe cheB1 mutant was used to feed C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"this sentence\\nThe cheB1 mutant was used to feed C\\nReply in key value pair as Tag and Entity.Response:[]\"\n",
            "}\n",
            "1326/1344 (0.41 seconds):\n",
            "{\n",
            "  \"id\": 1325,\n",
            "  \"input\": \"Do NER on this sentence\\nInterestingly , the cheB1 mutant showed no significant reduction in C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1327/1344 (1.45 seconds):\n",
            "{\n",
            "  \"id\": 1326,\n",
            "  \"input\": \"Do NER on this sentence\\nTen female C3H / HeN mice each were inoculated with 7\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:C3H', 'I-Organism:HeN', 'E-Organism:mice']\",\n",
            "  \"response\": \"['B-Organism:C57BL', 'E-Organism:mice']\"\n",
            "}\n",
            "1328/1344 (0.46 seconds):\n",
            "{\n",
            "  \"id\": 1327,\n",
            "  \"input\": \"Do NER on this sentence\\nPathohistological Analysis of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1329/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1328,\n",
            "  \"input\": \"Do NER on this sentence\\nDiscussion C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1330/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 1329,\n",
            "  \"input\": \"Do NER on this sentence\\nelegans is a convenient model for studying bacterial virulence mechanisms [ 7 ] , [ 49 ] and it was shown that virulence factors important in the killing of C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:elegans']\",\n",
            "  \"response\": \"this sentence\\nelegans is a convenient model for studying bacterial virulence mechanisms [ 7 ]\"\n",
            "}\n",
            "1331/1344 (0.61 seconds):\n",
            "{\n",
            "  \"id\": 1330,\n",
            "  \"input\": \"Do NER on this sentence\\nGrowth conditions and host - pathogen interactions affect the physiology of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1332/1344 (0.36 seconds):\n",
            "{\n",
            "  \"id\": 1331,\n",
            "  \"input\": \"Do NER on this sentence\\nWe developed a liquid - based high throughput C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1333/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1332,\n",
            "  \"input\": \"Do NER on this sentence\\nWe used our killing assay to screen a STM library of the P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1334/1344 (0.76 seconds):\n",
            "{\n",
            "  \"id\": 1333,\n",
            "  \"input\": \"Do NER on this sentence\\naeruginosa CF isolate TB [ 20 ]\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['I-Organism:aeruginosa', 'I-Organism:CF', 'I-Organism:isolate', 'E-Organism:TB']\",\n",
            "  \"response\": \"this sentence\\naeruginosa CF isolate TB [ 20 ]\"\n",
            "}\n",
            "1335/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1334,\n",
            "  \"input\": \"Do NER on this sentence\\nWe identified several factors , not yet associated with virulence in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1336/1344 (0.46 seconds):\n",
            "{\n",
            "  \"id\": 1335,\n",
            "  \"input\": \"Do NER on this sentence\\nWe tested additional virulence and motility related phenotypes including C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1337/1344 (0.37 seconds):\n",
            "{\n",
            "  \"id\": 1336,\n",
            "  \"input\": \"Do NER on this sentence\\nThis mutant , named TB0173s , has a transposon insertion in PA0173 , cheB2 , one of four cheB genes found in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['S-Organism:TB0173s']\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1338/1344 (0.96 seconds):\n",
            "{\n",
            "  \"id\": 1337,\n",
            "  \"input\": \"Do NER on this sentence\\nAs many as 26 MCPs have been identified in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1339/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1338,\n",
            "  \"input\": \"Do NER on this sentence\\nIn the high throughput C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1340/1344 (0.26 seconds):\n",
            "{\n",
            "  \"id\": 1339,\n",
            "  \"input\": \"Do NER on this sentence\\nAs cluster I of che genes plays the dominant role in P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1341/1344 (2.07 seconds):\n",
            "{\n",
            "  \"id\": 1340,\n",
            "  \"input\": \"Do NER on this sentence\\nIt has been shown that a cheW mutant in Helicobacter pylori is unable to establish a normal long - term infection in mice , remaining only in one portion of the stomach\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['B-Organism:Helicobacter', 'E-Organism:pylori', 'S-Organism:mice']\",\n",
            "  \"response\": \"['B-Organism:hominis', 'E-Organism:faecalis', 'B-Organism:Faecalibacterium', 'E-Organism:jejuni']\"\n",
            "}\n",
            "1342/1344 (0.86 seconds):\n",
            "{\n",
            "  \"id\": 1341,\n",
            "  \"input\": \"Do NER on this sentence\\nIt is a possibility that within a host , the cluster II che genes of P\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"[]\"\n",
            "}\n",
            "1343/1344 (1.36 seconds):\n",
            "{\n",
            "  \"id\": 1342,\n",
            "  \"input\": \"Do NER on this sentence\\nWhile some bacteria require host - derived factors for their growth and are difficult to grow in vitro , others , such as S\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"[]\",\n",
            "  \"response\": \"['E-Organism:enterocolitica']\"\n",
            "}\n",
            "1344/1344 (0.06 seconds):\n",
            "{\n",
            "  \"id\": 1343,\n",
            "  \"input\": \"Do NER on this sentence\\nmarcescens during its infection of C\\nReply in key value pair as Tag and Entity.\",\n",
            "  \"expected\": \"['E-Organism:marcescens']\",\n",
            "  \"response\": \"\"\n",
            "}\n",
            "Processing completed.\n"
          ]
        }
      ],
      "source": [
        "import json\n",
        "import time\n",
        "\n",
        "processed_texts = []\n",
        "\n",
        "# Assume prompts is a list of prompt strings\n",
        "# Assume df is a DataFrame with a 'response' column containing expected responses\n",
        "\n",
        "for idx, prompt in enumerate(prompts):\n",
        "    start_time = time.time()  # Start time of the iteration\n",
        "\n",
        "    try:\n",
        "        prompt = prompt.strip(\"\\nInstruction:\\n\").strip(\"\\\\\").replace(\"\\\\n\", \"\").replace('\\\"', \"\").replace(\"'\", \"\").replace(\"Do NER on this sentence.\", \"Do NER on this sentence\\n\")\n",
        "        result = gen(f\"\"\"{prompt}\"\"\")  # Assuming gen is your function to generate response\n",
        "        response = result[0]['generated_text']\n",
        "        response = response[:response.find(\"]\") + 1]\n",
        "        response = response[response.find(\"Response:\\n\") + len(\"Response:\\n\"):].strip()\n",
        "        processed_text = {\n",
        "            'id': idx,\n",
        "            \"input\": prompt.strip(\"\\nInstruction:\\n\"),\n",
        "            \"expected\": df.response[idx].strip(\"\\n\\nResponse:\\n\"),\n",
        "            \"response\": response\n",
        "        }\n",
        "        processed_texts.append(processed_text)\n",
        "\n",
        "        # Calculate elapsed time for this iteration\n",
        "        elapsed_time = time.time() - start_time\n",
        "\n",
        "        print(f\"{idx + 1}/{len(prompts)} ({elapsed_time:.2f} seconds):\")\n",
        "        print(json.dumps(processed_text, indent=2))\n",
        "\n",
        "        # Write to file each time in case of failure in middle of the process\n",
        "        with open(\"/content/drive/MyDrive/bloom3b/responses_bloom3b_species_fast.json\", \"w\") as f:\n",
        "            json.dump(processed_texts, f)\n",
        "\n",
        "    except Exception as e:\n",
        "        print(\"Error on prompt\", idx, \":\", str(e))\n",
        "        continue\n",
        "\n",
        "print(\"Processing completed.\")\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "bXEyBsn6vj6W"
      },
      "execution_count": null,
      "outputs": []
    }
  ],
  "metadata": {
    "colab": {
      "machine_shape": "hm",
      "provenance": [],
      "gpuType": "L4",
      "collapsed_sections": [
        "JDYuXgClQtrl",
        "DGfv4vZHQyTN",
        "QqOGlPj8Q3JN",
        "V6-RG0lBRO-F"
      ]
    },
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "name": "python"
    },
    "widgets": {
      "application/vnd.jupyter.widget-state+json": {
        "cdcb1e3947354fa398d4cbb64cb10fcf": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_25048bfc6b6a44b287db0fcef1fbcd20",
              "IPY_MODEL_0a0036442a6c499395b27756e526d273",
              "IPY_MODEL_df4cd6edbe8b4780b87b2d6f7ad7207a"
            ],
            "layout": "IPY_MODEL_3ac8cc0c89b94975b7579657180609ac"
          }
        },
        "25048bfc6b6a44b287db0fcef1fbcd20": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_abf40bbf88bd44cd8c48f2f18784ac3f",
            "placeholder": "",
            "style": "IPY_MODEL_1bfc6fea94474e4eb13abba0204dbc27",
            "value": "Loadingcheckpointshards:100%"
          }
        },
        "0a0036442a6c499395b27756e526d273": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_de402b69291c4b74a77294a80d190004",
            "max": 2,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_83eb5628623e49fbbd8a4880310fac65",
            "value": 2
          }
        },
        "df4cd6edbe8b4780b87b2d6f7ad7207a": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_7f6f544bf3984a7bace828b8312d9042",
            "placeholder": "",
            "style": "IPY_MODEL_10432bc2ee904a6685e4cee084499d5d",
            "value": "2/2[00:02&lt;00:00,1.10it/s]"
          }
        },
        "3ac8cc0c89b94975b7579657180609ac": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "abf40bbf88bd44cd8c48f2f18784ac3f": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "1bfc6fea94474e4eb13abba0204dbc27": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "de402b69291c4b74a77294a80d190004": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "83eb5628623e49fbbd8a4880310fac65": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "7f6f544bf3984a7bace828b8312d9042": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "10432bc2ee904a6685e4cee084499d5d": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        }
      }
    },
    "accelerator": "GPU"
  },
  "nbformat": 4,
  "nbformat_minor": 0
}